{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": []
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "cells": [
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "GfPQzvJwaNqC"
      },
      "outputs": [],
      "source": [
        "#pubmed api\n",
        "search = 'https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=pmc&'\n",
        "'''\n",
        "# txt 파일을 통한 쿼리 입력\n",
        "with open(\"term.txt\", \"r\") as term_file:\n",
        "    query = term_file.read().strip()\n",
        "'''\n",
        "\n",
        "# case1\n",
        "#query = '(Monkeypox virus[Title/Abstract] OR monkeypox virus[Title/Abstract] OR monkeypox*[Title/Abstract] OR MPXV[Title/Abstract] OR Orthopoxvirus[Title/Abstract] OR Orthopox*[Title/Abstract] OR OPXV[Title/Abstract]) AND (PCR[Title/Abstract] OR polymerase chain reaction*[Title/Abstract] OR real time*[Title/Abstract] OR real-time PCR[Title/Abstract] OR real time PCR[Title/Abstract]) AND (primer[All Fields] OR probe[All Fields] OR molecule[All Fields] OR diagnostic[All Fields])'\n",
        "# case2\n",
        "query = '(Shigella[All Fields] OR Shigella spp.[All Fields] OR Shigella sp.[All Fields]) AND (PCR[TIAB] OR polymerase chain reaction*[TIAB] OR real time*[TIAB] OR real-time PCR[TIAB]) AND (primer[All Fields] OR probe[All Fields]) AND (molecular[All Fields] OR diagnostic[All Fields])'\n",
        "\n",
        "retmax = \"100\"\n",
        "url=search+\"term=\"+query+\"&retmode=json&retmax=\"+retmax\n"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "print(url)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "zx-oHNGNRvQa",
        "outputId": "c403c6a8-2062-4e5a-b309-c45c555aa631"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=pmc&term=(Shigella[All Fields] OR Shigella spp.[All Fields] OR Shigella sp.[All Fields]) AND (PCR[TIAB] OR polymerase chain reaction*[TIAB] OR real time*[TIAB] OR real-time PCR[TIAB]) AND (primer[All Fields] OR probe[All Fields]) AND (molecular[All Fields] OR diagnostic[All Fields])&retmode=json&retmax=100\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "import pprint\n",
        "import requests\n",
        "data = requests.get(url).json()\n",
        "pprint.pprint(data)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "XmS3joHPgUp-",
        "outputId": "95decb96-394f-427d-a942-8f92d71bd798"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "{'esearchresult': {'count': '19250',\n",
            "                   'errorlist': {'fieldsnotfound': [], 'phrasesnotfound': []},\n",
            "                   'idlist': ['11342756',\n",
            "                              '11342206',\n",
            "                              '11341105',\n",
            "                              '11340603',\n",
            "                              '11340327',\n",
            "                              '11336092',\n",
            "                              '11326835',\n",
            "                              '11326090',\n",
            "                              '11324020',\n",
            "                              '11323537',\n",
            "                              '11323402',\n",
            "                              '11322368',\n",
            "                              '11321392',\n",
            "                              '11319344',\n",
            "                              '11319304',\n",
            "                              '11318556',\n",
            "                              '11316331',\n",
            "                              '11315647',\n",
            "                              '11315073',\n",
            "                              '11313837',\n",
            "                              '11313783',\n",
            "                              '11312530',\n",
            "                              '11312258',\n",
            "                              '11312129',\n",
            "                              '11311926',\n",
            "                              '11311727',\n",
            "                              '11311408',\n",
            "                              '11311048',\n",
            "                              '11310248',\n",
            "                              '11304742',\n",
            "                              '11304630',\n",
            "                              '11304591',\n",
            "                              '11303641',\n",
            "                              '11302663',\n",
            "                              '11302304',\n",
            "                              '11300865',\n",
            "                              '11300684',\n",
            "                              '11300682',\n",
            "                              '11300174',\n",
            "                              '11237486',\n",
            "                              '11299273',\n",
            "                              '11298530',\n",
            "                              '11293832',\n",
            "                              '11293673',\n",
            "                              '11293325',\n",
            "                              '11292733',\n",
            "                              '11291472',\n",
            "                              '11291149',\n",
            "                              '11289621',\n",
            "                              '11289428',\n",
            "                              '11288818',\n",
            "                              '11285316',\n",
            "                              '11284562',\n",
            "                              '11281598',\n",
            "                              '11281493',\n",
            "                              '11279407',\n",
            "                              '11279152',\n",
            "                              '11278787',\n",
            "                              '11275779',\n",
            "                              '11274912',\n",
            "                              '11274296',\n",
            "                              '11273848',\n",
            "                              '11272550',\n",
            "                              '11271939',\n",
            "                              '11271649',\n",
            "                              '11269248',\n",
            "                              '11268173',\n",
            "                              '11267873',\n",
            "                              '11265448',\n",
            "                              '11263920',\n",
            "                              '11262882',\n",
            "                              '11261454',\n",
            "                              '11260827',\n",
            "                              '11260685',\n",
            "                              '11257890',\n",
            "                              '11255850',\n",
            "                              '11253639',\n",
            "                              '11252519',\n",
            "                              '11251922',\n",
            "                              '11249912',\n",
            "                              '11246856',\n",
            "                              '11244529',\n",
            "                              '11243892',\n",
            "                              '11243103',\n",
            "                              '11242183',\n",
            "                              '11241808',\n",
            "                              '11240908',\n",
            "                              '11240787',\n",
            "                              '11240761',\n",
            "                              '11240451',\n",
            "                              '11239581',\n",
            "                              '11238125',\n",
            "                              '11237755',\n",
            "                              '11237395',\n",
            "                              '11343591',\n",
            "                              '11342760',\n",
            "                              '11341846',\n",
            "                              '11340806',\n",
            "                              '11340127',\n",
            "                              '11339456'],\n",
            "                   'querytranslation': '((\"shigella\"[MeSH Terms] OR '\n",
            "                                       '\"shigella\"[All Fields]) OR Shigella '\n",
            "                                       'spp.[All Fields] OR Shigella sp.[All '\n",
            "                                       'Fields]) AND (PCR[All Fields] OR '\n",
            "                                       '(polymerase chain reaction[All Fields] '\n",
            "                                       'OR polymerase chain reaction/16s[All '\n",
            "                                       'Fields] OR polymerase chain '\n",
            "                                       'reaction/amplification[All Fields] OR '\n",
            "                                       'polymerase chain reaction/aso[All '\n",
            "                                       'Fields] OR polymerase chain '\n",
            "                                       'reaction/base[All Fields] OR '\n",
            "                                       'polymerase chain '\n",
            "                                       'reaction/capillary[All Fields] OR '\n",
            "                                       'polymerase chain reaction/cdna[All '\n",
            "                                       'Fields] OR polymerase chain '\n",
            "                                       'reaction/classification[All Fields] OR '\n",
            "                                       'polymerase chain '\n",
            "                                       'reaction/denaturing[All Fields] OR '\n",
            "                                       'polymerase chain reaction/direct[All '\n",
            "                                       'Fields] OR polymerase chain '\n",
            "                                       'reaction/dna[All Fields] OR polymerase '\n",
            "                                       'chain reaction/economics[All Fields] '\n",
            "                                       'OR polymerase chain '\n",
            "                                       'reaction/electrospray[All Fields] OR '\n",
            "                                       'polymerase chain reaction/enzyme[All '\n",
            "                                       'Fields] OR polymerase chain '\n",
            "                                       'reaction/expression[All Fields] OR '\n",
            "                                       'polymerase chain '\n",
            "                                       'reaction/fluorescence[All Fields] OR '\n",
            "                                       'polymerase chain reaction/gene[All '\n",
            "                                       'Fields] OR polymerase chain '\n",
            "                                       'reaction/heteroduplex[All Fields] OR '\n",
            "                                       'polymerase chain reaction/high[All '\n",
            "                                       'Fields] OR polymerase chain '\n",
            "                                       'reaction/history[All Fields] OR '\n",
            "                                       'polymerase chain '\n",
            "                                       'reaction/hybridization[All Fields] OR '\n",
            "                                       'polymerase chain '\n",
            "                                       'reaction/immunohistochemistry[All '\n",
            "                                       'Fields] OR polymerase chain '\n",
            "                                       'reaction/in[All Fields] OR polymerase '\n",
            "                                       'chain reaction/instrumentation[All '\n",
            "                                       'Fields] OR polymerase chain '\n",
            "                                       'reaction/ligase[All Fields] OR '\n",
            "                                       'polymerase chain reaction/ligation[All '\n",
            "                                       'Fields] OR polymerase chain '\n",
            "                                       'reaction/loss[All Fields] OR '\n",
            "                                       'polymerase chain reaction/luminex[All '\n",
            "                                       'Fields] OR polymerase chain '\n",
            "                                       'reaction/mass[All Fields] OR '\n",
            "                                       'polymerase chain reaction/methods[All '\n",
            "                                       'Fields] OR polymerase chain '\n",
            "                                       'reaction/microarray[All Fields] OR '\n",
            "                                       'polymerase chain '\n",
            "                                       'reaction/microarrays[All Fields] OR '\n",
            "                                       'polymerase chain '\n",
            "                                       'reaction/molecular[All Fields] OR '\n",
            "                                       'polymerase chain '\n",
            "                                       'reaction/multiplex[All Fields] OR '\n",
            "                                       'polymerase chain reaction/nucleic[All '\n",
            "                                       'Fields] OR polymerase chain '\n",
            "                                       'reaction/oligonucleotide[All Fields] '\n",
            "                                       'OR polymerase chain reaction/pcr[All '\n",
            "                                       'Fields] OR polymerase chain '\n",
            "                                       'reaction/polyacrylamide[All Fields] OR '\n",
            "                                       'polymerase chain '\n",
            "                                       'reaction/pyrosequencing[All Fields] OR '\n",
            "                                       'polymerase chain reaction/qpcr[All '\n",
            "                                       'Fields] OR polymerase chain '\n",
            "                                       'reaction/quantitative[All Fields] OR '\n",
            "                                       'polymerase chain '\n",
            "                                       'reaction/restriction[All Fields] OR '\n",
            "                                       'polymerase chain reaction/reverse[All '\n",
            "                                       'Fields] OR polymerase chain '\n",
            "                                       'reaction/rflp[All Fields] OR '\n",
            "                                       'polymerase chain reaction/rna[All '\n",
            "                                       'Fields] OR polymerase chain '\n",
            "                                       'reaction/sanger[All Fields] OR '\n",
            "                                       'polymerase chain reaction/sequence[All '\n",
            "                                       'Fields] OR polymerase chain '\n",
            "                                       'reaction/sequencing[All Fields] OR '\n",
            "                                       'polymerase chain reaction/single[All '\n",
            "                                       'Fields] OR polymerase chain '\n",
            "                                       'reaction/southern[All Fields] OR '\n",
            "                                       'polymerase chain '\n",
            "                                       'reaction/standards[All Fields] OR '\n",
            "                                       'polymerase chain '\n",
            "                                       'reaction/transcript[All Fields] OR '\n",
            "                                       'polymerase chain reaction/trends[All '\n",
            "                                       'Fields] OR polymerase chain '\n",
            "                                       'reaction/veterinary[All Fields] OR '\n",
            "                                       'polymerase chain reaction/viral[All '\n",
            "                                       'Fields] OR polymerase chain '\n",
            "                                       \"reaction's[All Fields] OR polymerase \"\n",
            "                                       'chain reaction,[All Fields] OR '\n",
            "                                       'polymerase chain reactiona[All Fields] '\n",
            "                                       'OR polymerase chain '\n",
            "                                       'reactionanalysis[All Fields] OR '\n",
            "                                       'polymerase chain reactionassay[All '\n",
            "                                       'Fields] OR polymerase chain '\n",
            "                                       'reactionbased[All Fields] OR '\n",
            "                                       'polymerase chain reactiond[All Fields] '\n",
            "                                       'OR polymerase chain '\n",
            "                                       'reactionerestriction[All Fields] OR '\n",
            "                                       'polymerase chain reactionexclusion[All '\n",
            "                                       'Fields] OR polymerase chain '\n",
            "                                       'reactionpcr[All Fields] OR polymerase '\n",
            "                                       'chain reactionreacao[All Fields] OR '\n",
            "                                       'polymerase chain '\n",
            "                                       'reactionrestriction[All Fields] OR '\n",
            "                                       'polymerase chain reactionrt[All '\n",
            "                                       'Fields] OR polymerase chain '\n",
            "                                       'reactions[All Fields] OR polymerase '\n",
            "                                       \"chain reactions'[All Fields] OR \"\n",
            "                                       'polymerase chain '\n",
            "                                       'reactionvalidation[All Fields]) OR '\n",
            "                                       '(real time[All Fields] OR real '\n",
            "                                       'time/1[All Fields] OR real time/3d[All '\n",
            "                                       'Fields] OR real time/4d[All Fields] OR '\n",
            "                                       'real time/accelerated[All Fields] OR '\n",
            "                                       'real time/active[All Fields] OR real '\n",
            "                                       'time/asynchronous[All Fields] OR real '\n",
            "                                       'time/automated/continuous[All Fields] '\n",
            "                                       'OR real time/batch[All Fields] OR real '\n",
            "                                       'time/bci[All Fields] OR real '\n",
            "                                       'time/bedside[All Fields] OR real '\n",
            "                                       'time/behavioral[All Fields] OR real '\n",
            "                                       'time/best[All Fields] OR real '\n",
            "                                       'time/bilateral[All Fields] OR real '\n",
            "                                       'time/c1000[All Fields] OR real '\n",
            "                                       'time/chip[All Fields] OR real '\n",
            "                                       'time/clinical[All Fields] OR real '\n",
            "                                       'time/close[All Fields] OR real '\n",
            "                                       'time/color[All Fields] OR real '\n",
            "                                       'time/conditioned[All Fields] OR real '\n",
            "                                       'time/continuous[All Fields] OR real '\n",
            "                                       'time/conventional[All Fields] OR real '\n",
            "                                       'time/cost[All Fields] OR real '\n",
            "                                       'time/current[All Fields] OR real '\n",
            "                                       'time/daily[All Fields] OR real '\n",
            "                                       'time/deadline/timeliness[All Fields] '\n",
            "                                       'OR real time/diagnostic[All Fields] OR '\n",
            "                                       'real time/digital[All Fields] OR real '\n",
            "                                       'time/direct[All Fields] OR real '\n",
            "                                       'time/doppler[All Fields] OR real '\n",
            "                                       'time/dynamic[All Fields] OR real '\n",
            "                                       'time/dynamic/operational[All Fields] '\n",
            "                                       'OR real time/effective[All Fields] OR '\n",
            "                                       'real time/emergency[All Fields] OR '\n",
            "                                       'real time/endpoint[All Fields] OR real '\n",
            "                                       'time/error[All Fields] OR real '\n",
            "                                       'time/fast[All Fields] OR real '\n",
            "                                       'time/feedback[All Fields] OR real '\n",
            "                                       'time/flexible[All Fields] OR real '\n",
            "                                       'time/fluorescence[All Fields] OR real '\n",
            "                                       'time/fresh[All Fields] OR real '\n",
            "                                       'time/gravimetric[All Fields] OR real '\n",
            "                                       'time/high[All Fields] OR real '\n",
            "                                       'time/historic[All Fields] OR real '\n",
            "                                       'time/historical[All Fields] OR real '\n",
            "                                       'time/in[All Fields] OR real '\n",
            "                                       'time/indirect[All Fields] OR real '\n",
            "                                       'time/influence[All Fields] OR real '\n",
            "                                       'time/intermittent[All Fields] OR real '\n",
            "                                       'time/intraoperative[All Fields] OR '\n",
            "                                       'real time/ivr[All Fields] OR real '\n",
            "                                       'time/just[All Fields] OR real '\n",
            "                                       'time/level[All Fields] OR real '\n",
            "                                       'time/lfs[All Fields] OR real '\n",
            "                                       'time/life[All Fields] OR real '\n",
            "                                       'time/limited[All Fields] OR real '\n",
            "                                       'time/live[All Fields] OR real '\n",
            "                                       'time/live/interactive[All Fields] OR '\n",
            "                                       'real time/live/robotic[All Fields] OR '\n",
            "                                       'real time/living[All Fields] OR real '\n",
            "                                       'time/longitudinal[All Fields] OR real '\n",
            "                                       'time/low[All Fields] OR real '\n",
            "                                       'time/mass[All Fields] OR real '\n",
            "                                       'time/medium[All Fields] OR real '\n",
            "                                       'time/microarray[All Fields] OR real '\n",
            "                                       'time/momentary[All Fields] OR real '\n",
            "                                       'time/narrow[All Fields] OR real '\n",
            "                                       'time/near[All Fields] OR real '\n",
            "                                       'time/nested[All Fields] OR real '\n",
            "                                       'time/offline[All Fields] OR real '\n",
            "                                       'time/online[All Fields] OR real '\n",
            "                                       'time/online/adaptive[All Fields] OR '\n",
            "                                       'real time/operational[All Fields] OR '\n",
            "                                       'real time/or[All Fields] OR real '\n",
            "                                       'time/orbitrap[All Fields] OR real '\n",
            "                                       'time/pacbio[All Fields] OR real '\n",
            "                                       'time/pcr[All Fields] OR real '\n",
            "                                       'time/pcr,[All Fields] OR real '\n",
            "                                       'time/performance[All Fields] OR real '\n",
            "                                       'time/poc[All Fields] OR real '\n",
            "                                       'time/point[All Fields] OR real '\n",
            "                                       'time/polymerase[All Fields] OR real '\n",
            "                                       'time/potential[All Fields] OR real '\n",
            "                                       'time/practical[All Fields] OR real '\n",
            "                                       'time/qpcr[All Fields] OR real '\n",
            "                                       'time/quality[All Fields] OR real '\n",
            "                                       'time/quantitative[All Fields] OR real '\n",
            "                                       'time/rapid[All Fields] OR real '\n",
            "                                       'time/real[All Fields] OR real '\n",
            "                                       'time/recent[All Fields] OR real '\n",
            "                                       'time/recorded[All Fields] OR real '\n",
            "                                       'time/reference[All Fields] OR real '\n",
            "                                       'time/remote[All Fields] OR real '\n",
            "                                       'time/retrospective[All Fields] OR real '\n",
            "                                       'time/reverse[All Fields] OR real '\n",
            "                                       'time/rt[All Fields] OR real '\n",
            "                                       'time/safety[All Fields] OR real '\n",
            "                                       'time/sensor[All Fields] OR real '\n",
            "                                       'time/serial[All Fields] OR real '\n",
            "                                       'time/short[All Fields] OR real '\n",
            "                                       'time/solution[All Fields] OR real '\n",
            "                                       'time/space[All Fields] OR real '\n",
            "                                       'time/spot[All Fields] OR real '\n",
            "                                       'time/static[All Fields] OR real '\n",
            "                                       'time/still[All Fields] OR real '\n",
            "                                       'time/stored[All Fields] OR real '\n",
            "                                       'time/streaming[All Fields] OR real '\n",
            "                                       'time/synchronous[All Fields] OR real '\n",
            "                                       'time/temperature[All Fields] OR real '\n",
            "                                       'time/temporal[All Fields] OR real '\n",
            "                                       'time/time[All Fields] OR real '\n",
            "                                       'time/timeliness[All Fields] OR real '\n",
            "                                       'time/trace[All Fields] OR real '\n",
            "                                       'time/transit[All Fields] OR real '\n",
            "                                       'time/truenat[All Fields] OR real '\n",
            "                                       'time/ultrasound[All Fields] OR real '\n",
            "                                       'time/unblinded[All Fields] OR real '\n",
            "                                       'time/updated[All Fields] OR real '\n",
            "                                       'time/use[All Fields] OR real '\n",
            "                                       'time/wearable[All Fields] OR real '\n",
            "                                       'time/wide[All Fields] OR real '\n",
            "                                       'time/xpc[All Fields] OR real '\n",
            "                                       \"time'for[All Fields] OR real \"\n",
            "                                       \"time's[All Fields] OR real \"\n",
            "                                       \"time'taqman[All Fields] OR real \"\n",
            "                                       'time00205pcr[All Fields] OR real '\n",
            "                                       'time1[All Fields] OR real '\n",
            "                                       'time129xe[All Fields] OR real '\n",
            "                                       'time16s[All Fields] OR real time18[All '\n",
            "                                       'Fields] OR real time18,19,24[All '\n",
            "                                       'Fields] OR real time1h[All Fields] OR '\n",
            "                                       'real time2[All Fields] OR real '\n",
            "                                       'time33[All Fields] OR real time36[All '\n",
            "                                       'Fields] OR real time3d[All Fields] OR '\n",
            "                                       'real time6[All Fields] OR real '\n",
            "                                       'time8[All Fields] OR real timea[All '\n",
            "                                       'Fields] OR real timeaccessto[All '\n",
            "                                       'Fields] OR real timeafter[All Fields] '\n",
            "                                       'OR real timeanalysis[All Fields] OR '\n",
            "                                       'real timeas[All Fields] OR real '\n",
            "                                       'timeastandard[All Fields] OR real '\n",
            "                                       'timeat[All Fields] OR real timeb[All '\n",
            "                                       'Fields] OR real timebasis[All Fields] '\n",
            "                                       'OR real timebayesian[All Fields] OR '\n",
            "                                       'real timebike[All Fields] OR real '\n",
            "                                       'timebiosensor[All Fields] OR real '\n",
            "                                       'timebrain[All Fields] OR real '\n",
            "                                       'timebus[All Fields] OR real '\n",
            "                                       'timecell[All Fields] OR real '\n",
            "                                       'timecellular[All Fields] OR real '\n",
            "                                       'timecharacterization[All Fields] OR '\n",
            "                                       'real timechemistry[All Fields] OR real '\n",
            "                                       'timecommunication[All Fields] OR real '\n",
            "                                       'timeconsecutive[All Fields] OR real '\n",
            "                                       'timeconstraint[All Fields] OR real '\n",
            "                                       'timecourses[All Fields] OR real '\n",
            "                                       'timed[All Fields] OR real '\n",
            "                                       'timedagger[All Fields] OR real '\n",
            "                                       'timedental[All Fields] OR real '\n",
            "                                       'timedetection[All Fields] OR real '\n",
            "                                       'timedh[All Fields] OR real '\n",
            "                                       'timediagnosis[All Fields] OR real '\n",
            "                                       'timedna[All Fields] OR real '\n",
            "                                       'timedocument[All Fields] OR real '\n",
            "                                       'timedoes[All Fields] OR real '\n",
            "                                       'timeecg[All Fields] OR real '\n",
            "                                       'timeelastography[All Fields] OR real '\n",
            "                                       'timeevaluation[All Fields] OR real '\n",
            "                                       'timeexpression[All Fields] OR real '\n",
            "                                       'timefe[All Fields] OR real '\n",
            "                                       'timefeedback[All Fields] OR real '\n",
            "                                       'timefleet[All Fields] OR real '\n",
            "                                       'timefmrifunctional[All Fields] OR real '\n",
            "                                       'timeforecast[All Fields] OR real '\n",
            "                                       'timeglotrade[All Fields] OR real '\n",
            "                                       'timehand[All Fields] OR real '\n",
            "                                       'timehaptic[All Fields] OR real '\n",
            "                                       'timehead[All Fields] OR real '\n",
            "                                       'timeigm[All Fields] OR real '\n",
            "                                       'timeimagesof[All Fields] OR real '\n",
            "                                       'timeimaging[All Fields] OR real '\n",
            "                                       'timeimagingof[All Fields] OR real '\n",
            "                                       'timeime[All Fields] OR real timein[All '\n",
            "                                       'Fields] OR real timeinfluenza[All '\n",
            "                                       'Fields] OR real timeinteractions[All '\n",
            "                                       'Fields] OR real timelike[All Fields] '\n",
            "                                       'OR real timelimited[All Fields] OR '\n",
            "                                       'real timeline[All Fields] OR real '\n",
            "                                       'timelines[All Fields] OR real '\n",
            "                                       'timeliness[All Fields] OR real '\n",
            "                                       'timelogarithmic[All Fields] OR real '\n",
            "                                       'timely[All Fields] OR real timem[All '\n",
            "                                       'Fields] OR real timemanage[All Fields] '\n",
            "                                       'OR real timemastermix[All Fields] OR '\n",
            "                                       'real timemeasurement[All Fields] OR '\n",
            "                                       'real timemethods[All Fields] OR real '\n",
            "                                       'timemetro[All Fields] OR real '\n",
            "                                       'timemicro[All Fields] OR real '\n",
            "                                       'timemodeling[All Fields] OR real '\n",
            "                                       'timemonitoring[All Fields] OR real '\n",
            "                                       'timemonitoringas[All Fields] OR real '\n",
            "                                       'timemonitoringof[All Fields] OR real '\n",
            "                                       'timemovie[All Fields] OR real '\n",
            "                                       'timempcr[All Fields] OR real '\n",
            "                                       'timemri[All Fields] OR real '\n",
            "                                       'timemultiplex[All Fields] OR real '\n",
            "                                       'timenapcr[All Fields] OR real '\n",
            "                                       'timenear[All Fields] OR real '\n",
            "                                       'timeness[All Fields] OR real '\n",
            "                                       'timenoforwardingfull[All Fields] OR '\n",
            "                                       'real timeobser[All Fields] OR real '\n",
            "                                       'timeobservation[All Fields] OR real '\n",
            "                                       'timeolymerase[All Fields] OR real '\n",
            "                                       'timepcr[All Fields] OR real '\n",
            "                                       'timepcrdetection[All Fields] OR real '\n",
            "                                       'timepcrilluminar[All Fields] OR real '\n",
            "                                       'timepcrlncrna[All Fields] OR real '\n",
            "                                       'timepcrsystem[All Fields] OR real '\n",
            "                                       'timepcrtargetingvirbgene[All Fields] '\n",
            "                                       'OR real timeperson[All Fields] OR real '\n",
            "                                       'timephotoacoustic[All Fields] OR real '\n",
            "                                       'timepm2[All Fields] OR real '\n",
            "                                       'timepoints[All Fields] OR real '\n",
            "                                       'timepolymerase[All Fields] OR real '\n",
            "                                       'timepolymerasechainreaction[All '\n",
            "                                       'Fields] OR real timeprimer[All Fields] '\n",
            "                                       'OR real timeprimers[All Fields] OR '\n",
            "                                       'real timeproduct[All Fields] OR real '\n",
            "                                       'timeq[All Fields] OR real timeqpcr[All '\n",
            "                                       'Fields] OR real timequantitative[All '\n",
            "                                       'Fields] OR real '\n",
            "                                       'timequantitativereverse[All Fields] OR '\n",
            "                                       'real timer[All Fields] OR real '\n",
            "                                       'timereflectance[All Fields] OR real '\n",
            "                                       'timerelease[All Fields] OR real '\n",
            "                                       'timerepeat[All Fields] OR real '\n",
            "                                       'timeresponsiveness[All Fields] OR real '\n",
            "                                       'timereverse[All Fields] OR real '\n",
            "                                       'timereversetranscriptionpolymerasechainreaction[All '\n",
            "                                       'Fields] OR real '\n",
            "                                       'timernamycyclertmpcrcdnasybr[All '\n",
            "                                       'Fields] OR real timerpa[All Fields] OR '\n",
            "                                       'real timerpermanentdesign[All Fields] '\n",
            "                                       'OR real timers[All Fields] OR real '\n",
            "                                       'timert[All Fields] OR real '\n",
            "                                       'timertrade[All Fields] OR real '\n",
            "                                       'times[All Fields] OR real '\n",
            "                                       'timesbiotechnology[All Fields] OR real '\n",
            "                                       'timescale[All Fields] OR real '\n",
            "                                       'timescales[All Fields] OR real '\n",
            "                                       'timescripttrade[All Fields] OR real '\n",
            "                                       'timesemantic[All Fields] OR real '\n",
            "                                       'timesensitive[All Fields] OR real '\n",
            "                                       'timesequencing[All Fields] OR real '\n",
            "                                       'timespectrum[All Fields] OR real '\n",
            "                                       'timessa[All Fields] OR real '\n",
            "                                       'timestyle[All Fields] OR real '\n",
            "                                       'timesystem[All Fields] OR real '\n",
            "                                       'timesystems[All Fields] OR real '\n",
            "                                       'timetaqman[All Fields] OR real '\n",
            "                                       'timetarget[All Fields] OR real '\n",
            "                                       'timetechnique[All Fields] OR real '\n",
            "                                       'timetelephonic[All Fields] OR real '\n",
            "                                       'timetest[All Fields] OR real '\n",
            "                                       'timethermal[All Fields] OR real '\n",
            "                                       'timetime[All Fields] OR real '\n",
            "                                       'timetissue[All Fields] OR real '\n",
            "                                       'timetm[All Fields] OR real '\n",
            "                                       'timetrade[All Fields] OR real '\n",
            "                                       'timetumortracking[All Fields] OR real '\n",
            "                                       'timetwo[All Fields] OR real '\n",
            "                                       'timetype[All Fields] OR real '\n",
            "                                       'timeultrasonography[All Fields] OR '\n",
            "                                       'real timeultrasound[All Fields] OR '\n",
            "                                       'real timeuse[All Fields] OR real '\n",
            "                                       'timevideo[All Fields] OR real '\n",
            "                                       'timew[All Fields] OR real '\n",
            "                                       'timewater[All Fields] OR real '\n",
            "                                       'timewireless[All Fields]) OR '\n",
            "                                       '(\"real-time polymerase chain '\n",
            "                                       'reaction\"[MeSH Terms] OR '\n",
            "                                       '(\"real-time\"[All Fields] AND '\n",
            "                                       '\"polymerase\"[All Fields] AND '\n",
            "                                       '\"chain\"[All Fields] AND \"reaction\"[All '\n",
            "                                       'Fields]) OR \"real-time polymerase '\n",
            "                                       'chain reaction\"[All Fields] OR '\n",
            "                                       '(\"real\"[All Fields] AND \"time\"[All '\n",
            "                                       'Fields] AND \"pcr\"[All Fields]) OR '\n",
            "                                       '\"real time pcr\"[All Fields])) AND '\n",
            "                                       '((\"dna primers\"[MeSH Terms] OR '\n",
            "                                       '(\"dna\"[All Fields] AND \"primers\"[All '\n",
            "                                       'Fields]) OR \"dna primers\"[All Fields] '\n",
            "                                       'OR \"primer\"[All Fields]) OR probe[All '\n",
            "                                       'Fields]) AND (molecular[All Fields] OR '\n",
            "                                       '(\"diagnosis\"[MeSH Terms] OR '\n",
            "                                       '\"diagnosis\"[All Fields] OR '\n",
            "                                       '\"diagnostic\"[All Fields]))',\n",
            "                   'retmax': '100',\n",
            "                   'retstart': '0',\n",
            "                   'translationset': [{'from': 'Shigella[All Fields]',\n",
            "                                       'to': '\"shigella\"[MeSH Terms] OR '\n",
            "                                             '\"shigella\"[All Fields]'},\n",
            "                                      {'from': 'real-time PCR[All Fields]',\n",
            "                                       'to': '\"real-time polymerase chain '\n",
            "                                             'reaction\"[MeSH Terms] OR '\n",
            "                                             '(\"real-time\"[All Fields] AND '\n",
            "                                             '\"polymerase\"[All Fields] AND '\n",
            "                                             '\"chain\"[All Fields] AND '\n",
            "                                             '\"reaction\"[All Fields]) OR '\n",
            "                                             '\"real-time polymerase chain '\n",
            "                                             'reaction\"[All Fields] OR '\n",
            "                                             '(\"real\"[All Fields] AND '\n",
            "                                             '\"time\"[All Fields] AND \"pcr\"[All '\n",
            "                                             'Fields]) OR \"real time pcr\"[All '\n",
            "                                             'Fields]'},\n",
            "                                      {'from': 'primer[All Fields]',\n",
            "                                       'to': '\"dna primers\"[MeSH Terms] OR '\n",
            "                                             '(\"dna\"[All Fields] AND '\n",
            "                                             '\"primers\"[All Fields]) OR \"dna '\n",
            "                                             'primers\"[All Fields] OR '\n",
            "                                             '\"primer\"[All Fields]'},\n",
            "                                      {'from': 'diagnostic[All Fields]',\n",
            "                                       'to': '\"diagnosis\"[MeSH Terms] OR '\n",
            "                                             '\"diagnosis\"[All Fields] OR '\n",
            "                                             '\"diagnostic\"[All Fields]'}],\n",
            "                   'translationstack': [{'count': '2703',\n",
            "                                         'explode': 'Y',\n",
            "                                         'field': 'MeSH Terms',\n",
            "                                         'term': '\"shigella\"[MeSH Terms]'},\n",
            "                                        {'count': '60747',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': '\"shigella\"[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        'GROUP',\n",
            "                                        {'count': '8233',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'Shigella spp.[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1536',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'Shigella sp.[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        'GROUP',\n",
            "                                        {'count': '1751861',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'PCR[All Fields]'},\n",
            "                                        {'count': '623862',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction[All Fields]'},\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/16s[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/amplification[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/aso[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/base[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/capillary[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/cdna[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '3',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/classification[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '19',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/denaturing[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '6',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/direct[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '25',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/dna[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '161',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/economics[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '59',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/electrospray[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '64',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/enzyme[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/expression[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/fluorescence[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '5',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/gene[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '17',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/heteroduplex[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '3',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/high[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '6',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/history[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '19',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/hybridization[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/immunohistochemistry[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '136',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/in[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '203',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/instrumentation[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '131',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/ligase[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/ligation[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '5',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/loss[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/luminex[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/mass[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '12639',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/methods[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/microarray[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/microarrays[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/molecular[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/multiplex[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '11',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/nucleic[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '11',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/oligonucleotide[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '12',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/pcr[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/polyacrylamide[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '7',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/pyrosequencing[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/qpcr[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '9',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/quantitative[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '218',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/restriction[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '24',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/reverse[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '14',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/rflp[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/rna[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '21',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/sanger[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '18',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/sequence[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '13',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/sequencing[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '44',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/single[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '3',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/southern[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '453',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/standards[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/transcript[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '13',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/trends[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1277',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/veterinary[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction/viral[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 \"reaction's[All Fields]\"},\n",
            "                                        'OR',\n",
            "                                        {'count': '22629',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reaction,[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reactiona[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reactionanalysis[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reactionassay[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reactionbased[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reactiond[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reactionerestriction[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reactionexclusion[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reactionpcr[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reactionreacao[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '8',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reactionrestriction[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reactionrt[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '13894',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reactions[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 \"reactions'[All Fields]\"},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'polymerase chain '\n",
            "                                                 'reactionvalidation[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        'GROUP',\n",
            "                                        'OR',\n",
            "                                        {'count': '1300889',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time[All Fields]'},\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/1[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/3d[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/4d[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/accelerated[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/active[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/asynchronous[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real '\n",
            "                                                 'time/automated/continuous[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/batch[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/bci[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/bedside[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/behavioral[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/best[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/bilateral[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/c1000[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/chip[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/clinical[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/close[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '3',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/color[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/conditioned[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '4',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/continuous[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '6',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/conventional[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/cost[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/current[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/daily[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real '\n",
            "                                                 'time/deadline/timeliness[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '3',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/diagnostic[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '40',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/digital[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/direct[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/doppler[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '4',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/dynamic[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real '\n",
            "                                                 'time/dynamic/operational[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/effective[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/emergency[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/endpoint[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/error[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/fast[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/feedback[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/flexible[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/fluorescence[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/fresh[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/gravimetric[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/high[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/historic[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '5',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/historical[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '14',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/in[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/indirect[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/influence[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/intermittent[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/intraoperative[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/ivr[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/just[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/level[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/lfs[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/life[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/limited[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '14',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/live[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real '\n",
            "                                                 'time/live/interactive[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/live/robotic[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/living[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/longitudinal[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/low[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/mass[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/medium[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/microarray[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/momentary[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/narrow[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '9',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/near[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/nested[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '6',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/offline[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '17',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/online[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real '\n",
            "                                                 'time/online/adaptive[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/operational[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/or[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/orbitrap[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/pacbio[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/pcr[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/pcr,[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/performance[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/poc[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/point[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '3',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/polymerase[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/potential[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/practical[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '3',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/qpcr[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/quality[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '61',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/quantitative[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/rapid[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '76',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/real[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/recent[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/recorded[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/reference[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/remote[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/retrospective[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '16',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/reverse[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '7',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/rt[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/safety[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/sensor[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/serial[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/short[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/solution[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/space[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/spot[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/static[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/still[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/stored[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/streaming[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '10',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/synchronous[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/temperature[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/temporal[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '18',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/time[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/timeliness[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/trace[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/transit[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/truenat[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/ultrasound[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/unblinded[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/updated[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/use[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/wearable[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/wide[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time/xpc[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': \"real time'for[All Fields]\"},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': \"real time's[All Fields]\"},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': \"real time'taqman[All \"\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time00205pcr[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '3',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time1[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time129xe[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time16s[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time18[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time18,19,24[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time1h[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '3',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time2[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time33[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time36[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '4',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time3d[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time6[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real time8[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '3',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timea[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeaccessto[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeafter[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '3',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeanalysis[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeas[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeastandard[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeat[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeb[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timebasis[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timebayesian[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timebike[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timebiosensor[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timebrain[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timebus[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timecell[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timecellular[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real '\n",
            "                                                 'timecharacterization[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timechemistry[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timecommunication[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeconsecutive[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeconstraint[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timecourses[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '6',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timed[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timedagger[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timedental[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timedetection[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timedh[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timediagnosis[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timedna[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timedocument[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timedoes[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeecg[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeelastography[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeevaluation[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeexpression[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timefe[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timefeedback[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timefleet[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timefmrifunctional[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeforecast[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeglotrade[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timehand[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timehaptic[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timehead[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeigm[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeimagesof[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '3',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeimaging[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeimagingof[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeime[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '10',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timein[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeinfluenza[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeinteractions[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timelike[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timelimited[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeline[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timelines[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '13',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeliness[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timelogarithmic[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '201',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timely[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timem[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timemanage[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timemastermix[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timemeasurement[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timemethods[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timemetro[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timemicro[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timemodeling[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timemonitoring[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timemonitoringas[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timemonitoringof[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timemovie[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timempcr[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timemri[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timemultiplex[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timenapcr[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timenear[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '9',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeness[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real '\n",
            "                                                 'timenoforwardingfull[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeobser[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeobservation[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeolymerase[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '181',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timepcr[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timepcrdetection[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timepcrilluminar[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timepcrlncrna[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timepcrsystem[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real '\n",
            "                                                 'timepcrtargetingvirbgene[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeperson[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timephotoacoustic[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timepm2[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timepoints[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '28',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timepolymerase[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real '\n",
            "                                                 'timepolymerasechainreaction[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeprimer[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeprimers[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeproduct[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeq[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeqpcr[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '35',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timequantitative[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real '\n",
            "                                                 'timequantitativereverse[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '70',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timer[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '6',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timereflectance[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timerelease[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timerepeat[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeresponsiveness[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '4',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timereverse[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real '\n",
            "                                                 'timereversetranscriptionpolymerasechainreaction[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real '\n",
            "                                                 'timernamycyclertmpcrcdnasybr[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timerpa[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real '\n",
            "                                                 'timerpermanentdesign[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timers[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '7',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timert[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timertrade[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '403',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real times[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timesbiotechnology[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '6',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timescale[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timescales[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timescripttrade[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timesemantic[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timesensitive[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timesequencing[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timespectrum[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timessa[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timestyle[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '10',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timesystem[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timesystems[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timetaqman[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timetarget[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timetechnique[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timetelephonic[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timetest[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timethermal[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timetime[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timetissue[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '34',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timetm[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '114',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timetrade[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timetumortracking[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timetwo[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timetype[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeultrasonography[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeultrasound[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timeuse[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timevideo[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timew[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '1',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timewater[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '3',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'real timewireless[All '\n",
            "                                                 'Fields]'},\n",
            "                                        'OR',\n",
            "                                        'GROUP',\n",
            "                                        'OR',\n",
            "                                        {'count': '25130',\n",
            "                                         'explode': 'Y',\n",
            "                                         'field': 'MeSH Terms',\n",
            "                                         'term': '\"real-time polymerase chain '\n",
            "                                                 'reaction\"[MeSH Terms]'},\n",
            "                                        {'count': '1300889',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': '\"real-time\"[All Fields]'},\n",
            "                                        {'count': '1139497',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': '\"polymerase\"[All Fields]'},\n",
            "                                        'AND',\n",
            "                                        {'count': '1971544',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': '\"chain\"[All Fields]'},\n",
            "                                        'AND',\n",
            "                                        {'count': '2715744',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': '\"reaction\"[All Fields]'},\n",
            "                                        'AND',\n",
            "                                        'GROUP',\n",
            "                                        'OR',\n",
            "                                        {'count': '125627',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': '\"real-time polymerase chain '\n",
            "                                                 'reaction\"[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2446748',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': '\"real\"[All Fields]'},\n",
            "                                        {'count': '7790750',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': '\"time\"[All Fields]'},\n",
            "                                        'AND',\n",
            "                                        {'count': '1751861',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': '\"pcr\"[All Fields]'},\n",
            "                                        'AND',\n",
            "                                        'GROUP',\n",
            "                                        'OR',\n",
            "                                        {'count': '538365',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': '\"real time pcr\"[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        'GROUP',\n",
            "                                        'OR',\n",
            "                                        'GROUP',\n",
            "                                        'AND',\n",
            "                                        {'count': '24632',\n",
            "                                         'explode': 'Y',\n",
            "                                         'field': 'MeSH Terms',\n",
            "                                         'term': '\"dna primers\"[MeSH Terms]'},\n",
            "                                        {'count': '2581958',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': '\"dna\"[All Fields]'},\n",
            "                                        {'count': '1089909',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': '\"primers\"[All Fields]'},\n",
            "                                        'AND',\n",
            "                                        'GROUP',\n",
            "                                        'OR',\n",
            "                                        {'count': '36005',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': '\"dna primers\"[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '909571',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': '\"primer\"[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        'GROUP',\n",
            "                                        {'count': '1103034',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'probe[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        'GROUP',\n",
            "                                        'AND',\n",
            "                                        {'count': '4007506',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': 'molecular[All Fields]'},\n",
            "                                        {'count': '1446946',\n",
            "                                         'explode': 'Y',\n",
            "                                         'field': 'MeSH Terms',\n",
            "                                         'term': '\"diagnosis\"[MeSH Terms]'},\n",
            "                                        {'count': '3281368',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': '\"diagnosis\"[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        {'count': '2192693',\n",
            "                                         'explode': 'N',\n",
            "                                         'field': 'All Fields',\n",
            "                                         'term': '\"diagnostic\"[All Fields]'},\n",
            "                                        'OR',\n",
            "                                        'GROUP',\n",
            "                                        'OR',\n",
            "                                        'GROUP',\n",
            "                                        'AND']},\n",
            " 'header': {'type': 'esearch', 'version': '0.3'}}\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "import pandas as pd\n",
        "df = pd.DataFrame(data[\"esearchresult\"][\"idlist\"])\n",
        "df.columns = ['id']\n",
        "print(df)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "5KI8DxROggq8",
        "outputId": "81861216-e374-4aaf-8b0c-b30005665af0"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "          id\n",
            "0   11342756\n",
            "1   11342206\n",
            "2   11341105\n",
            "3   11340603\n",
            "4   11340327\n",
            "..       ...\n",
            "95  11342760\n",
            "96  11341846\n",
            "97  11340806\n",
            "98  11340127\n",
            "99  11339456\n",
            "\n",
            "[100 rows x 1 columns]\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# xml download"
      ],
      "metadata": {
        "id": "vGesQbqMQUcs"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "import requests\n",
        "import pandas as pd\n",
        "import time\n",
        "import os\n",
        "from bs4 import BeautifulSoup"
      ],
      "metadata": {
        "id": "g2z8xMVMw273"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "records = []\n",
        "# efetch api\n",
        "import xml.etree.ElementTree as ET\n",
        "for pmcid in df['id']:\n",
        "    efetch_url = f'https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id={pmcid}&rettype=xml'\n",
        "    print(efetch_url)\n",
        "    fetch_response = requests.get(efetch_url, params={'db': 'pmc', 'id': pmcid, 'rettype': 'xml'})\n",
        "    if fetch_response.status_code == 200:\n",
        "        try:\n",
        "            # parse xml to extract pmcid, doi, introduction\n",
        "            root = ET.fromstring(fetch_response.content)\n",
        "            # extract doi\n",
        "            doi = root.findtext('.//article-id[@pub-id-type=\"doi\"]')\n",
        "            if doi is None:\n",
        "                doi = 'N/A'\n",
        "            # extract introduction\n",
        "            introduction = root.find(\".//sec[@sec-type='intro']\") or root.find(\".//sec[title='introduction']\")\n",
        "            if introduction is not None:\n",
        "                intro_text = ''.join(introduction.itertext()).strip().replace('\\n', ' ')\n",
        "            else:\n",
        "                intro_text = 'N/A'\n",
        "\n",
        "            response = {\n",
        "                'pmcid': pmcid,\n",
        "                'doi': doi,\n",
        "                'introduction': intro_text\n",
        "            }\n",
        "            print(response)\n",
        "            records.append(response)\n",
        "\n",
        "        except ET.ParseError:\n",
        "             print(f'Failed to parse XML for PMCID {pmcid}')\n",
        "    else:\n",
        "        print(f'Failed to fetch data for PMCID {pmcid}. Status code: {fetch_response.status_code}')\n",
        "        continue"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "FJ_A3BpXK-TX",
        "outputId": "d7f9e439-1367-429f-9dbc-317c21d8d9f3"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11342756&rettype=xml\n",
            "{'pmcid': '11342756', 'doi': '10.1016/j.psj.2024.104099', 'introduction': 'N/A'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11342206&rettype=xml\n",
            "{'pmcid': '11342206', 'doi': '10.1016/j.bioactmat.2024.07.032', 'introduction': 'N/A'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11341105&rettype=xml\n",
            "{'pmcid': '11341105', 'doi': '10.7759/cureus.65185', 'introduction': 'Introduction       Pseudomonas aeruginosa is an opportunistic pathogen responsible for various healthcare-related infections [1]. These infections are challenging to treat due to the bacteria\\'s intrinsic and acquired resistance mechanisms [2]. The improper and unnecessary use of β-lactam antibiotics exacerbates bacterial resistance, leading to the selection of extended-spectrum β-lactamases (ESBL), AmpC β-lactamases, and metallo-β-lactamases (MBL), which pose significant challenges in antimicrobial therapy [3]. Pseudomonas aeruginosa, typically susceptible to ceftazidime, aztreonam, and carboxypenicillins, can develop resistance to third-generation cephalosporins through the hyperproduction of AmpC beta-lactamase [2]. AmpC β-lactamases hydrolyze cephalosporin antibiotics and are resistant to clavulanic acid. Unlike ESBLs, AmpC β-lactamases can deactivate a range of antimicrobials, including monobactams, aminopenicillins, cephamycins, and cephalosporins [4]. Both chromosomal and plasmid-mediated genes can produce these enzymes [5].       AmpC β-lactamases are class C enzymes according to the Ambler classification, they have serine residues that facilitate catalysis in their active site [6]. There\\xa0are three types of resistance mechanisms by AmpC β-lactamases can be distinguished: i) inducible resistance\\xa0by chromosomally encoded AmpC genes, accountable for resistance in bacteria like Serratia marcescens, Enterobacter cloacae, Citrobacter freundii, Pseudomonas aeruginosa,\\xa0etc.\\xa0ii) non-inducible chromosomal resistance caused by mutations in the promoter and attenuator\\xa0mechanisms (e.g., Acinetobacter baumannii, Shigella species, and Escherichia coli),\\xa0iii) or resistance mediated by plasmids (e.g., Salmonella species, Klebsiella pneumonia, E. coli, etc.) [7].       Even infections produced by originally susceptible isolates can develop considerable AmpC synthesis and β-lactam resistance as a result of exposure to β-lactams. Treatment considerations are complicated by species and β-lactam variations in the possibility of generating AmpC formation [8].\\xa0Three proteins, AmpG, AmpD, and AmpR, are involved in the regulation of AmpC β-lactamase induction. Mutations in the structural AmpD genes result in hyper-inducibility and overproduction of the AmpC enzyme [9]. The main resistance against β-lactam drugs in Pseudomonas aeruginosa\\xa0involves mutations boosting chromosomal AmpC production, and selecting resistant mutants. Pseudomonas aeruginosa also produces chromosomal cephalosporinases (\\u200b\\u200b\\u200b\\u200b\\u200b[Pseudomonas-derived cephalosporinases (PDC)] with extended-spectrum activity. Plasmid-mediated AmpC emergence adds to the challenge of treating Pseudomonas aeruginosa\\xa0infections [10].       Plasmid-mediated class C enzymes are named according to various criteria. Some names for them include Ambler class C (ACC) or AmpC type (ACT), which are β-lactamase types; cefoxitin (FOX), or cephamycins (CMY), which are antibiotics they resist; and Dhahran Hospital (DHA) or Miriam Hospital (MIR-1), which are the names of the hospitals where they were found. In rare cases, they are even given patient names, like Bilal (BIL-1) [11]. South Korea (CMY-1) is where the first pAmpC was found in 1989. Many other pAmpCs (FOX, CIT, MOX, DHA, EBC, and ACC) have since been characterized; the most frequent subtype is the CMY-2 enzyme (CIT-type) [12].       The study investigated beta-lactamase-mediated resistance in hospital isolates of Pseudomonas aeruginosa. \"The primary objective of this study is to investigate the prevalence and distribution of AmpC beta-lactamase-producing Pseudomonas aeruginosa in a tertiary care hospital setting in Karad, Maharashtra, India.\" \"The secondary objectives include phenotypic and genotypic characterization of AmpC beta-lactamase production and identification of the specific genes (blaPDC and blaCMY) responsible for this resistance.       The prevalence of multidrug-resistant (MDR) infections is rapidly increasing in hospital settings, largely attributed to the extensive utilization of broad-spectrum cephalosporins, which hampers the efficacy of control measures. In this regard, this study aimed to explore the production of AmpC β-lactamase in the strains of Pseudomonas aeruginosa\\xa0isolated from a tertiary care hospital in Karad, Maharashtra, India.'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11340603&rettype=xml\n",
            "{'pmcid': '11340603', 'doi': '10.1016/j.csbj.2024.07.017', 'introduction': 'N/A'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11340327&rettype=xml\n",
            "{'pmcid': '11340327', 'doi': '10.1128/jb.00049-24', 'introduction': \"INTRODUCTION       The gram-negative bacterial cell envelope confers protection from a variety of toxic agents, many of which are encountered by Escherichia coli and other enteric bacteria in their natural environments, such as solvents, acid, detergents, bile salts, and antibiotics (1). The envelope is also important for cellular responses to external stresses with alterations in membrane lipid and membrane protein composition playing critical roles in maintaining physiological homeostasis. Small RNAs (sRNAs) are well-characterized regulators that participate in maintaining cell envelope homeostasis through modulation of membrane protein synthesis (2–7). Several sRNAs are integral parts of important stress regulons, including the σE, Cpx, and OmpR responses. The transcription factor σE maintains outer membrane homeostasis by inducing the transcription of ~100 genes, comprised mainly of genes encoding proteins involved in membrane repair as well as three sRNAs, RybB, MicA, and MicL (3–5). RybB and MicA downregulate the synthesis of abundant membrane porins, including OmpA, OmpC, OmpD, and OmpF, upon entry into stationary phase (4, 5, 8, 9). MicL represses the translation of outer membrane lipoprotein Lpp, a highly abundant protein (3). Together, MicA, RybB, and MicL repress the synthesis of all of the most abundant outer membrane proteins in response to stress (3). While σE is responsible for maintaining homeostasis of the outer membrane, the Cpx response monitors inner membrane integrity (10). The Cpx response controls the production of an sRNA called CpxQ, which is processed from the 3' UTR of cpxP mRNA (2). Synthesis of CpxQ is triggered by membrane-damaging stresses, such as excessive synthesis of membrane proteins, alkaline pH, ethanol, and cationic antimicrobial peptides (2). Upon induction, CpxQ represses the production of numerous inner membrane proteins (2). Both CpxQ and the Cpx pathways are required for cell survival during conditions that lead to the dissipation of membrane potential (2). Lastly, the sRNAs OmrA and OmrB are induced by high osmolarity and regulated by the OmpR response regulator (6). Most mRNA targets are shared between OmrA and OmrB and encode outer membrane proteins. For example, OmrA and OmrB repress the translation of ompT, which encodes an outer membrane protease, along with cirA, fecA, and fepA, all encoding different outer membrane porins specific for iron–siderophore complexes (6).       Importantly, while changes in the structure of the fatty acids in phospholipids can alter critical membrane properties such as fluidity and permeability (1), the roles of sRNAs in modulating membrane lipids are just beginning to be explored. RydC is the first sRNA shown to regulate membrane fatty acid composition and plays a role in the alteration of cell membrane properties and resistance to stress (11, 12). RydC is a 65-nt sRNA that requires the RNA chaperone Hfq for stability and base-pairing interactions with target mRNAs (11, 13–15). Like other Hfq-dependent sRNAs, RydC base pairing with target mRNAs has different outcomes depending on the site of interaction. RydC’s best-characterized target is cfa mRNA, which codes for cyclopropane fatty acid (CFA) synthase, an enzyme that converts unsaturated fatty acids to cyclopropanated fatty acids (11, 12, 16). RydC base pairs at a site within the long 5' untranslated region (UTR) of cfa mRNA and modulates RNase E-dependent degradation and Rho-dependent transcription termination (11, 17). We showed previously that RydC-mediated activation of cfa promotes survival during acid stress (12). In addition to regulating fatty acid modification through activation of cfa, RydC has also been shown to impact mRNAs involved in sugar utilization (yejABEF), biofilm formation (csgD), and aromatic amino acid biosynthesis (pheA and trpE) (13, 15, 18).       Previous work identified two additional sRNAs, ArrS and CpxQ, that also directly regulate cfa (12). Like RydC, ArrS acts by reducing cfa mRNA turnover, leading to increased translation of cfa mRNA (12). In contrast, CpxQ binds cfa mRNA upstream from the site bound by activating sRNAs. We recently showed that like RydC, ArrS and CpxQ also regulate cfa through the modulation of Rho-dependent transcription termination in the long cfa 5' UTR (17). While the direct signals responsible for promoting sRNA regulation of cfa are not currently known, both CpxQ and ArrS were previously linked to cell envelope (2) and acid stress responses (19), respectively.       To better understand the physiological function of RydC and its role in cellular homeostatic or stress response pathways through cfa regulation, we sought to identify regulators that control RydC production. A candidate-based approach revealed that none of the known membrane stress-responsive transcription factors regulate rydC transcription. Instead, a transposon screen identified a GntR-like regulator, YieP, as a repressor of RydC. We showed that YieP auto-activates its own transcription and that the sRNA regulator FnrS represses yieP mRNA translation. Additionally, we demonstrate that the osmotic stress-responsive sRNA OmrB inhibits RydC activity on cfa, suggesting that OmrB acts as a sponge for RydC. Collectively, these results demonstrate that RydC is regulated directly at both transcriptional and post-transcriptional levels within a regulatory network associated with membrane stress signals.\"}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11336092&rettype=xml\n",
            "Failed to fetch data for PMCID 11336092. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11326835&rettype=xml\n",
            "{'pmcid': '11326835', 'doi': '10.1093/infdis/jiad585', 'introduction': 'N/A'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11326090&rettype=xml\n",
            "Failed to fetch data for PMCID 11326090. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11324020&rettype=xml\n",
            "{'pmcid': '11324020', 'doi': '10.1128/msphere.00060-24', 'introduction': 'INTRODUCTION       Enterobacter cloacae are saprophytic environmental bacteria and are part of the human gut microbiota (1). This Gram-negative bacterium is an important opportunistic pathogen in healthcare-associated infections (HAIs) in patients hospitalized in the intensive care unit. These infections encompass the lower respiratory tract, urinary tract, and meninges (2). E. cloacae virulence factors associated with biofilm formation and cytotoxic activity on eukaryotic cells have been described (3–6).       One of the virulence factors recently studied in E. cloacae is the type VI secretion system (T6SS). This nanomachine was discovered in 2006 as a complex macromolecular apparatus found in more than 25% of sequenced genomes of Gram-negative bacteria (7–9). At the molecular level, the T6SS is assembled via interactions between 13 canonical proteins, called Tss proteins. Several of these proteins share structural homologies with components of the T4 bacteriophage contractile tail, such as the major tail tube protein (Hcp), the cell puncturing device (VgrG), the sheath (TssB-C), and at least one component of the baseplate (TssE) (8, 10, 11). The genomic analysis of E. cloacae ATCC 13047 revealed that it possesses two T6SS clusters (2), which our group named T6SS-1 and T6SS-2 (12). The functional characterization of both systems showed that these two T6SSs are not expressed under the same environmental conditions (12). However, the regulation of the expression of these T6SSs in E. cloacae has not been studied.       The biogenesis and function of T6SS are paramount as it is energetically costly to bacterial cells, necessitating tight control over its gene expression to adapt its expression and assembly to changing environmental conditions. A plethora of environment modulators and regulatory systems have been reported in bacteria to control the transcription of T6SSs either directly or indirectly. Furthermore, beyond the transcription mechanisms, several T6SSs are post-translationally activated by a threonine phosphorylation pathway in response to cell damage or envelope stress.       Nucleoid-associated proteins (NAPs) are abundant proteins in bacterial cells involved in many important cellular processes such as genome architecture, physiology, metabolism, stress response, and virulence (13–15). One of the first NAPs to be described in detail was HU (Histone-like protein from Escherichia coli strain U93) (16). In bacteria that belong to the Enterobacteriaceae and Vibrionaceae families (13, 17), HU, the histone-like protein most abundant on the bacterial nucleoid, is a heterodimer formed of two subunits, HupA (HUα) and HupB (HUβ) (18). HU protein has three naturally occurring forms: the HUα2 and HUβ2 homodimers and the HUαβ heterodimer. The heterodimer binds DNA in a non-specific manner, contributing to DNA flexibility by bending the duplex, which is essential for the structural integrity of the chromosome (19–24) and regulation of virulence factors in several enterobacterial species (25–28). However, two genome analyses identified HU protein-binding motifs revealing A/T- and T/G-rich sequences in E. coli and Francisella tularensis, respectively (29, 30).       Currently, the role of the HU protein as a global regulator in the biology of E. cloacae is unknown. Nevertheless, in its close relative, E. coli HU regulates the expression of 353 genes that respond to anaerobiosis, acid stress, high osmolarity, and SOS response (17, 31).       In this study, we demonstrated that HU positively regulates the T6SS gene expression in E. cloacae by binding directly to their promoter regions in E. cloacae. Deleting either hupA or hupB genes results in a significantly reduced transcription of gene clusters encoding the T6SS-1 and T6SS-2. Consistently, virulence phenotypes such as inter-bacterial competition, biofilm formation, and adherence to epithelial cells were severely affected in the hupA or hupB mutants but restored when hupA or hupB genes were overexpressed in trans from plasmids. To our knowledge, this is the first study that addresses the transcriptional regulation of the T6SS gene cluster, in which HU acts as a direct activator, turning on the T6SS-associated phenotypes.'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11323537&rettype=xml\n",
            "Failed to fetch data for PMCID 11323537. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11323402&rettype=xml\n",
            "{'pmcid': '11323402', 'doi': '10.1186/s40001-024-02007-y', 'introduction': 'N/A'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11322368&rettype=xml\n",
            "{'pmcid': '11322368', 'doi': '10.3389/fmicb.2024.1437660', 'introduction': \"1       Introduction       Vibrio parahaemolyticus is one of the most important zoonotic pathogens and is mainly associated with the consumption of raw or undercooked contaminated aquatic products (Campbell et al., 2022; Yang et al., 2022). Antimicrobial agents are widely used for the prevention and treatment of bacterial infections in the fields of agriculture, aquaculture, and clinical medicine, and their main mechanisms are as follows: (I) inhibition of cell wall synthesis (such as β-lactams); (II) inhibition of protein synthesis (such as phenicols, tetracyclines, aminoglycosides, and macrolides); (III) inhibition of nucleic acid synthesis (such as quinolones); and (IV) inhibition of folate synthesis (such as sulfonamides and trimethoprim) (Zou et al., 2017). The misuse and overuse of antimicrobials have facilitated the rapid development and spread of antimicrobial resistance (Heffernan, 2022; Sagar et al., 2023). Bacterial infections caused by multidrug-resistant organisms are often responsible for increased morbidity, mortality, and medical costs. Globally, an estimated 4.95 million deaths were associated with antimicrobial resistance in 2019, of which 1.27 million deaths were directly attributable to antimicrobial resistance (Murray et al., 2022). The review on antimicrobial resistance by O'Neill (2014, 2016) reported that antimicrobial resistance could cause 10 million deaths every year by 2050.       The dissemination of antimicrobial resistance is largely attributed to the transfer of antimicrobial resistance genes (ARGs) among different bacterial hosts, including by vertical gene transfer (VGT) and horizontal gene transfer (HGT) (Li et al., 2019). HGT is the dominant mechanism for the transmission of ARGs, and consists of the following three primary pathways: conjugation, transformation, and transduction, with conjugation being the most important route (Li et al., 2021; Zhang et al., 2021). In general, these ARGs are located on mobile genetic elements (MGEs), such as plasmids, insertion sequences (ISs), transposons (Tns), and integrons (Ins) (Partridge et al., 2018; Zhang et al., 2019). Plasmids are extrachromosomal DNA molecules that can be gained and lost within and between bacterial populations, thereby playing a significant role in HGT (Ares-Arroyo et al., 2023). Additionally, it was discovered that plasmids of incompatibility groups IncF, IncI, IncA/C, IncH, IncN, and IncX were considered major carriers of ARGs and conferred a multidrug-resistance (MDR) phenotype, resulting in the widespread dispersion of many classes of ARGs, including β-lactamases (blaTEM, blaCTX-M, blaCMY, and blaOXA), quinolone resistance genes (PMQR), colistin resistance genes (mcr-1), enzyme-modifying genes of aminoglycosides (aadA, aphA1, aacC, strA, and strB), sulfonamides (sul1, sul2), and trimethoprim (dfrA), among others (Rozwandowicz et al., 2018).       At present, there are many reports on different types of resistance plasmids and their horizontal transmission in Enterobacteriaceae (Sun et al., 2019; Stosic et al., 2021; Lefèvre et al., 2023; Supa-amornkul et al., 2023) and Vibrio spp. (e.g., Vibrio cholerae and Vibrio alginolyticus) (Wang et al., 2018; Zheng et al., 2018), but there are still few studies on V. parahaemolyticus (Liu et al., 2013). In this study, we identified a conjugative plasmid recovered from a V. parahaemolyticus strain of seafood origin that carried 12 different ARGs. The objective of this study was to determine the structural features and transferability of the plasmid, and MDR phenotypes for the tested antimicrobials.\"}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11321392&rettype=xml\n",
            "Failed to fetch data for PMCID 11321392. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11319344&rettype=xml\n",
            "{'pmcid': '11319344', 'doi': '10.1038/s41598-024-69566-2', 'introduction': 'N/A'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11319304&rettype=xml\n",
            "Failed to fetch data for PMCID 11319304. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11318556&rettype=xml\n",
            "{'pmcid': '11318556', 'doi': '10.1016/j.psj.2024.104043', 'introduction': 'N/A'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11316331&rettype=xml\n",
            "{'pmcid': '11316331', 'doi': '10.1186/s12879-024-09709-y', 'introduction': 'N/A'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11315647&rettype=xml\n",
            "Failed to fetch data for PMCID 11315647. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11315073&rettype=xml\n",
            "{'pmcid': '11315073', 'doi': '10.1016/j.heliyon.2024.e34384', 'introduction': 'N/A'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11313837&rettype=xml\n",
            "{'pmcid': '11313837', 'doi': '10.3390/nu16152553', 'introduction': '1. Introduction       A mushroom is the fruiting body of a fungus, typically grown in soil or its food materials [1]. Mushrooms have been widely consumed as a food item for centuries due to their nutritional value, delicious taste, and health-promoting benefits. The global production of cultivated edible and medical mushrooms has increased more than 30-fold since 1978 [2]. Thus, studies on the health benefits of mushrooms have been gaining interest.       The mushroom cell wall contains β-glucan, a polysaccharide consisting of D-glucose monomers linked by β-1,3 glycosidic bonds as the backbone, with β-1,6 branch linkages [3]. β-glucan is not digested in the human gastrointestinal tract because the digestive enzymes secreted by the pancreas and brush-border epithelial cells cannot hydrolyze β-glycosidic bonds. Hence, β-glucan is resistant to digestion in the upper gastrointestinal tract and remains in the large intestine, where it can be utilized by the gut microbiota [4].        The gut microbiota comprise a complex community of 100 trillion microbes, with more than 1000 microbial species colonizing the human intestines. Most gut microbiota are obligate anaerobes, including Firmicutes (65%), Bacteroidetes (25%), Proteobacteria (8%), and Actinobacteria (5%) [5,6]. These gut microbes can ferment non-digestible polysaccharides because they have multiple metabolic genes encoded by the genome and produce several beneficial metabolites for the host [7]. The gut microbiota and their metabolites play important roles in various health benefits, including nutrient metabolism, glucose homeostasis, lipid metabolism, inhibition of pathogens, gut barrier integrity, regulation of immune responses, balance in mood disorders, and alleviation of neuroinflammation [8,9]. One class of metabolites produced by the gut microbiota are the SCFAs, which are volatile fatty acids, among which acetic acid, propionic acid, and butyric acid are the most abundant. Several studies have demonstrated that SCFAs are one of the most important microbe metabolites and are especially related to gut health. For instance, acetic acid plays an important role in inhibiting pathogenic bacteria. Propionic acid is involved in gut hormone stimulation. Furthermore, butyric acid increases mucin production, enhances epithelial barrier function, and is the main energy source of colonocytes [9,10].       The composition and abundance of colonic bacteria are affected by several factors, including genetics, the mode of delivery, medication, lifestyle, and diet patterns [11]. Therefore, one’s diet, especially dietary fiber, impacts the gut microbial community and gut homeostasis. β-glucan from mushrooms is one type of dietary fiber and is well-known for its prebiotic potential that shapes the gut microbial composition and contributes to health benefits, particularly in gut health [12]. Overall, previous findings have revealed that mushroom β-glucan-supplementation in a colitis mouse group increased the abundance of beneficial bacteria and inhibited the growth of pathogenic bacterial groups. Taken together, the production of SCFAs, including acetic acid, propionic acid, and butyric acid, was promoted, resulting in a reduction in intestinal inflammation-related disorders [13,14,15]. These studies have demonstrated that β-glucan from typically consumed mushrooms could alleviate gut inflammation via the modulation of intestinal microbes. Thus, further explorations of new edible mushrooms and their stages could expand knowledge about their positive effects on gastrointestinal health.       Phallus atrovolvatus (Syn. Dictyophora duplicata (Bosc.) E. Fisch.) is found in the southern forests of Thailand and has been recently identified as a new Thai strain. This mushroom has been histologically consumed by locals for decades without reports on any toxicity or discomfort after consumption. Although Phallus spp. are common edible wild mushrooms and regularly used in many cuisines [16,17], it is worth noting that not all wild mushrooms are edible; some of them can cause life-threatening issues. Nowadays, P. atrovolvatus can be commercially cultivated through the support of the Department of Agriculture, the Ministry of Agriculture and Cooperatives, Thailand. P. atrovolvatus is scientifically close to P. indusiatus, mostly found and widely consumed in China. These two mushrooms are commonly called ‘bamboo mushroom’. The Chinese strain of the bamboo mushroom P. indusiatus is regarded as the queen of mushrooms due to its nutritional value, taste, and unique appearance, and it also exhibits several health-promoting benefits, including antioxidative, anti-cancer, anti-inflammatory, neuroprotective, immunomodulatory, and gut microbiota-modulating effects [18].       The biological properties of the Thai strain of bamboo mushroom, P. atrovolvatus, have recently been reported. Crude polysaccharide, extracted from the fruiting bodies of P. atrovolvatus using hot water extraction, showed strong antioxidative activities in vitro and alleviated gut inflammation via the inhibition of myeloperoxidase activity and pro-inflammatory cytokines in dinitrobenzene sulfonic acid-induced colitis mice [19,20]. In Thailand, P. atrovolvatus is commonly consumed as the mature fruiting body; however, consumption at the egg stage has attracted interest due to its unique and specific texture. The growth of the bamboo mushroom’s fruiting body is divided into four maturity stages, starting from primordia and developing into ball-shaped, peach-shaped, and mature stages [21]. The immature stage, which is peach-shaped, is also called the egg stage. Different maturity stages of mushrooms can affect their taste, flavor, chemical compounds, polysaccharide contents, and biological properties [22]. A previous study reported that the total phenolic and flavonoid contents of Ganoderma lucidum stipes were highest in the spore maturity stage. In contrast, the fruiting body maturity stage contained the highest content of ganoderic acid, one of the triterpenoids found in Ganoderma spp. mushrooms [23]. The chemical components and anti-tumor activity of Pleurotus eryngii fruiting bodies increased with development to the mature stages [24].       Although the impacts of mushroom polysaccharides—in terms of molecular weight, polysaccharide composition, and structure—on gut microbiota modulation have been established in a review by Zhao et al. [25], the effects of the different stages and the link to the chemical compositions and consumption forms of mushroom have not yet been elucidated. Therefore, the current study aimed (1) to determine the chemical composition of the P. atrovolvatus fruiting body in both the egg and mature stages, which are commercially cultivated and consumed; and (2) to investigate gut microbiota-modulating activity, comparing both maturity stages (egg and mature fruiting body) and mushroom consumption forms (cooked mushroom and aqueous extract) to expand the current knowledge and promote the consumption of this unique and edible mushroom.'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11313783&rettype=xml\n",
            "{'pmcid': '11313783', 'doi': '10.3390/plants13152128', 'introduction': '1. Introduction       Leymus chinensis is a perennial rhizomatous grass widely distributed in northern China, Mongolia, the Korean peninsula, and Siberia (GBIF), with high leaf volume, rich in nutrients, and good palatability, and it is the main high-quality forage grass in northeastern China [1,2,3]. However, it is susceptible to infection by pathogens, which affects its economic and ecological values [1,4]. Grassland plant diseases, such as leaf spot, are the most diverse and widely distributed disease groups. Leptosphaeria avenaria and Parastagonospora nodorum could lead to the leaf spot disease of L. chinensis [1,5]. Thus, the plant biomass, nutrients, as well as growth rate decrease obviously, affecting its yield and quality [4]. Pathogens parasitizing perennial plants also have long-term cumulative impacts on grassland communities, even endangering other plants around the host and changing the direction of vegetation succession in the whole community [6,7,8].       Grazing is the main utilization method of natural grasslands; animal feeding, trampling, and feces may either reduce the incidence of disease or increase the chances of disease transmission and infestation. Animal feeding has the ability to eliminate pathogens by removing pathogen-infected plant tissues, reducing the primary source of infection, and mitigating disease incidence. Additionally, grazing increases the infection of certain pathogens and exacerbates the spread of disease by creating new wounds on plant surfaces [9,10,11]. The movement of herbivores and mechanical damage caused by trampling can also accelerate the spread of some pathogens across grasslands [11]. In addition, animal feces can increase soil nitrogen and phosphorus concentrations, significantly alter soil chemistry, and increase the risk of plants infecting themselves with fungal pathogens [12]. Damage caused by grazing can also cause plant tissues to activate jasmonic acid-dependent defense responses more quickly and efficiently, therefore reducing pathogen invasion [13,14]. At the same time, grazing can also indirectly affect the occurrence of plant diseases by altering the plant community. Studies have shown that due to the selective feeding on plants by herbivores can lead to changes in species composition, canopy characteristics, and the phylogenetic structure of plant communities [15,16]. Microenvironmental factors, such as humidity, temperature, and wind speed, closely linked to disease incidence and pathogen spread are also influenced by grazing [17]. Thus far, many studies have proved that moderate grazing effectively controls certain diseases in natural grasslands by reducing the occurrence of diseases, such as leaf spot diseases, through direct or indirect factors [11,18].       It is well known that microbial communities are closely related to plant health, and beneficial micro-organisms can protect plants from pathogens by antagonizing, competing with pathogens, or establishing a reciprocal, symbiotic relationship with the host plant [19,20]. Whereas, some micro-organisms are potential phytopathogens, and growth and reproduction under suitable environmental conditions can jeopardize plant health [21,22]. Several studies have focused on the relationship among soil, inter-root microbial communities, and plant health, and reported that plant diseases strongly influence microbial community structure, and in turn, microbes influence disease development through complex interactions as well. Jiao et al. (2022) discussed that the diversity of microbial communities in soil samples of healthy Fritillaria ussuriensis was higher than that in soil samples of F. ussuriensis infected with Fusarium sp. [23]. The probable reason could be that the root secretions of healthy plants can provide more nutrients to soil micro-organisms, thus increasing the species richness and diversity of the microbial community. Fernandez-Gonzalez et al. (2020) reported that susceptibility/tolerance to soil-borne pathogens in olive was correlated with changes in the inter-root microbial network [24]. Like soil micro-organisms, phyllospheric micro-organism have similar effects on plant health. The plant leaf provides a vast living space and nutrients for micro-organisms, which can inhibit the growth of plant pathogens through mechanisms, such as nutrient competition and antagonism [25,26,27]. It can also affect host plants through the production of volatile organic compounds (VOCs), produced by the Poaceae endophytic fungi, Epichloë sp. It has fungitoxic effects at ecologically significant concentrations, where Chokol K inhibits spore germination of plant pathogenic fungi [28]. Thus, plants can maintain health by regulating the phyllospheric micro-organism composition and diversity, as well as preventing flora dysbiosis. Relevant studies have mainly focused on crops and model plants, with greenhouse pot experiments, and little has been reported in complex natural grassland ecosystems [29,30,31,32,33].       Few other studies suggest that animal trampling, selective feeding, and defecation alter the abundance, composition, and diversity of soil micro-organisms by affecting soil nutrient cycling and soil structure, which are dependent on the intensity and duration of grazing [34,35]. Jing et al. (2023) found that yak grazing altered the structure of soil microbial communities [36], however, the network complexity of bacteria and fungi responded differently to grazing. Eldridge and Delgado-Baquerizo (2018) showed that sheep grazing had direct and indirect negative effects on the abundance of plant pathogens [37]. Existing studies of grazing-affected microbial community changes and grazing-affected plant diseases have been conducted independently, with few studies linking the two parts. Consequently, there is a paucity of research on the effects of grazing on microbial communities and plant health, focused mainly on soil micro-organisms. However, the destruction of plant above-ground tissues by feeding and trampling by grazing livestock creates favorable conditions for the colonization of phyllospheric micro-organism. Simultaneously, it may also disrupt the interactions between micro-organisms that have already been effectively colonized, compromising the normal growth and development of the plant. Excreta from domestic animals could also inevitably affect plant phyllospheric micro-organisms through direct contact with plant above-ground tissues or by altering the physiological and biochemical properties of the soil, thereby affecting the normal growth of the plant.       Theoretically, the pathogens that cause leaf spot disease in L. chinensis mainly infect the wounds on plant leaves caused by livestock feeding and trampling. Livestock causes wounds more conducive to the horizontal transmission of micro-organisms, and livestock feeding may reduce the load of pathogens and the source of primary infestation in the field. The saliva and excreta of livestock may change the phyllospheric micro-organism community, and some of them may also act as antagonists to the pathogen, parasitism, and other complex relationships. In summary, the microecological balance composed of grazing livestock, pasture, and phyllospheric micro-organism is important for the healthy growth of L. chinensis in natural grasslands.       Exploring the interactions and synergistic development of livestock-pasture diseases and phyllospheric micro-organism under grazing conditions could be a meaningful study. Therefore, this study investigated the effects of grazing intensity on the phyllospheric micro-organism community and took a typical leaf spot disease in L. chinensis, as the research object. The study was aimed at elucidating the interactions between livestock-quality pasture leaf disease and phyllospheric micro-organism in natural grassland ecosystems. The current research could provide a theoretical basis for the sustainable utilization of natural grassland and the biological prevention and control of high-quality pasture diseases.'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11312530&rettype=xml\n",
            "{'pmcid': '11312530', 'doi': '10.1101/2024.07.30.605899', 'introduction': 'N/A'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11312258&rettype=xml\n",
            "{'pmcid': '11312258', 'doi': '10.3390/foods13152343', 'introduction': '1. Introduction       Cholesterol is an important factor in the maintenance of cellular homeostasis, and it is synthesized in the liver (about 700–900 mg/day) and ingested through diet (about 300–500 mg/day) [1]. An excessive consumption of foods rich in saturated and trans fats will increase cholesterol levels and lead to hypercholesterolemia. Hypercholesterolemia is characterized by high levels of total cholesterol (TC), low-density lipoprotein cholesterol (LDL-C), or triglycerides (TG), as well as a decrease in high-density lipoprotein cholesterol (HDL-C) [2]. Hypercholesterolemia often leads to atherosclerosis, coronary artery disease, and a range of cardiovascular and cerebrovascular diseases. Statins are commonly used as cholesterol-lowering drugs, but they may cause adverse reactions, such as liver and kidney damage, rhabdomyolysis, elevated blood sugar-induced diabetes, etc., and they are not considered to be an ideal long-term treatment [3]. Therefore, it is important to find new effective drugs and new treatment strategies for preventing or alleviating hypercholesterolemia.       Systemic cholesterol homeostasis is a strictly regulated process involving de novo biosynthesis, dietary cholesterol absorption, biliary clearance, and excretion, and the liver is the main organ to maintain cholesterol homeostasis [4]. The metabolism of cholesterol in the body mainly through the following ways: 1. Cholesterol is transformed into cholesterol derivatives with important physiological functions, such as Vitamin D, sex hormones, and adrenocortical hormones; 2. Cell-membrane formation; 3. Esterification; 4. Excretion in feces; 5. Cholesterol is hydroxylated and degraded to produce bile acids (BAs) [5]. BAs are steroid substances synthesized by the liver from cholesterol. Various bile salts (BS) formed by BAs are the main components of bile, and the conversion of cholesterol into BAs accounts for 80% of total metabolism. There are two main pathways for BA synthesis in hepatocytes: The classical pathway with cholesterol 7α hydroxylase (CYP7A1), and the alternative pathway with sterol 27 hydroxylase (CYP27A1) [6]. When cholesterol levels are high, cholesterol is broken down into BAs and stored in the gallbladder. They are then released into the duodenum and proximal jejunum after cholecystokinin stimulation. After entering the intestine, a part of the BAs is absorbed by the mucosa and returned to the liver to complete the hepatoenteric circulation. In contrast, the unabsorbed parts are degraded into fecal matter by intestinal microorganisms and expelled [7]. Promoting the conversion of liver cholesterol to BAs is an important method to reduce cholesterol, which is of great significance for the prevention and treatment of hypercholesterolemia.       Intestinal flora is related to digestion, nutrient metabolism, immune function, etc., and is important in body health and physiological metabolism. Diet is one of the most critical factors in the change of gut microbiota structure, and the functional components in food can regulate the growth and metabolic activities of gut microbiota, thus affecting the microbial composition. Under normal circumstances, the gut flora and the host benefit from a mutual dynamic ecological balance. Once this balance is disrupted, pathogenic bacteria can cause infectious diseases, metabolic syndromes (such as hyperlipidemia and high blood pressure), and certain neurological disorders [8,9]. Studies have shown that a high-fat diet disrupts the body’s original microbial balance and induces more conditioned pathogens, such as Betaproteobacteria, Clostridium_bolteae, Desulfovibrio, and Enterobacter_cloacae [10,11,12]. In addition, intestinal flora participates in and regulates host metabolism by producing metabolites and short-chain fatty acids (SCFAs) to alleviate hypercholesterolemia [13]. SCFAs can stimulate the proliferation and differentiation of intestinal epithelial cells and help maintain the mineral balance of intestinal epithelial cells and the absorption of iron, calcium, and magnesium. Common SCFAs include acetic acid, propionic acid, and butyric acid, while other SCFAs such as isobutyric acid, valeric acid, and isovaleric acid are relatively low in the colon [14].       Polysaccharides are essential biomacromolecules in living organisms, playing important roles in life activities such as cell–cell connections, signaling pathways, cell adhesion, and immune system molecular recognition. In recent years, more and more studies have reported the isolation and identification of bioactive polysaccharides from natural sources, indicating that polysaccharides have a variety of biological functions, including anti-hypercholesterolemic activity [15,16,17]. Cyclocarya paliurus is a species native only to China, and its leaves are used in medicinal preparations in traditional Chinese medicine, as well as an ingredient in functional foods [18,19]. Previous studies have shown that polysaccharides from Cyclocarya paliurus leaves can regulate lipid metabolism, relieve hyperlipidemia, and relieve diabetes, but its effect on hypercholesterolemia has not been reported [18,19,20]. Therefore, this study attempted to explore the effect of Cyclocarya paliurus polysaccharides on hypercholesterolemia through in vivo and in vitro experiments to explore the mechanism of action.'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11312129&rettype=xml\n",
            "{'pmcid': '11312129', 'doi': '10.3390/ijms25158347', 'introduction': '1. Introduction       Antibiotic-resistant Enterobacterales, especially E. coli and Salmonella spp., are considered major enteric pathogens, posing a threat to global public health. They can occupy multiple niches, including human and animal hosts [1,2]. The United States Center for Disease Control and Prevention (CDC) has designated foodborne drug-resistant Salmonella spp. as a significant public health hazard.       Infective multidrug-resistant (MDR) Enterobacterales in the poultry industry have been linked to increased morbidity and mortality. The transmission of these pathogens via food may be responsible for causing foodborne illnesses and deaths.       Plasmids serve as reservoirs for multiple antibiotic-resistance genes among closely related bacterial species. Conjugative incompatibility F (IncF) plasmids have received much attention among 27 incompatibility (Inc) groups previously recognized in Enterobacterales alone. Plasmid incompatibility refers to multiple plasmids within one cell having the same replicon or partitioning system [3]. IncF plasmids were sub-grouped into replicons FII, FIA and FIB, which are the most frequently encountered among Enterobacterales, especially E. coli. They represent 23.1% of plasmids [4] that have been deposited in the plasmid multilocus sequence typing (pMLST) database (http://pubmlst.org/plasmid, accessed on 20 December 2023). Conjugative plasmids (CPs) transmitting antibiotic resistance genes are the major direct cause of outbreaks of antibiotic-resistant Enterobacterales. Actually, Gram-negative bacilli have a remarkable capacity to transfer conjugative antibiotic resistance genes [5]. Transfer of a single plasmid to a bacterium allows it to be resistant to multiple antibiotics. Therefore, plasmids harboring antimicrobial resistance genes may result in plasmid endemics [6,7]. Despite the relatively low abundance of conjugative plasmids, many resistance genes of sulfonamide (sulI and sulII), chloramphenicol (cat), trimethoprim (dfr), colistin (mcr), New Delhi metallo-beta-lactamase (blaNDM) and extended-spectrum β-lactamase (ESBL; blaTEM, blaCTX-M and blaSHV) were significantly located in these plasmids [8,9,10].       The widespread nature of antimicrobial resistance (AMR) genes mediates plasmid epidemics among different members of Enterobacterales [11]. The bacterial-resistant phenotypes observed during epidemics can persist for months even in the absence of ongoing specific selection with risk of prolonged carriage [12]. Plasmid epidemics arise from the massive exchange of AMR genes (more than 86%) not only across other distant plasmid categories but also between bacterial chromosomes [13]. The mobility of CPs is a strong factor facilitating the accumulation of AMR genes as these plasmids possess the ability to pass across phylogenetically distant bacteria [14,15]. Transfer and stabilization of mutations in insertion sequence (IS)-associated AMR genes are affected by the use of antibiotics, which also promote bacterial conjugation [16,17,18], support bacterial fitness and persistence in adverse environmental conditions, providing virulence (bacteriocins, siderophores, cytotoxins, and adhesion factors) and antimicrobial resistance determinants [19]. Antibiotic selective pressure on commensals and pathogenic AMR Salmonella spp. and E. coli leads to clonal expansion of these strains [20]. Using oral antibiotics during the broiler production period for therapeutic, prophylactic, or subtherapeutic growth promotion purposes modifies gut microbiota [21]. Lateral transfer of CPs to recipient cells confers genes encoding antimicrobial resistance, exerts strong selective pressure, and facilitates the spread of genetic elements of insertion sequences (IS26 and IS1294), integrons, and transposons [22,23].       In Egypt, IncF grouping studies have focused only on the association of resistance to an antibiotic with a specific replicon using whole-genome sequencing. However, there is a need to investigate the molecular subtyping of conjugative IncF plasmids [24]. In the present work, PCR-based replicon grouping was applied using conserved IncF primers following the GenBank entries, and the replicon type was subjected to allele subtyping [25].'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11311926&rettype=xml\n",
            "{'pmcid': '11311926', 'doi': '10.1186/s13099-024-00633-9', 'introduction': 'N/A'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11311727&rettype=xml\n",
            "{'pmcid': '11311727', 'doi': '10.3390/diagnostics14151598', 'introduction': '1. Introduction       In the realm of molecular biology and diagnostics, the emergence of digital polymerase chain reaction (dPCR) has sparked a revolution in the way we perceive and manipulate nucleic acids. This novel technique offers unparalleled precision and sensitivity, enabling researchers and clinicians to detect and quantify nucleic acids with unprecedented accuracy. As we delve into the intricate world of dPCR, it becomes evident that its applications extend far beyond traditional PCR methodologies. dPCR represents a technological leap from conventional polymerase chain reaction methods, offering an accurate absolute quantification of target nucleic acid molecules present in a sample [1]. This advanced technique partitions the sample into numerous compartments within which nucleic acids undergo stochastic partitioning followed by clonal amplification [2]. Concentrations are determined by analyzing the proportion of nonfluorescent particles according to the Poisson distribution. Recent advancements have led to the development of several commercial PCR droplet platforms, aimed at expediting the clinical adoption of digital PCR.       This review aims to provide an overview of dPCR, focusing on its applications in infectious disease diagnostics and environmental surveillance. We will delve into the fundamental principles underlying dPCR, highlighting its unique ability to partition nucleic acids into individual reactions, thereby facilitating absolute quantification. Beyond its technical intricacies, the true essence of dPCR lies in its transformative potential across various fields. From precision medicine to environmental monitoring, dPCR offers a versatile toolkit for addressing complex biological questions and tackling real-world challenges.       As we navigate through the potential applications, it is imperative to address the challenges, the limitations, and the future perspective in dPCR technology. From optimizing experimental conditions to standardizing data analysis workflows, there are several obstacles that must be overcome to fully realize the potential of dPCR in clinical and research settings. By critically evaluating these challenges, we can chart a course toward harnessing the full power of dPCR for advancing science and improving healthcare outcomes.'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11311408&rettype=xml\n",
            "Failed to fetch data for PMCID 11311408. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11311048&rettype=xml\n",
            "Failed to fetch data for PMCID 11311048. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11310248&rettype=xml\n",
            "Failed to fetch data for PMCID 11310248. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11304742&rettype=xml\n",
            "Failed to fetch data for PMCID 11304742. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11304630&rettype=xml\n",
            "{'pmcid': '11304630', 'doi': '10.1186/s12934-024-02484-0', 'introduction': 'N/A'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11304591&rettype=xml\n",
            "{'pmcid': '11304591', 'doi': '10.1093/g3journal/jkae119', 'introduction': \"Introduction       Underpinning every aspect of an organism's biology is its evolutionary history. Phylogenetic methods strive to reconstruct these histories using genetic data. With the advent of “Next-” and “Third-gen” sequencing methodologies, the amount of genetic data in public databases has increased dramatically. At writing, there are currently 2,018,201 bacterial genome assemblies available through NCBI alone (ncbi.nlm.nih.gov/assembly/?term = bacteria). The wealth of available genetic data has transformed the fields of bacterial phylogenetics and taxonomy (Konstantinidis and Tiedje 2007; Varghese et al. 2015; Jain et al. 2018). Unfortunately, most whole genome phylogenetic analysis methods require specialized bioinformatic data processing and expertise (Smith 2013; Lozano-Fernandez 2022). There are few very large-scale trees to place these sequences into an evolutionary framework. The available large-scale trees are under-resolved and built by reconciling many disparate phylogenetic studies, such as NCBI taxonomy (Schoch et al. 2020) or are generated with methods unsuited for narrow and broad evolutionary distances (Hördt et al. 2020) reducing their utility at multiple evolutionary scales. This flood of genomic data necessitates an easy-to-use phylogenetic analysis pipeline to help reveal the evolutionary context for the myriad of sequenced bacteria.                Analysis options                    Single and multi-locus           A classic way to generate bacterial trees is to compare single homologous loci (e.g. 16S ribosomal, recA, gyrA, rpoB, or dnaK genes). While largely replaced by other methods, single gene trees have some advantages, which include the ability to use conserved primer sites to amplify sequences, very low computational intensity, and the availability of curated databases (SILVA, RDP, NCBI GenBank, etc.). Using genes other than rDNA allows for greater resolution at certain evolutionary distances but still lacks broad resolution (i.e. Near resolution for swiftly evolving genes, far for conserved) (Yang 1998). However, these trees are prone to misleading topologies due to horizontal gene transfers, incomplete lineage sorting, paralogous gene conversions, and limited phylogenetic information (Huerta-Cepas et al. 2007). Using multi-locus alignments to build phylogenetic trees increases the total amount of phylogenetic information (more sequence used), reduces the effects of incongruent gene tree topologies due to recombination or horizontal gene transfer, and widens the breadth of evolutionary resolution (Gontcharov et al. 2004). However, in multi-locus analyses of taxa without well-vetted multi-locus methods, locus selection is non-trivial and the effort and cost of primer design and amplification optimization scales with the number of sequences used, which can rapidly become prohibitive for many labs. Additionally, strains of many taxa, like Brucella, are not differentiated well by available multi-locus methods, leading to unknown evolutionary histories (Sankarasubramanian et al. 2019).                             Whole genome           Classically, whole genome alignments were required to identify Single Nucleotide Polymorphisms (SNPs) of assemblies (see Shakya et al. 2020; Treangen et al. 2014 for method details). These methods add considerable phylogenetic data for tree building and can produce different topologies than multi-locus alignments, illustrating their potential utility (Baltrus et al. 2014). A major drawback of whole genome alignments is that they tend to produce short alignments or fail outright, for more distantly related genomes (Darling et al. 2004; Angiuoli and Salzberg 2011; Chung et al. 2018; Shakya et al. 2020). Thus, phylogenetic information drops quickly as a function of evolutionary distance. The practical result is a lack of resolution at even moderate evolutionary distances. For alignments to a reference, having the “missing” data enriched for the more divergent sites will possibly lead to an underestimation of the sequence to reference evolutionary distances for more divergent sequences (Spencer et al. 2007; Shavit Grievink et al. 2013; Bertels et al. 2014). Alternatively, one can use all-vs-all whole genome alignments, however, they scale poorly and are limited in the number of input taxa that can be aligned (Angiuoli and Salzberg 2011).                             K-mer overlap           An emergent method to quickly estimate evolutionary histories from genome assemblies is to use k-mer content. This involves identifying subsequences of length k within each assembly and calculating the shared k-mer content (for details see Bussi et al. 2021). The advantage of this class of methods is that it is extremely fast relative to alignment-based methods. The comparison of k-mer overlap between sequences naturally leads to pairwise distance matrices, thus neighbor-joining trees can be generated directly from the results (Saitou and Nei 1987). Recent work has integrated more complicated distance statistics with success (Tang et al. 2023). Crucially, k-mer distances become non-linearly associated with true percent identity, leading to erroneous estimated branch lengths (Jain et al. 2018).           An alternative k-mer-based approach is kSNP (Gardner and Hall 2013; Gardner et al. 2015; Hall and Nisbet 2023), which identifies SNPs by flanking conserved k-mers. This approach allows the use of tree-building methods including parsimony, maximum likelihood, and Bayesian. Like whole genome alignments and k-mer overlap methods, kSNP suffers from a rapid decline in phylogenetic information as sequences diverge. Additionally, the signal/noise ratio degrades quickly because non-homologous k-mers are increasingly common as sequences diverge (Gardner and Hall 2013).                             Predicted gene alignments           To use the wealth of data generated by NGS methods and simultaneously simplify the problem of whole genome alignments, CDS or protein sequences derived from annotations or transcriptome sequencing can be compared. This method breaks the problem of identifying homologous loci into two tractable parts: gene identification, then identification of homologous sequences. This approach identifies many phylogenetically informative sites (like whole genome), but alignments are more tractable (like single loci). However, gene-based alignments are negatively affected by inferred paralogs, i.e. duplicated genes or contamination. Thus, most methods filter out gene families with paralogs. However, see ASTRAL-Pro for a method that is “paralog aware” (Zhang et al. 2020). A widely used pipeline that accomplishes this task is GToTree, which annotates assemblies, uses precomputed hidden Markov models to identify single copy orthologs, aligns predicted amino acid sequences, then generates trees with FastTree2 or IQTree2 from concatenated alignments (Lee 2019). While focused on protein alignments, GToTree can also be run in nucleotide mode to use CDS alignments for phylogenetic inference instead.           A trade-off arises while using predicted coding or protein sequences to build species trees. Coding sequences (CDS) have the most phylogenetic information due to codon degeneracy but are hard to align at high divergencies (States et al. 1991). Proteins remain alignable at high divergences but lack the information content of CDS alignments. This can be addressed by converting protein alignments to corresponding codon alignments, leveraging nucleotide phylogenetic information and protein alignment accuracy (Wernersson and Pedersen 2003; Bininda-Emonds 2005). However, at increasing evolutionary distances, even with accurate alignments, mutational saturation in nucleotide data can negatively affect phylogenetic estimates. Evolutionary model selection, avoiding compositional bias, and dense taxon sampling can largely alleviate these issues, see (Kapli et al. 2023) for details.                                Benefits of an automated workflow         The skills required to generate bacterial phylogenetic trees from whole genomes are extensive. Some of the many steps include gathering and annotating assemblies, identifying and filtering orthologs, sequence alignment, trimming and concatenation, and finally, tree inference (Ashford et al. 2020; Lozano-Fernandez 2022). Each one of these steps, especially for large data sets, requires expertise in picking parameters, file management, and data format manipulation and filtering. Many of these steps also require familiarity with UNIX command line.         Beyond taxonomic studies, the phylogenetic context of organisms is becoming increasingly important for standard molecular and evolutionary studies. Even with the available data and a clear need for the phylogenetic placement of focal study species, technical barriers preclude many researchers from inferring their evolutionary histories. An easy-to-use, accurate phylogenomic pipeline will encourage wide adoption of phylogenomic methods to complement ongoing environmental, evolutionary, and clinical microbiological research, and perhaps help standardize the estimation of phylogenetic trees within and between research labs.         To meet this demand, we present OrthoPhyl, a phylogenomic pipeline that takes bacterial genomes as input, annotates them, identifies orthologs, converts protein to nucleotide alignments, and builds species trees with both concatenated alignments and gene tree to species tree reconciliation. The workflow accepts an arbitrarily large number of input genomes (tested up to 1,200 here). To accelerate analysis, a subset of samples representing the whole dataset's diversity are identified, their proteomes are used to identify orthologs and build hidden Markov models, which are then expanded to the full input dataset by iterative searches. This strategy allows the generation of trees for 689 Brucella assemblies in ∼48\\u2005hrs using 30 cpus and 58.3GB memory with no hands-on time required. This pipeline is designed to be an easy-to-install, scalable, turn-key solution for generating high-resolution bacterial trees from diverse clades.\"}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11303641&rettype=xml\n",
            "{'pmcid': '11303641', 'doi': '10.1186/s11671-024-04075-9', 'introduction': 'N/A'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11302663&rettype=xml\n",
            "{'pmcid': '11302663', 'doi': '10.1128/spectrum.00052-24', 'introduction': 'INTRODUCTION       Retinal vein occlusion (RVO) is a prevalent blinding disorder within the realm of ophthalmology, ranking as the second most common retinal vascular disease globally (1, 2). Annually, approximately 16.4 million individuals develop RVO (3). The etiology of RVO is multifaceted, with specific pathogenic mechanisms and causative factors yet to be fully elucidated. Associated risk factors for RVO encompass advancing age, diabetes mellitus, hypertension, arteriosclerosis, cardiovascular diseases (characterized by augmented blood viscosity), obesity (accompanied by hyperlipidemia), hematological disorders (leukemia, lymphoma, and polycythemia), dysproteinemias, systemic vasculitis or autoimmune diseases (causing endothelial injury), malignancies, tobacco use, and anemia, among others (4–8). RVO primarily manifests through either partial or complete occlusion of the retinal veins, attributed to these diverse factors. This blockage disrupts venous blood return, leading to venous dilation and subsequent retinal hemorrhage. These initial pathological changes frequently progress to macular edema, which significantly deteriorates visual acuity and can lead to blindness in severe cases (9). Although the incidence of RVO is slightly lower than that of the most prevalent diabetic retinopathy (DR), its prevalence increases gradually with progressive aging, imposing significant economic burdens on both seniors and society (10). Consequently, investigating the pathogenesis underlying RVO holds vital societal and economic values.       Our bodies harbor many ever-evolving microbial communities, known as microbiotas, whose intricate configuration and functionality play crucial roles in our well-being and the onset of diseases (11). The harmonious coexistence of microbial commensals exerts an extensive influence over the intricate web of immune development and homeostasis, shaping the delicate equilibrium between well-being and illness. A growing body of evidence supports the hypothesis that these gut-dwelling allies not only play a role in intestinal disorders but also extend their impact to remote tissues. Emerging scientific evidence has uncovered an intriguing phenomenon termed the “gut–eye axis,” demonstrating how the dynamic presence of gut microbiota influences ocular health, including the retinal structure (12–14).       Various retinal disorders, such as age-related macular degeneration, DR, or retinitis pigmentosa, may fall under the influence of the gut microbiota, hinting at the existence of a direct gut–retina axis (15, 16). However, current research does not directly link RVO onset with gut microbiota imbalance; in other words, whether RVO is related to the gut–retinal axis is still unknown. Systemic diseases that alter blood flow, blood viscosity, coagulation, or vascular structure are known to lead to RVO (17–21), yet the specific pathogenesis remains poorly understood. Therefore, we speculate whether gut dysbiosis could prompt vascular alterations and affect vascular health (i.e., changes in blood viscosity and abnormal blood coagulation), thereby initiating the onset of RVO.       Gut microbes profoundly influence the physiology of their hosts, encompassing vital aspects such as metabolism, immune system homeostasis, nervous system regulation, and resistance against pathogen colonization (22, 23). With the vigorous advancement of multiomics technologies, there has been a shift from meticulous reductionist investigations toward holistic systemic research. These technologies allow for the cross-validation of findings, enhancing the robustness and applicability of our conclusions about gene functionality. However, genomic analysis of gut microbiomes merely deciphers microbial composition and potential functionalities, without delving into their actual activities at a viable state. In contrast, metabolomics represents a discipline that explores the variations in metabolites of a community of organisms resulting from external perturbations and unveils patterns of temporal changes—a field more closely aligned with phenotypic studies (24). Thus, our study endeavors to unravel potential mechanisms underlying the onset of RVO by integrating various omics technologies, including genomics and metabolomics, offering significant implications for guiding clinical therapeutic development.'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11302304&rettype=xml\n",
            "{'pmcid': '11302304', 'doi': '10.1128/spectrum.00759-24', 'introduction': 'INTRODUCTION       Recent meta-analyses of semen parameters from 1973 to 2019 indicate a significant global decline in sperm concentration (SC) and total sperm count (TSC), with the rate of reduction accelerating (1). Inferior sperm quality has been strongly associated with adverse reproductive outcomes (2). Nonetheless, the predictive capacity of routine semen analysis for assessing male fecundity is constrained (3). The DNA fragmentation index (DFI), a measure of sperm DNA fragmentation, has gained clinical relevance (4). Factors, such as reactive oxygen species (ROS) (5) and inflammatory agents (6), have been identified as contributors to sperm DNA damage. High levels of sperm DNA fragmentation are associated with reduced fertilization rates, poor embryo quality, lower pregnancy success, and increased miscarriage rates (7).       Current treatments for high DNA fragmentation index (HDFI) include antioxidant therapy, infection management, and lifestyle changes (8), but these approaches often prove inadequate, especially in cases of idiopathic HDFI (9). This has led to an increased reliance on assisted reproductive technologies (ARTs) (5), yet the success rates vary, underscoring the need for alternative treatment strategies.       The human microbiome, encompassing the collective genetic material of microbiota in various body niches, plays a vital role in physiological homeostasis (10). Imbalances in the microbiome can lead to various pathologies (11). Recent research has begun to explore the connection between the semen microbiome and semen quality (12). Notably, a study by Sergio et al. using 16S rRNA gene sequencing revealed a negative correlation between certain bacteria and sperm DNA fragmentation in a Western Mediterranean cohort (13). Thus, our study aims to further investigate the role of the semen microbiome in HDFI using both 16S rRNA and metabolome sequencing. We hypothesize that the seminal microbiota could significantly influence sperm DNA integrity, potentially offering new avenues for HDFI treatment.'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11300865&rettype=xml\n",
            "{'pmcid': '11300865', 'doi': '10.1038/s41598-024-69162-4', 'introduction': 'N/A'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11300684&rettype=xml\n",
            "Failed to fetch data for PMCID 11300684. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11300682&rettype=xml\n",
            "Failed to fetch data for PMCID 11300682. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11300174&rettype=xml\n",
            "Failed to fetch data for PMCID 11300174. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11237486&rettype=xml\n",
            "{'pmcid': '11237486', 'doi': '10.1128/spectrum.04278-23', 'introduction': 'INTRODUCTION       Bacteria, which function as social organisms, engage in continual interactions with adjacent cells to outcompete their nonkin counterparts for vital nutrients and space (1). In this microbial competition, they have evolved an array of strategic mechanisms aimed at surpassing neighboring species, including the production of diffusible molecules, the release of phage or phage-like particles, and the use of contact-dependent antibacterial weaponry (2, 3). Among these strategies, the Type VI secretion system (T6SS) is a lethal weapon that functions as a crossbow and effectively delivers toxic effectors to target cells (4, 5). The T6SS ejects a spear-like tube carrying toxic effectors outward within milliseconds, possessing sufficient force to puncture through the formidable barrier of the cell wall and two cellular membranes of Gram-negative cells (6, 7). Intriguingly, the T6SS has unveiled a myriad of effectors across a diverse spectrum of bacterial species, and antibacterial toxins are the most extensively studied among these effectors (8, 9). Notably, each antibacterial toxin is encoded in tandem with a cognate immune gene within a precise proximity, a defensive mechanism shielding the bacterium from self-intoxication or T6SS-mediated killing by neighboring sister cells (4). The bactericidal efficacy of the T6SS introduces a novel approach for modulating the composition of the intestinal and rhizosphere flora in practical applications. This discovery holds promise prospects for addressing challenges related to drug-resistant bacteria, as well as for the prevention and control of plant pathogens and environmental ecological management.       Many species have evolved to produce various antimicrobial compounds that inhibit growth of or kill rival species, encompassing an array of defenses such as broad-spectrum antibiotics, proteinaceous exotoxins, lysozymes, and bacteriocins (10). Bacteriocins, a group of potent antibacterial protein, have been identified across almost all bacterial lineages (11). Remarkably, these bacteriocins are peptides synthesized by bacterial ribosomes, closely phylogenetically related species are selected as target cells, and producers have specialized immune mechanisms to counteract the effects of these peptides (12, 13). Bacteriocins are narrow-spectrum antibacterial agents that have biological effects on closely related species, and their encoding genes are found on transposable elements, plasmids, or the producer’s genome (10). The multifaceted modes of action of bacteriocins include the depolarization of lipid bilayer membranes, disruption of cell wall production (14, 15), suppression of protein synthesis (16, 17), and destruction of host nucleic acids (18). A noteworthy aspect of bacteriocins is the dependence of cellular import mechanisms on the specific target organism.       The Gram-negative bacterium Yersinia pseudotuberculosis is widely distributed in the natural environment, and as an intestinal pathogen, it can infect many animal species (19, 20). The genome of Y. pseudotuberculosis encompasses four complete sets of T6SSs that play pivotal roles in bacterial quorum sensing, host defense, and antibacterial activities (21, 22). The third Type VI secretion system (T6SS-3) gene cluster contains several effector-immune protein pairs of unknown function. Previous studies have shown that T6SS-3 secretes a novel Ca2+- and Mg2+-dependent DNA hydrolase effector protein, YPK_0954, which exhibits a contact-independent bactericidal function (23). Considering that YPK_0952 is also an effector protein secreted by T6SS-3, it could be inferred that YPK_0952 may possess similar antibacterial properties.       Here, YPK_0952 was identified as a pyocin-like effector, demonstrating its ability to be secreted via the T6SS-3, a mechanism crucial for its pronounced toxicity. Our results demonstrated that YPK_0952 relies on various bivalent metal ions to exert DNase activity and that this activity is inhibited by the immune protein YPK_0953. Furthermore, YPK_0952 exhibits intra-species and inter-species bactericidal activity through a T6SS-mediated contact-dependent and contact-independent mechanisms. Our data provide insight into important bacterial tactics that enable cells to establish a foothold in competitive niches, not only through the secretion of specialized metabolites and bacteriocins that antagonize microbial competitors but also through contact-dependent and contact-independent suppression of rival bacteria by toxins. The cross-species bactericidal action of pyocin-like effectors released by the T6SS in Y. pseudotuberculosis makes bacteriocins appealing therapeutic agents, particularly in the fight against multidrug-resistant diseases.'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11299273&rettype=xml\n",
            "Failed to fetch data for PMCID 11299273. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11298530&rettype=xml\n",
            "Failed to fetch data for PMCID 11298530. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11293832&rettype=xml\n",
            "{'pmcid': '11293832', 'doi': '10.1099/acmi.0.000830.v4', 'introduction': 'Introduction       In the realm of environmental sustainability and bioremediation, biosurfactants have emerged as valuable tools due to their eco-friendly nature and versatile applications. Derived from microbial sources, particularly bacteria, biosurfactants exhibit unique surface-active properties that enable them to reduce interfacial tension and enhance the solubilization and emulsification of hydrophobic compounds, making them promising agents for oil spill remediation. The increasing frequency of oil spills in marine environments is a serious threat to biodiversity and ecosystems. According to the Statista Research Department report, between 2010 and 2019, there was an average of 1.8 significant oil spills due to tanker incidents annually. Over 700 metric tons of oil was reported to have leaked in four separate oil spills in 2022 [1]. These incidents highlight the ongoing risk and environmental impact of oil spills. Despite efforts to improve safety measures and regulations, the frequency and magnitude of such accidents remain a concern. According to the International Tanker Owners Federation (ITOPF), between 1970 and 2019, approximately 5.86\\u2009million tons of oil was spilled into marine environments due to tanker transportation alone [1]. Every year, transoceanic shipping carries about 35\\u2009million barrels of petroleum, and there are many ship accidents, most of which result in a significant spill of petroleum into marine environments. Large-scale, toxic oil spills typically occur at marine sites. Spills and leaks of crude oil have the potential to cause terrible harm to aquatic environments [2]. There is a possibility that these hazardous substances will enter the marine environment during the extraction, transportation and refining of crude oil, which would have negative effects on all biotic life [3]. Conventional techniques for cleaning up oil spills involve the use of mechanical tools like oil booms and skimmers, but these are labor- and money-intensive procedures [1]. Chemicals can be used as oxidants, reductants, polymers, or precipitants in chemical remediation processes [4]. Chemical surfactants, on the other hand, have toxic environmental effects. Biosurfactants are surface active biomolecules produced by microbial (bacteria, yeast and fungi) metabolism, containing both hydrophilic group and lipophilic group [5]. Evaluable biotechnology technique known as ‘bioremediation’ is the process of using micro-organisms to mitigate and remove pollutants from a projected contaminated environment [6]. Because biosurfactants are more environmentally friendly, less toxic, biodegradable and have lower critical micelle concentration (CMC) values than chemical surfactants, they can be used as substitutes for chemical surfactants [7]. The critical micelle concentration refers to the minimum amount of surfactant required to achieve the lowest surface tension [8]. Biosurfactants are a type of surface-active compounds produced by micro-organisms, including bacteria, yeast and fungi. Biosurfactants are amphiphilic compounds that contain both hydrophilic (polar head) and hydrophobic (non-polar tail) domains. Surface-active compounds are categorized based on their molecular weight. Low-molecular-weight (LMW) surfactants lower the surface tension between two immiscible liquids, and high-molecular-weight (HMW) emulsifiers allow water-in-oil or oil-in-water emulsions [910]. The effectiveness of a surfactant is based on the extent to which it can lower surface tension. A highly effective biosurfactant can decrease the surface tension of water from 72 to below 35 mN/m [10]. Low molecular weight glycolipids (rhamnolipids, sophorolipids, etc.) are among the most widely used biosurfactants, and they have been thoroughly and extensively studied for their use in biotechnological applications. A glycolipid is a compound comprising a carbohydrate section and fatty acid chains connected by a glycosidic bond, acting as hydrophilic and hydrophobic links, respectively [11]. Biosurfactants can be produced by a number of bacterial species, including Bacillus, Pseudomonas and Enterobacter. Rooney et al. (2009) [12] reported the isolation of E. hormaechei from the biodiesel-contaminated soil, which was able to produce rhamnolipid-type biosurfactant [12]. Some other studies also reported the production of biosurfactants from Enterobacter spp. [13]. The delve of this study is to explore and characterize a newly isolated bacterium capable of producing biosurfactants, with the goal of assessing its potential for use in oil remediation processes.'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11293673&rettype=xml\n",
            "{'pmcid': '11293673', 'doi': '10.1371/journal.pntd.0012348', 'introduction': 'Introduction       Borrelia spp. are globally distributed, vector-borne bacteria that cause diverse diseases, rendering these pathogens an immense public health concern [1,2]. Borrelia can be divided into two distinct groups, relapsing fever (RF) and Lyme disease (LD) Borrelia, based on associated disease courses, vector/enzootic cycles, and additional phylogenetic differences [1–3]. Infection with RF spirochetes is characterized by recurring severe, febrile episodes that coincide with repeated bouts of high-level bacteremia (up to 108 bacteria/ml of blood) [1,4–8]. Furthermore, infection by RF Borrelia can lead to more severe symptoms and outcomes, such as acute respiratory distress syndrome, jaundice, meningitis, splenomegaly, and perinatal mortality [4,9,10]. Alternatively, mammalian infection with LD Borrelia results in lower levels of bacteremia (103−104 bacteria/ml of blood) that is restricted to early stages of infection, after which it disseminates into multiple host tissues [11]. Therefore, symptoms of RF are primarily associated with acute bacteremia, whereas most LD symptoms occur because of chronic tissue colonization (e.g., arthritis, neuroborreliosis, and carditis) [2]. In addition, arthropod vectors differ greatly between RF and LD Borrelia. RF bacteria are transmitted by the human body louse [louse-borne relapsing fever (LBRF)] or Ornithodoros or Ixodes spp. ticks [tick-borne relapsing fever (TBRF)], while LD spirochetes are only transmitted by Ixodes spp. ticks [12–17]. The different disease courses resulting from infection by LD and RF Borrelia and their disparate enzootic cycles involving distinct vectors suggest that these two groups of bacteria have likely evolved to encode different factors required for vector acquisition and colonization, transmission, and vertebrate infection. Because most molecular research has focused primarily on LD Borrelia, a significant gap exists in our understanding of the biology of RF spirochetes [18,19].       RF Borrelia can be further subdivided into New World (found in the Americas) and Old World (found in Africa, Asia, and Europe) species [4,5,16,20]. New World RF spirochetes include Borrelia turicatae, which is found in the arid climate of the southwestern United States, and Borrelia hermsii and Borrelia parkeri, the causes of RF in the mountainous regions of the western United States and southwestern Canada [4,5,12,13,21–23]. Old World RF spirochetes include Borrelia crocidurae and Borrelia duttonii, which are endemic to western and eastern Africa, respectively, Borrelia persica, a causative agent of TBRF in Eurasia, and Borrelia recurrentis, the LBRF spirochete responsible for epidemic RF in eastern Africa [24–30]. RF spirochetes were historically classified into New World and Old World species based solely on geographic separation, but phylogenetic analyses have revealed genomic differences in these bacteria, supporting the separation of the groups into two distinct clades [26,31–34].       Although RF is globally distributed, the public health impact varies throughout the world. While RF is a relatively rare and possibly underreported disease in the Americas, this infection is endemic and highly prevalent in parts of Africa [1,4,5,20,25–27,32]. In fact, RF has been reported as the most common bacterial infection in Senegal, the most prevalent cause of fever in rural Zaire, and a top 10 cause of death in children under the age of 5 in Tanzania [1,25,27,35–37]. Furthermore, cases are often unreported or are misdiagnosed as another disease, such as malaria, in endemic regions of Africa, implying that the actual incidence is higher than previously reported [28,38]. Despite the clear public health threat that Old World RF spirochetes represent, very little is known regarding the molecular pathogenesis of these bacteria.       Use of molecular Koch’s postulates provides the most direct way to determine a causal relationship between a bacterial gene, the protein it encodes, and a requirement during the bacterial lifecycle [39,40]. To apply molecular Koch’s postulates, the gene of interest is inactivated, and the phenotype of the mutant is compared to that of the wild-type parent. Additionally, expression of the gene must be restored in the mutant to assess if the phenotype of interest is reversed. While a limited number of studies have applied genetic manipulation in RF Borrelia to investigate proteins potentially required during the bacteria’s enzootic cycle, these studies have only been conducted with the New World RF spirochetes, B. hermsii and B. turicatae [41–50]. Extensive geographic separation and phylogenetic differences between New World and Old World RF spirochetes suggests that these two groups likely evolved distinct regulatory pathways and proteins required for mammalian virulence, as well as vector acquisition, colonization, and transmission. Therefore, there is a significant need to develop molecular techniques for the study of Old World RF Borrelia.       Herein, we aimed to develop a genetic system for use in the Old World TBRF spirochete, B. duttonii. First, we developed shuttle vector- and integration-based constructs and methods for electroporation with B. duttonii, creating green fluorescent protein (gfp)-expressing B. duttonii strains to validate the approach. Allelic exchange mutagenesis was then used to inactivate the gene encoding the B. duttonii homolog of P66, a known virulence factor of the LD spirochete, Borrelia burgdorferi [51,52]. Using a murine model of RF, we demonstrated that the B. duttonii p66 mutant was attenuated. Finally, this p66 mutant was complemented using shuttle vector- and cis integration-based approaches to restore production of P66 and murine infectivity of the p66 mutant strains. The techniques described in this manuscript represent a foundation for the application of molecular tools to study Old World RF Borrelia and will ultimately guide research to better understand these understudied pathogens.'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11293325&rettype=xml\n",
            "Failed to fetch data for PMCID 11293325. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11292733&rettype=xml\n",
            "Failed to fetch data for PMCID 11292733. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11291472&rettype=xml\n",
            "{'pmcid': '11291472', 'doi': '10.3389/fvets.2024.1436807', 'introduction': '1       Introduction       With the continuous improvement of the scale and intensity of poultry farming, the production performance of poultry has been gradually improved. However, broilers are susceptible to external factors such as disease, nutrition, and environment due to the imperfect development of intestinal function and low immunity, which frequently result in poor health, enhanced stress response, and intestinal imbalance (1). In the era of a complete ban on antibiotics as feed additives and a focus on healthy breeding, the development and utilization of feed additives like medicinal plant extracts and probiotics have garnered much attention (2, 3).       Glycyrrhiza uralensis Fisch is a traditional medicinal and edible plant with a long history of dietary and pharmacological applications, and its edible and medicinal parts are the root and rhizome (4, 5). The primary active ingredients in Glycyrrhiza uralensis extract (GUE) include glycyrrhiza polysaccharides, triterpene saponins (glycyrrhizic acid, glycyrrhizinic acid, etc.), and flavonoids (chalcone, isoflavone, etc.) (6). GUE exhibits various pharmacological effects, such as anti-inflammatory (7), antioxidant (8), antiviral (9), immune regulation (10) and improvement intestinal microbiota (11). Lactobacillus acidophilus is a dominant microbiota in the gastrointestinal tract (GIT) of humans and animals, often used as a probiotics due to its health-promoting properties (12). Probiotics Lactobacillus is typically employed as Direct-Fed microbiota to poultry and other livestock to enhance intestinal health (13), enhancing immunity (14, 15) and reducing colonization of pathogens in the GIT (16).       The intestinal tract is a crucial organ for digestion and absorption of feed nutrients in animals, closely linked to a range of physiological and biochemical processes, contributing to the animal’s nutrition and overall health (17). The intestinal microbiota is critical for maintaining host intestinal health, acting as the organism’s “second brain,” preventing pathogen colonization, modulating the intestinal epithelial barrier and inflammatory response (18, 19). The colonization of gut microbiota is relatively stable, and their composition and structure are influenced by factors such as diet, age, and feeding methods in animals (20). Changes in the GIT microbial community impact feed efficiency, productivity, and the health of chickens (21). The cecum, with the highest microbial colonization and the most abundant and diverse microbial community in the broiler gut, is crucial for the overall performance in poultry (22, 23). Generally, the chyme stays in the cecum for an extended period, allowing thorough decomposition of many feed ingredients by microbiota to improve nutrient utilization (24). Therefore, the balance of cecum microbiota is essential for enhancing overall performance of poultry.       Several recent studies have confirmed that GUE (25, 26) and probiotics (27, 28) promote growth and improve intestinal health in broilers. However, the potential effects of the combination of GUE and Lactobacillus on growth performance and intestinal health are not yet fully understood. Thus, the purpose of the present study was to investigate the impact of GUE, Lactobacillus acidophilus and their combination on growth performance and intestinal health in Liangfenghua broiler chickens, a medium-growing broiler strain known for its popularity due to the excellent meat quality. The findings of this study would be beneficial in comprehending the intestinal microecology of chickens and offering novel perspectives on the upkeep of intestinal wellness in broiler chickens that are fed antibiotic-free diets.'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11291149&rettype=xml\n",
            "Failed to fetch data for PMCID 11291149. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11289621&rettype=xml\n",
            "Failed to fetch data for PMCID 11289621. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11289428&rettype=xml\n",
            "{'pmcid': '11289428', 'doi': '10.1002/hsr2.2275', 'introduction': 'N/A'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11288818&rettype=xml\n",
            "{'pmcid': '11288818', 'doi': '10.3389/fonc.2024.1416806', 'introduction': '1       Introduction       Colorectal cancer (CRC), which includes cancer of the colon and rectum, has a high incidence and mortality rate, which seriously affects people’s physical health. According to the latest epidemiological data in 2020, the number of new cases of CRC in the world is approximately 1.9 million, and 935,000 deaths, ranking as the third and second among all malignant tumors (1). Both the incidence and mortality of CRC are still increasing year by year. It is estimated that by 2030, the global burden of CRC is expected to increase by 60%, with more than 2.2 million new cases and 1.1 million cancer deaths (2).       The incidence of CRC is on the rise in developing countries, while it has stabilized or declined in developed countries where the incidence remains high (2, 3). This is due to the early implementation of population-specific CRC early screening and lifestyle management in developed countries. As we all know, the occurrence and development of CRC mostly follow the “adenoma–cancer” sequence, and the whole development process generally lasts 5–10 years (4). Timely detection of precancerous lesions and intervention treatment can effectively prevent the occurrence of CRC and improve the prognosis (4).       The increasing number of CRC cases over the past decade indicates that in addition to genetic factors, unhealthy dietary habits, such as high intake of red meat, processed meats, refined grains, sugary beverages, desserts and potatoes (5); lack of fiber intake; and a high-fat diet, are also important factors in the development of CRC (6). In addition, intestinal flora is also a specific environmental risk factor for CRC (7). Relevant clinical studies have found that the intestinal flora colonized in the colorectal directly or indirectly participate in the body’s digestion and absorption, substance metabolism, immune regulation, and other physiological processes (8). Moreover, it can maintain the stability of intestinal microecosystem (8). The status of intestinal flora is closely related to the occurrence and development of CRC. A large number of animal models and clinical studies have proven that changes in the composition and quantity of intestinal flora promote the occurrence and development of CRC (9).       In recent years, the research on intestinal microecology has gradually increased, and more and more researchers have paid attention to this field. Studies on the relationship between intestinal flora and CRC are also increasing. This paper reviewed the correlation between intestinal flora and CRC, as well as the application of intestinal microecology in CRC screening and diagnosis to provide evidence for early screening and treatment of CRC.'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11285316&rettype=xml\n",
            "{'pmcid': '11285316', 'doi': '10.1186/s12864-024-10621-7', 'introduction': 'N/A'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11284562&rettype=xml\n",
            "Failed to fetch data for PMCID 11284562. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11281598&rettype=xml\n",
            "Failed to fetch data for PMCID 11281598. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11281493&rettype=xml\n",
            "{'pmcid': '11281493', 'doi': '10.3390/vetsci11070324', 'introduction': '1. Introduction       Research on the microbiota of the female reproductive tract in horses using high-throughput sequencing is a developing area of study, and little is known about it compared to other mammals. This niche has received limited attention in the equine species, particularly in comparison with other body regions, such as the gastrointestinal tract [1,2,3,4]. However, there is increasing evidence that the reproductive tracts of mammals harbour distinctive bacterial communities [5,6,7]. High-throughput sequencing studies have supported those using culture-dependent methods, suggesting that the uterus may not be a sterile environment [7]. However, the results of the culture-dependent method studies were limited, particularly in underestimating the abundance of certain bacteria, but the newer sequencing technologies allow researchers a broader identification of the bacteria present in this body system [8]. Despite this, few studies have characterized the microbiota of the uterus and vagina of healthy mares, particularly comparing both organs within the same animal. Most published studies currently use 16s rRNA sequencing to identify the bacterial microbiota, technical limitations or confounding factors can lead to discrepancies in the results, especially when determining which bacterial communities and at what proportion they are present in the uterus and vagina. It is difficult to establish a consensus on the normal microbiota of the reproductive tract in healthy mares unless sequencing is performed at a very deep level (shotgun sequencing), which is yet to be described in mares. In addition, determining a normal or core microbiota present in the reproductive tract of mares can be challenging because relevant factors such as the sampling site, the stage of the estrous cycle, age, nutritional regime, systemic disease, previous administration of antimicrobial drugs, season, and geographic location among other factors may impact the reproductive host’s health and therefore the microbiota. Therefore, this narrative literature review aims to summarize the current information regarding the microbial communities present in the reproductive tract of mares, the factors associated with their development and establishment, and their association with fertility, health, and disease.'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11279407&rettype=xml\n",
            "{'pmcid': '11279407', 'doi': '10.3390/pathogens13070583', 'introduction': '1. Introduction       When gastrointestinal pathogens are detected in stool samples of patients with infectious gastroenteritis living in regions where such pathogens as well as associated infectious gastroenteritis are frequent, it can be difficult to say whether the detected pathogen is really the cause of the observed clinical symptoms. This is particularly true if more pathogens than just a single one are detected in the same stool sample. The problem that most enteric pathogens in regions with high prevalence for infectious gastroenteritis can both cause enteric disease or just persist as harmless colonizers is called “facultative pathogenicity” [1,2,3,4,5]. Previously, other researchers have tried to link the quantity of pathogens in stool samples with their likelihood of causing infectious gastroenteritis in such patients. In the case of diagnostic real-time PCR, so-called cycle threshold (Ct) values are an indirect option for target quantification because low Ct values indicate high quantities of the PCR target and vice versa. However, such attempts at linking pathogen quantity in stool samples with the likelihood of this pathogen causing clinical disease were only partly successful [2,3,6]. Consequently, generally accepted cut-offs for a Ct-value-based assignment of etiological relevance of enteric pathogens do not exist.       Obviously, more factors influence the likelihood of an association between clinically observed infectious gastroenteritis and the detection of an enteric pathogen in a human stool sample. One of these factors is “semi-immunity”, which means immunological adaptation of the human gut to rapid cycles of repeated pathogen exposure under poor hygiene conditions. This has been observed repeatedly in resource-limited tropical regions [7,8,9]. Although it is not yet completely understood how enteric semi-immunity works on a cellular or molecular level, some well-defined hypotheses regarding host–pathogen coexistence have been proposed, particularly for enteric helminth infections, as summarized elsewhere [10]. Furthermore, the semi-immunity concept is already in preventive medical use. Oral typhoid fever vaccination is the most commonly known example of induced short-term immunity on enteric mucous membranes [11].       Next to immunological adaptations, the enteric microbiome’s composition is believed to play a role in the degree of susceptibility towards the virulence of enteric pathogens. For both laboratory animals and human individuals, favorable microbiome compositions have been demonstrated to mitigate colonization resistance towards enteric pathogens, thus making gastroenteric infections less likely [12,13,14]. In addition, there is an increasing body of evidence suggesting likely interaction between enteric pathogens and commensals, which affects the clinical outcome [5,15].       In order to contribute to deciphering such microbial interactions in the gut of individuals from high-endemicity settings for infectious gastroenteritis, our study group recently conducted a cluster analysis assessing gastroenteric pathogens in stool samples of a Colombian indigenous population [16]. Within this population, a cluster consisting of Blastocystis hominis, Campylobacter spp., and Giardia duodenalis was shown to interact with Dientamoeba fragilis and Ascaris lumbricoides in a microbial-density-dependent way [16]. During the interpretation of this finding [16], however, it remained uncertain whether this observation just represented a regional peculiarity or a general pattern of microbial interaction in the human gut.       As such, it seemed promising to repeat the analysis with populations from other tropical high-endemicity settings for gastroenteric infections. Ghana, in the West African region, is an example of a country where enteric pathogens can be detected both in association with diarrheal disease and in asymptomatic individuals [4,17]. Especially high detection rates of gastroenteric pathogens can be expected in Ghanaian children [18,19,20], and in older studies, when diagnostic approaches with poor diagnostic accuracy, like Widal testing, were still in use [21], even underestimations of the true prevalence were likely. In such populations, co-occurrence of various microbial agents in stool samples as well as rapid pathogen acquisition cycles have been reported previously [22,23]. Furthermore, the common co-occurrence of genetic resistance determinants in Ghanaian stool samples [24,25] bears the risk of resistance transmission to bacterial enteric pathogens via mobile genetic elements. For Campylobacter spp. detections in Ghana, resistance was pronounced in cases of HIV-positive patients [26]. Enteric protozoan parasite infections of the gut have been reported to be particularly frequent in Ghanaian patients with diabetes [27]. Ghanaian farm environments provide reservoirs for several enteric pathogens like, e.g., salmonellae [28] and Cryptosporidium spp. [29]. Fecal contamination of regional environments is common [30], and food-borne or water-borne transmissions of infectious gastroenteritis are frequent events, particularly for poor Ghanaians [31,32,33]. Consequently, evidence of long-term efficiency of water filtration has recently been confirmed for Ghana [34], and, in line with the abovementioned information, colonized food vendors play a relevant role for such food-borne transmission events in Ghana [35].       Based on such previous experiences, Ghana was chosen as a suitable candidate region for cross-checking the experience from Colombia [16] in another tropical high-endemicity setting. To do so, cluster analyses were performed both with the set of microbial parameters chosen for the Colombian assessment [16] alone as well as in comparison with the Colombian results published elsewhere [16] and also with a broadened dataset available for the Ghanaian samples only. Furthermore, the dataset on Ghanaian individuals was sub-divided into subsets comprising children under 2 years of age and HIV-positive individuals. The rationale of these analytical steps is as follows. If microbial interactions are stable, comparable clustering should appear despite regionally different populations and despite interindividual differences, including factors like medical conditions and environmental factors. In summary, the study aimed at identifying stable clusters as markers of microbial interaction with potential importance for etiological relevance assignment in cases of multiple enteric pathogen detections, and the inclusion of different subpopulations, including children and HIV-positive individuals, was performed to further challenge the stability of potentially observed clustering.'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11279152&rettype=xml\n",
            "Failed to fetch data for PMCID 11279152. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11278787&rettype=xml\n",
            "{'pmcid': '11278787', 'doi': '10.3390/microorganisms12071328', 'introduction': '1. Introduction       Water is commonly contaminated by enteric bacteria, which inhabit the human gastrointestinal tract and are excreted in fecal matter [1]. Consuming water contaminated with these microorganisms can lead to intestinal infections [2]. Total coliforms are a key indicator group for assessing water quality [3].       Traditional methods for detecting and enumerating coliform bacteria typically involve lengthy processes, including enrichment with culture media in the laboratory [4]. However, advancements in molecular techniques have introduced faster, more sensitive, and more specific methods for microbiological water analysis [5], many of which can be automated. These new techniques enable the identification of pathogens, including novel and emerging strains [6]. Consequently, portable biosensors are now being discussed for their high sensitivity in detecting low coliform bacteria concentrations in water samples [2].       Conventional enumeration of viable pathogens in the water relies on fecal indicator bacteria culture methods using selective media. Research has shown that bacteria, including pathogens like Escherichia coli, can enter a viable but non-culturable (VBNC) state under environmental stress and starvation conditions [7,8,9]. This state affects the accuracy of traditional enumeration methods for viable bacteria in water supplies and wastewater treatment systems. RT-PCR also quantifies the target organism’s DNA in real time, providing information on the concentration of pathogens in the sample. However, these methods can overestimate viable bacteria counts due to the persistence of DNA after cell death [10,11]. To address this, mRNA detection via RT-PCR has been developed to test the viability of VBNC bacteria [12,13].       Water quality testing often focuses on detecting Escherichia coli, considered the best indicator of fecal contamination [14,15,16,17,18]. According to WHO guidelines, water poses an intermediate risk when E. coli counts range from 10 to 100 CFU per mL and a high risk when counts are between 100 and 1000 CFU per mL [14].       Preferred methods for microbiological water quality testing include membrane filtration and colony counts. E. coli confirmation typically employs immunoassays (ELISA) and PCR with species-specific DNA primers [17]. Traditional culture-based methods allow for sufficient bacterial growth for detection and enumeration for at least 24 h [19]. These methods involve culturing the bacteria on selective media, followed by colony counting and biochemical testing to determine the concentration of viable cells in the sample [20]. Although practical and low-cost, these methods are time-consuming, less sensitive, and less specific.       Slow and time-consuming traditional methods can delay disease diagnosis and treatment [20]. They are also expensive and require highly trained personnel. Nucleic-acid-based strategies, particularly PCR, offer high sensitivity and rapid detection but are labor-intensive, time-consuming, and costly [21,22]. Despite these drawbacks, nucleic-acid-based methods are powerful tools for pathogen detection in food, overcoming long enrichment stages due to their ease of use and automation potential.       The current molecular methods based on microarrays, nucleic acid sequence amplification, PCR, PCR hybridization, and real-time PCR allow for the detection of various pathogens in the same test. These techniques enable high specificity and sensitivity rates, ensuring rapid and accurate identification of multiple pathogens [23]. In addition, using PCR for pathogen detection requires knowledge of target DNA sequences to design specific primers [24], which limits the technique’s detection capacity.       Multiplex PCR enables the detection of various pathogens, including E. coli strains and their verotoxins. Additionally, this technique can offer high sensitivity and specificity, making it a valuable tool for food safety testing [25]. However, inhibitors in samples can affect the technique’s sensitivity and specificity, leading to false negatives or positives. Additionally, multiplex PCR requires careful validation for each pathogen and toxin.       ELISA is a widely used technique in immunology and diagnostics due to its ability to detect and quantify specific E. coli antigens or antibodies in biological samples. ELISA typically takes about 24 h from sample preparation to results [26,27], though this can vary depending on sample preparation, incubation times, and detection methods [28,29]. ELISA quantitatively determines antigen concentrations in samples, with signal intensity from labeled antibodies correlating with the antigen amount. This quantitative aspect makes ELISA powerful in clinical diagnostics, immunology research, and biotechnology [30,31]. The method involves immobilizing the target antigen onto a solid substrate, typically a plastic surface, followed by detection using a labeled selective antibody [32]. The sensitivity of ELISA can be influenced by various factors, including the quality of antibodies used, the efficiency of antigen capture and detection, and the presence of interfering substances in the sample matrix. Therefore, validating the ELISA method thoroughly and comparing its performance against reference methods is essential to ensure reliable results.       Capture ELISA (cELISA) is a widely used technique for detecting E. coli. Through this method, antibodies that specifically target E. coli are fixed to a plate. When a sample containing E. coli antigens is introduced, these antibodies capture and bind the antigens, allowing for their detection. After capturing the antigens, primary antibodies specific to E. coli are added, followed by secondary antibodies. These secondary antibodies are linked to an enzyme, such as horseradish peroxidase (HRP), or directly conjugated to the detection molecule. Although cELISA offers rapid detection, it may sacrifice some sensitivity compared to more time-consuming methods, balancing speed and accuracy for various applications [33].       Direct ELISA (iELISA) simplifies assays by requiring only one antibody—the primary antibody binding to the target antigen—compared to cELISA, which involves two antibodies [30]. This approach streamlines the experimental setup and is suitable when a specific antibody is available for the antigen of interest.       Biosensors provide advantages over conventional detection methods, including speed and sensitivity [34,35]. They offer high specificity, detecting specific pathogens in water and food samples with precision [36]. Biosensors offer multiplexing. Additionally, biosensors can target various pathogens or contaminants simultaneously, reducing the timing of analysis and providing detection accuracy by providing comprehensive information about the sample’s microbial or contaminant profile [37], making them valuable tools in food safety and environmental monitoring.       Optical biosensors provide rapid and efficient responses for ensuring food safety, particularly in scenarios where timely detection is crucial to prevent outbreaks or minimize economic losses [38]. They can be integrated into automated systems, enabling real-time monitoring of low levels of target analytes and providing high sensitivity and specificity in food samples. In addition, optical biosensors offer portability and ease of use, making them well-suited for deployment in field environments [38].       Integrating photonic biosensors onto silicon integrated circuits or PICs revolutionizes point-of-care diagnostics by offering compact, sensitive, and multiplexed detection platforms highly compatible with CMOS technology [39]. Sensitivity is crucial for biosensor performance, as indicated by limits of detection (LoD) and quantification (LOQ). Enhanced sensitivity improves biosensor efficacy in detecting and measuring specific substances [40]. The sensor’s sensitivity is determined by how effectively the evanescent field interacts with the sample molecules [41,42].       Microfluidic-based biosensors with microchannels for fluidic samples are becoming increasingly relevant due to their on-chip immunoassay capabilities [43]. These systems perform various laboratory processes concurrently on a single chip, including detection, sampling, separation, and mixing [44,45]. Besides being extremely sensitive [46], biosensors can be incorporated into portable sensing systems [47]. Portable biosensors can determine spatio-temporal variations in water quality by deploying sensors in water sources or installing them at the point of source [47].       Ultra-sensitive biosensors with low LoD can detect single microbial cells. While this sensitivity is ideal for assessing water quality, the uneven spread of microbial cells in large volumes of water (e.g., lakes, rivers, wells) can affect accurate cell count. This criterion must be considered when designing biosensors for water quality assessment.       While optical microfluidic biosensors demonstrate competitive sensitivities, they still face challenges that must be addressed, including high costs and complex procedures for integrating optical components with microfluidic channels and functionalized surfaces [48]. Current E. coli biosensors still involve labor-intensive manual procedures, which can limit their practical use in laboratory settings.       This research aims to optimize a photonic immunosensor capable of detecting both viable and non-viable forms of E. coli in water samples for human consumption. The detection method uses ring resonator transduction within functionalized Photonic Integrated Circuits (PICs) coated with specific antibodies targeting lipopolysaccharides (LPSs) from the outer membrane of E. coli or E.-coli-specific β-galactosidase, a peptide carrier protein with biological activity. By enabling early detection and quantification of viable and non-viable bacteria, this technology promises to predict pathogen contamination in water samples, addressing current limitations in pathogen detection systems within the agrifood industry and environmental sector.'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11275779&rettype=xml\n",
            "Failed to fetch data for PMCID 11275779. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11274912&rettype=xml\n",
            "Failed to fetch data for PMCID 11274912. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11274296&rettype=xml\n",
            "Failed to fetch data for PMCID 11274296. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11273848&rettype=xml\n",
            "{'pmcid': '11273848', 'doi': '10.3390/antibiotics13070613', 'introduction': '1. Introduction       Staphylococcus aureus, a member of the ESKAPE (Enterococcus faecium, S. aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa, and Enterobacter species) group of pathogens, is a leading culprit of nosocomial infections [1]. The emergence of biofilm-forming, multidrug-resistant S. aureus strains, particularly methicillin-resistant S. aureus (MRSA), poses a formidable challenge in healthcare and agricultural settings [2]. S. aureus can cause a wide range of infections and exhibit a remarkable ability to develop resistance against conventional antibiotics [3,4]. Its pathogenicity is largely attributed to its ability to form robust biofilms, which are microbial communities encased in an extracellular matrix that enables evasion of host immune responses, antimicrobial resistance, and tenacious surface adhesion [5,6]. Additionally, the multilayered biofilms are facilitated by the icaADBC operon, which facilitates the synthesis of an extracellular matrix [7]. S. aureus employs quorum-sensing (QS) mechanisms to coordinate the expression of virulence factors, exacerbating its pathogenicity. Combating biofilm formation and disrupting QS signaling have emerged as critical objectives in mitigating the impact of S. aureus infections, underscoring the urgent need for novel therapeutic approaches tailored for healthcare and veterinary applications [8]. The QS mechanism in S. aureus is governed by two main cascades: the staphylococcal accessory regulator (Sar) and the accessory global regulator (Agr). Within the Agr QS system, oligopeptides serve as signaling molecules to modulate the expression of various virulence factors. SarA (Staphylococcal accessory regulator A) is a pivotal regulatory protein in the S. aureus QS cascade, significantly impacting bacterial pathogenicity and resistance mechanisms [9,10]. It controls the expression of numerous virulence factors, influencing the transcription of gene-encoding toxins, enzymes, and surface proteins, thus playing a crucial role in the bacteria’s ability to cause infections ranging from minor skin conditions to severe diseases such as pneumonia and sepsis. SarA promotes biofilm formation by upregulating genes responsible for extracellular polysaccharide production and repressing proteases that could degrade the biofilm matrix, thereby enhancing bacterial resistance to antibiotics and host immune responses. Furthermore, SarA contributes to antibiotic resistance by regulating stress response and cell wall synthesis genes, including the methicillin resistance gene mecA. Given its central role in virulence and biofilm formation, SarA is a promising target for new therapeutic strategies aimed at attenuating S. aureus pathogenicity and enhancing susceptibility to antibiotics [8,11].       Bioactive phytoconstituents have emerged as promising agents for combating infectious diseases caused by bacteria that form biofilms. These compounds, derived from plants, exert their therapeutic effects by targeting genes responsible for disease progression, particularly by disrupting virulence factors and inhibiting biofilm formation associated with quorum sensing (QS). Traditional medicine has a rich history of utilizing plants to combat microbial infections [12,13]. Among these botanical remedies, Paederia foetida (known as Sembukan in Indonesian and King’s Tonic in English), a climbing herb belonging to the Rubiaceae family, stands out as one of Asia’s indigenous medicinal plants, thriving in both temperate and tropical regions. Despite its bitter taste and unpleasant odor, the leaves of this plant are traditionally employed to address a range of ailments, including diarrhea, rheumatism, inflammation, piles, dysentery, and stomachache [14,15,16]. This traditional wisdom serves as a valuable resource for uncovering potential drugs. Notably, compounds such as quercetin, catechin, rosmarinic acid, limonoid, ichangin, apigenin, kaempferol, and naringenin have demonstrated efficacy against biofilm-associated infections [17]. Consequently, there is a pressing need to identify potent QS inhibitors (QSIs), preferably sourced from natural reservoirs. The secondary metabolites found in plants have shown promise in effectively treating various infections [18].       Exploring anti-quorum-sensing and anti-biofilm agents from natural sources, such as plant extracts and biologically synthesized agents, offers promising avenues for controlling S. aureus infections [19,20]. Molecular docking simulations predict binding modes and affinities between bioactive compounds and key proteins in bacterial QS pathways, guiding the design of novel, potent QS inhibitors [21,22]. This interdisciplinary approach aligns with the One Health initiative, recognizing the interconnectedness of animals, humans, and the environment [23,24]. The present study investigated the antimicrobial, anti-QS, and anti-biofilm formation abilities of P. foetida Linn. leaf extract against S. aureus, focusing on managing infections in healthcare and veterinary settings while embracing the One Health approach to combat this global threat.'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11272550&rettype=xml\n",
            "{'pmcid': '11272550', 'doi': '10.3389/fmed.2024.1428432', 'introduction': '1       Introduction       The treatment of fracture-related infections (FRI) is challenging. Particularly, the involvement of severe bone defects and multi-resistant bacteria frequently leads to complex situations. The treatment of FRIs mainly consists of two key aspects: First, control of the infection and causative microorganisms must be achieved. Second, bone reconstruction with subsequent successful bone healing needs to be obtained (1).       Both local and systemic antibiotic therapy are hallmarks of fracture-related infection treatment. However, there are often only limited antibiotic treatment options when multi-drug resistant bacteria are involved. An alternative or complementary approach for the treatment of complex infections with multi-drug resistant bacteria is bacteriophages (short phages). The bactericidal effect of phages was first identified more than 100\\u2009years ago by Felix d’Herelle (2). Since then, phages have been studied and used mainly in the former Soviet Union during the second half of the 20th century. In recent years, bacteriophage therapy has also gained interest in the Western world due to the rise of multi- and pan-drug-resistant bacteria, including difficult-to-treat bone and joint infections (3). However, there are only a few published case series or case reports on the treatment of musculoskeletal infections with bacteriophages (4). A recent review identified 33 bone and joint infection cases treated with phages, out of which notably 29 (87%) achieved either microbiological or clinical success. Furthermore, 8 of the 33 cases (24%) experienced mild and temporary adverse effects, but there were no reports of serious complications (5). In addition, underlining the efficacy of phage therapy in resolving difficult-to-treat infections, Pirnay et al. have just recently published clinical outcomes of their first 100 cases treated with personalized phage therapy. Bone infections were the three most common indications. Overall, in 77.2% of targeted infections, patients experienced clinical improvement. In 61.3% of infections where relevant bacteriological follow-up data were available, eradication of the targeted bacteria was observed (6). Ongoing clinical trials aim to refine phage application mode and advance our understanding of how to effectively manage bone infections. Currently there are a few clinical trials registered, which already entered the phase of recruiting patients for phage treatment.       Hitherto, however, the optimal administration of bacteriophages has not yet been elucidated. Phages can be administered intravenously or locally. Different approaches with varying quantity, frequency, and type of application have been described to date (4, 7). Recently, Ferry et al. published a case report with the use of hydrogel as carrier material for the application of bacteriophages in a patient with a mega prosthetic hip infection (8). This study showed a rapid release of the phages from the DAC hydrogel in an in vitro setting. However, no in vivo data for the release of the phages in the patient were presented.       Therefore, in this study, we present a clinical case involving the loading of bacteriophages into a hydrogel in an FRI. Additionally, we assess the in vivo release kinetics of the phages from the hydrogel in the patient’s wound using high-throughput metagenomic sequencing-based characterization of the administered bacteriophage cocktail, followed by real-time PCR quantification of selected bacteriophages in the drainage fluid.'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11271939&rettype=xml\n",
            "Failed to fetch data for PMCID 11271939. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11271649&rettype=xml\n",
            "Failed to fetch data for PMCID 11271649. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11269248&rettype=xml\n",
            "{'pmcid': '11269248', 'doi': '10.3389/fmicb.2024.1423610', 'introduction': 'Introduction       The genus Yersinia contains 28 species, three of which are pathogenic for humans (access: January 2024).1 Yersinia (Y.) pestis is the causative agent of plague, transmitted by the bite of an infected flea. On the other hand, Y. pseudotuberculosis and particularly Y. enterocolitica are foodborne enteropathogenic species, which may cause gastrointestinal diseases termed yersiniosis (Le Guern et al., 2016). In 2022, yersiniosis was the third most commonly reported bacterial foodborne zoonotic disease in the European Union (EU) (European Food Safety Authority and European Centre for Disease Prevention and Control, 2023). Typical symptoms are fever, abdominal pain and diarrhea (Bancerz-Kisiel and Szweda, 2015). While human yersiniosis is mainly caused by Y. enterocolitica, Y. pseudotuberculosis is more often found in wild and zoo animals causing various infections, some of which are lethal and exhibit tuberculosis-like symptoms. By contrast, in Europe and many other countries, the presence of Y. enterocolitica is strongly associated with domestic pigs, which do not get ill, but represent a natural reservoir for the bacteria (Fredriksson-Ahomaa et al., 2006a,b; Bancerz-Kisiel and Szweda, 2015; European Food Safety Authority, 2015). Human infections by this species are predominantly linked to the consumption of raw or insufficiently cooked pork (Laukkanen et al., 2008). However, raw milk, water and vegetables such as mixed salad contaminated with Y. enterocolitica have also been reported as possible sources of infection (Jamali et al., 2015; MacDonald et al., 2016; Terech-Majewska et al., 2016).       Y. enterocolitica is divided into six biotypes (1A, 1B, 2, 3, 4 and 5) and more than 70 serotypes (O) (Bancerz-Kisiel et al., 2018). In Europe, the most clinically important bioserotype is 4/O:3, followed by 2/O:9 and 2/O:5,27. Biotype 1A strains frequently occur in the environment and food and are generally regarded as nonpathogenic, although some reports have isolated this biotype from sick individuals (Joutsen et al., 2016; Bancerz-Kisiel et al., 2018).       Since the use of antibiotics is generally avoided in food production, alternative methods are needed to reduce these bacteria along the food chain. One alternative approach to treating bacterial infections or reducing pathogens in food is the application of virulent (strictly lytic) phages, which can also be effective against multidrug-resistant bacteria (Xu, 2021). A number of phages lysing Y. enterocolitica has been described, most of them are podoviruses and myoviruses (Salem et al., 2015; Gwak et al., 2018; Salem and Skurnik, 2018). Many podoviruses infecting Y. enterocolitica are related to T7 and have a rather narrow host range as they mainly lyse O:3 strains (Pajunen et al., 2001; Leon-Velarde et al., 2014; Liang et al., 2016). An exemption is the T7-like phage vB_YenP_Rambo that lysed 50 out of 62 and 54 out of 62 pathogenic Y. enterocolitica strains belonging to several bioserotypes at 28°C and 37°C, respectively (Hammerl et al., 2022b). Y. enterocolitica myoviruses mostly exhibited a broader host range than podoviruses, because they could additionally lyse strains belonging to the serotypes O:9 and O:5,27. One myovirus (fHe-Yen9-01) that lysed 61.3% of 106 tested Yersinia strains was used for reduction experiments with food samples (pork and milk) and kitchenware and was able to reduce an O:9 strain significantly (Jun et al., 2018). The widest host range of all Yersinia phages described thus far showed the myoviruses PY100 and the very similar vB_YenM_P281. These two phages not only infected various bioserotypes of Y. enterocolitica (61.1% of 72 strains at 28°C and 95.8% at 37°C) but also strains of Y. pseudotuberculosis if the medium was supplemented with calcium and magnesium cations (Hammerl et al., 2022b). A combination of vB_YenM_P281 and vB_YenP_Rambo reduced an O:3 and an O:9 strain efficiently down to a temperature of 17°C. Phage-induced lysis of Enterobacteriaceae at low temperatures generally occurs much more slowly than at higher temperatures, such as 37°C, due to the limited propagation of the host bacteria. Additionally, higher multiplicities of infection (MOIs) are often required to achieve strong lytic activity, as shown for the phages vB_YenM_P281 and vB_YenP_Rambo, where a tenfold higher MOI was required for a significant reduction of the tested strains at 17°C (Hammerl et al., 2022b). The access to host cell receptors, important for binding of a phage may also change with the temperature.       In this study, we isolated and characterized three novel virulent myoviruses from game animals (vB_YenM_P744 and vB_YenM_P778) and sewage (vB_YenM_P8). These phages demonstrated excellent lytic activity at low temperatures and low MOIs, complementing each other well regarding their host range. We further evaluated a cocktail of these three phages, together with vB_YenM_P281 and vB_YenP_Rambo, which efficiently lysed a mixture of pathogenic Y. enterocolitica strains.'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11268173&rettype=xml\n",
            "Failed to fetch data for PMCID 11268173. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11267873&rettype=xml\n",
            "Failed to fetch data for PMCID 11267873. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11265448&rettype=xml\n",
            "{'pmcid': '11265448', 'doi': '10.1128/msystems.00520-24', 'introduction': 'INTRODUCTION       The giant panda (Ailuropoda melanoleuca) is a rare and large endemic species in China and is one of the most adored and protected species worldwide. At present, there are more than 600 captive giant pandas according to the studbook of giant pandas of 2020 reported by the Chinese Association of Zoological Gardens. More than 1,800 wild giant pandas are found in the mountainous areas of Sichuan, Shaanxi, and Gansu provinces of China according to the Fourth National Survey of Giant Pandas completed in 2015. The threat level of giant pandas has been downgraded from \"endangered\" to \"vulnerable\" (http://www.forestry.gov.cn). However, giant pandas are often threatened by intestinal diseases, the main reason for mortality caused by ascarid nematodes and bacterial infections, such as Escherichia coli, Klebsiella pneumoniae, Clostridium perfringens (1–4). Thus, it is critical to identify efficient ways to prevent and cure intestinal diseases of this beloved species.       Some lactic acid bacteria (LAB) play a key role for the host animal by providing health benefits. Animal studies have shown that LAB significantly prolonged the life span of Caenorhabditis elegans (5) and enabled blood urate control in mice (6). LAB could potentially help to control some cancers (7, 8) and regulate the normal gastrointestinal microbiota, maintain the microecological balance, inhibit pathogen proliferation, and prevent inflammation (9–11).       Owing to the important physiological role of intestinal LAB, screening functional LAB from giant pandas is a good way to potentially use them as probiotics and to prevent and treat intestinal diseases. As of now, the isolated LAB strains from giant pandas mainly included Bifidobacterium (12), Lactobacillus (11, 13–16), and Weissella (17, 18). In these studies, some authors reported the characteristics and drug sensitivity of these strains (12, 17, 18), their survival at low pH and high bile salt concentrations, their activity against pathogens, and their ability to alleviate the inflammatory response (11, 13–15). However, more knowledge about LAB in giant pandas is needed to aid in conservation of giant pandas. A good way to analyze the diversity of LAB and to screen functional strains in giant pandas is to culture the microbiota with culturomics (19).       Cultivation is necessary to screen strains to experimentally investigate microbial traits, to confirm the activities of novel genes through functional characterization studies, and to identify potential health benefits for the development of probiotics (19–21). Culturomics is a comprehensive culture-based approach that applies multiple cultivation conditions to identify bacteria and genes that contribute to potential functions of interest (22, 23). The culturomics was used to successfully culture lower-abundance bacteria (24) and “uncultivable” bacteria (25) in human gut, and to unveil members of the swine gut microbiota (26).       In addition, symbiotic gut bacteria produce different classes of metabolites, such as short-chain fatty acids (SCFAs), which can not only serve as energy substrates but also as signaling molecules with diverse functional roles, including protection of the gut barrier function, modulation of energy metabolism, alteration of immune mechanisms, regulation of metabolic homeostasis, and neuroprotective effects for combating disease and improving health (27, 28).       Thus, in this study, we used culture-independent and culture-enriched methods to reveal the diversity of LAB in giant pandas, compare the difference between the two methods, and establish a pipeline to screen novel strains with the potential for use as probiotics following the combination of liquid and solid cultures. This study provides guidance for future giant panda gut microbiota cultivation.'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11263920&rettype=xml\n",
            "{'pmcid': '11263920', 'doi': '10.2903/j.efsa.2024.8896', 'introduction': 'N/A'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11262882&rettype=xml\n",
            "{'pmcid': '11262882', 'doi': '10.1155/2024/8772009', 'introduction': 'N/A'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11261454&rettype=xml\n",
            "{'pmcid': '11261454', 'doi': '10.1016/j.psj.2024.103975', 'introduction': 'N/A'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11260827&rettype=xml\n",
            "{'pmcid': '11260827', 'doi': '10.3389/fmicb.2024.1394463', 'introduction': \"1 Introduction       The animal gut houses a wide range of microbes (Bankers et al., 2021; Levin et al., 2021), which play crucial roles in various aspects of the host digestion (Francoeur et al., 2020), detoxification (Levin et al., 2021), and immune system (Xiong et al., 2019). Despite the importance of these functions for most animals (Moran et al., 2019), the processes that shape gut microbiota remain largely unexplored, particularly in the case of freshwater snails. Understanding the mechanisms shaping snail gut microbiota is a crucial first step in developing microbial approaches to control invasive snails and preserve native snails.       After years of research, we have gained some insights into the gut microbiota of freshwater snails, with a primary focus on their diversity, composition, and functions (Liu et al., 2018; Chen et al., 2021). Studies have revealed that the gut microbiota in freshwater snails is shaped by both host-related factors, such as species (Zhou et al., 2022), sexes (Chen et al., 2021), and developmental stages (Chen et al., 2021), and environmental factors like season (Li et al., 2022b), temperature (Li et al., 2022a), and toxins (Liu et al., 2023). These studies have systematically uncovered the variations in the gut microbiota of freshwater snails. Among these, the invasive apple snail Pomacea canaliculata attracts attention not only as a pest of rice (Halwart, 1994; Attademo et al., 2016) but also for its ecological advantages in invaded habitats (Fang et al., 2016; Liu et al., 2021b). Research on this snail indicates that its gut microbiota may facilitate adaptation to new environments (Li et al., 2022b; Zhou et al., 2022), thereby providing a basis for developing invasive management strategies that leverage snail gut microbiota. However, there is still much to be explored to harness the potential of gut microbiota, such as understanding the assembly processes of freshwater snail gut microbiota.       According to the current consensus of community assembly, both deterministic and stochastic processes jointly govern the assembly of communities (Zhou and Ning, 2017; Ning et al., 2019, 2020). The neutral theory highlights the stochastic processes, like random birth or death events, immigration, and ecological drift (Chave, 2004), while the niche-based theory assumes that deterministic processes, including environmental filtering and biological interactions (Chesson, 2000). Quantifying the importance of these two processes in governing microbiota assembly has sparked significant scientific interest (Liu et al., 2020; Wang et al., 2020; Ge et al., 2021). In the study of gut microbiota, these processes have also been extensively investigated (Burns et al., 2016; Zhu et al., 2016; Xu et al., 2023) and are considered ecology and evolution significant (Kohl, 2020; Ge et al., 2021). For instance, researchers have found that changes in the importance of these processes can effectively explain the differences in the composition of gut microbiota from different honeybee species and geographical sites (Ge et al., 2021). However, there has been limited research focusing on the potential association between community assembly and gut microbiota diversity. The diversity of animal gut microbiota is thought to be influenced by host filtering, which includes factors such as the immune system and stomach pH (Reese and Dunn, 2018). Host filtering functions as the host's selection for specific microbes (Ley et al., 2006; Kohl, 2020), corresponding to deterministic processes (selection) in community assembly (Xiong et al., 2017; Xiao et al., 2021). It can be inferred that stronger host filtering may lead to a more deterministic assembly, ultimately limiting the diversity of gut bacterial communities. One example of this is that a lower stomach pH can act as a filter, potentially reducing the diversity of gut microbiota (Reese and Dunn, 2018).       Thus, the core objectives of this study are to elucidate how assembly processes shape freshwater snail gut microbiota. We investigated the gut bacterial communities of two freshwater snails, the invasive snail (Pomacea canaliculata) and the native snail (Viviparidae), co-existing across different sites and thus exposed to the same pool of potential microbial colonizers. Although previous studies have reported on the gut bacterial communities of these two snails, their assembly patterns remain unknown, which is crucial for understanding how bacteria coexist in snail gut. To achieve this, we first comprehensively compared the differences in diversity and composition of the gut microbiota between these two snails. Subsequently, we attempted to decipher the observed differences, especially in diversity, using community assembly theory.\"}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11260685&rettype=xml\n",
            "Failed to fetch data for PMCID 11260685. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11257890&rettype=xml\n",
            "{'pmcid': '11257890', 'doi': '10.3389/froh.2024.1425937', 'introduction': \"1       Introduction       Periodontitis is one of the most prevalent microbially-induced disorders worldwide (1). The primary cause of this inflammatory disease, which affects the tissues supporting the teeth, is the presence of a dysbiotic microbial community in the subgingival crevice (2). In a healthy state, subgingival biofilms are formed predominantly by commensal microbes; however, periodontal pathogens may elevate the pathogenicity of the whole community and trigger an exacerbated inflammatory response that leads to tissue destruction and ultimately tooth loss (3, 4). Periodontitis is linked to numerous systemic conditions, such as rheumatoid arthritis, cardiovascular disease, diabetes, cancer and Alzheimer's disease (5–7). The microbial aetiology of periodontal disease has long been established, and among the periodontal pathogens, three species are most often associated with disease development—Porphyromonas gingivalis, Tannerella forsythia and Treponema denticola (8, 9). These bacteria employ an array of virulence factors to establish their pathogenicity within the oral cavity (10, 11), including several proteolytic enzymes. In the case of P. gingivalis, gingipains—aggressive cysteine proteases (RpA, RgpB, and Kgp)—are attaining increasing interest for their wide-reaching effects on the host immune system and systemic diseases (12). For example, they exert proteolysis at the cell surface of human gingival epithelial cells (13) and enhance blood-brain barrier permeability (14). Similarly, for T. forsythia, a plethora of proteases has been linked to the bacterium's pathogenicity, including, e.g., PrtH proteases transferring attachment loss (15), a trypsin-like cysteine protease with both arginine- and citrulline-specific activities (16), and secretory KLIKK proteases exhibiting activity towards diverse protein substrates, including collagen, gelatine, elastin and casein (17).       A less-investigated class of proteases in oral pathogens are HtrA (high-temperature requirement A) enzymes. For pathogens, HtrA family members are generally crucial for survival, adaptation to environmental changes, and tolerance of harsh conditions such as elevated temperatures (heat-shock response), extreme pH and oxidative or osmotic stress (18). In several cases, loss of HtrA function correlates with reduced virulence and restricted bacterial growth under external stresses (19). Thus, bacterial HtrAs are regarded as promising candidates for developing novel antibacterial strategies (20). Mechanistically, HtrAs have ATP-independent dual chaperone-protease activity and mediate protein quality control (19, 21). Members of the HtrA protein family consist of a conserved chymotrypsin-like serine protease domain with the catalytic triad composed of a histidine, aspartate and serine residue (22), and at least one C-terminal regulatory PDZ (post synaptic density of 95\\u2005kDa, discs large, and Zonula Occludens 1) domain for substrate recognition, binding and oligomerization (21, 22). The two major structural subdivisions within this protein family are the DegP/O and DegS enzymes, which ensure protein quality control in the periplasm of E. coli (21). In addition to the serine-protease domain, they contain either two PDZ domains and an N-terminal signal peptide for cytoplasmic membrane translocation (DegP/O), or a single PDZ domain preceded by an N-terminal transmembrane segment for membrane anchoring (DegS) (18). Other HtrA proteins can be translocated to the cell surface or secreted, enabling them to participate in bacterial colonization or host invasion (22, 23). It is proposed that the extracellular activity of HtrAs involves their transport through outer membrane vesicles (22).       HtrA enzymes are crucial for the virulence of several important pathogens due to their diverse functionalities. These pathogens include E. coli (24), Listeria monocytogenes (25), Legionella pneumophila (26), Borrelia burgdorferi (27), Helicobacter pylori (28), Campylobacter jejuni (29), and Bacillus anthracis, among others. In H. pylori, HtrA acts as a secreted virulence factor that cleaves the ectodomain of the cell-adhesion protein E-cadherin leading to E-cadherin shedding and disruption of the epithelial barrier functions, allowing H. pylori to access the intercellular space (30). In C. jejuni, secreted HtrA facilitates transepithelial migration by degrading E-cadherin (31). A C. jejuni deletion mutant lacking HtrA was shown to induce lower levels of apoptosis and reduced secretion of the pro-inflammatory cytokines monocyte chemoattractant protein (MCP)-1, interleukin (IL)-6, tumour necrosis factor (TNF)-α, and interferon (IFN)-γ in a mouse infection model, underscoring the protease's role in the bacterium's virulence (32). Recently, it was found that surface-bound HtrA of C. jejuni, but not secreted HtrA, disrupts epithelial cell-to-cell junctions (33). In B. anthracis, HtrA plays a regulatory role, influencing expression of more than 1,000 genes under stress (34). A B. anthracis ΔhtrA deletion mutant was unable to proliferate in macrophages and impaired their lysis (35). In the context of periodontitis, in P. gingivalis, HtrA was shown to regulate gingipain activity and adaptation to oxidative and long-term heat stress (36, 37). The protease also influenced bacterial survival in a mouse model, and a P. gingivalis ΔhrtA-deletion mutant displayed cell-dependent effects on invasiveness compared to the P. gingivalis W83 parent wild-type strain, with increased invasion of epithelial cells, while invasion of endothelial cells was unaffected (37). These findings suggest a prominent role of HtrA in the interaction of this periodontal pathogen with the host.       In this study, we obtained evidence that T. forsythia ATCC 43037 produces a proteolytically active HtrA ortholog. We identified the htrA gene in T. forsythia ATCC 43037, produced the predicted HtrA recombinantly in E. coli cells, and demonstrated proteolytic activity of the T. forsythia enzyme on different substrates in vitro. Furthermore, we constructed a T. forsythia ΔhtrA deletion mutant. Using this mutant along with the T. forsythia parent strain and recombinant HtrA, we conducted infection studies to assess the enzyme’s effect on cellular viability and pro-inflammatory cytokine production in human gingival fibroblasts and macrophages.\"}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11255850&rettype=xml\n",
            "Failed to fetch data for PMCID 11255850. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11253639&rettype=xml\n",
            "Failed to fetch data for PMCID 11253639. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11252519&rettype=xml\n",
            "Failed to fetch data for PMCID 11252519. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11251922&rettype=xml\n",
            "Failed to fetch data for PMCID 11251922. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11249912&rettype=xml\n",
            "{'pmcid': '11249912', 'doi': '10.34133/research.0415', 'introduction': 'N/A'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11246856&rettype=xml\n",
            "{'pmcid': '11246856', 'doi': '10.3389/fnut.2024.1407736', 'introduction': '1       Introduction       Helicobacter pylori (H. pylori) infection is a global health problem, resulting in chronic inflammation and the increased risk of peptic ulcer disease, atrophic gastritis, gastric carcinoma, and mucosa-associated lymphoid tissue (MALT) lymphoma (1). The global pooled prevalence of H. pylori infection has been reported to be 48.5%, ranging from 70.1% in Africa to 24.4% in Oceania (2). A recent meta-analysis reported that the prevalence of H. pylori infection in mainland China was 40.0% during the period 2015–2019, highlighting China as one of the countries with a substantial burden of H. pylori infection, despite a declining trend from 2000 to 2019 (3). Besides, considering the high resistance rates of clarithromycin and metronidazole (4), bismuth quadruple therapy is recommended as the first-line treatment in China (5), including 2 antibiotics (with five different regimens), a standard dose of a proton pump inhibitor (PPI), and bismuth (6). Both antibiotics and PPIs are known to disrupt the gut bacterial equilibrium, and the bacterial shifts caused by eradication therapy may contribute to the disturbance of carbohydrate and lipid metabolism (7). Consequently, it is necessary to develop a therapeutic regimen for H. pylori infection that not only achieves a satisfactory eradication rate but also minimizes the impact on the gut microbiota.       Fucoidan is a complex sulfated polysaccharide extracted from various species of brown seaweed (8). There is a growing body of evidence suggesting that fucoidan possesses various physiological properties, including high antioxidant activity (9), anti-tumor effect (10), and anti-allergic property (11). Besides, fucoidan can exert anti-inflammatory effects by modulating neutrophil migration and the expression of genes related to COX-2 and the NF-κB signaling pathway (12). Therefore, fucoidan is regarded as a promising component in functional food or adjuvant therapy. Fucoidan has two primary structural backbones: Type I, such as fucoidan from Laminaria japonica, is composed of repeated α (1\\u2009→\\u20093)-linked α-L-fucopyranose, with fucose-linked sulfate groups in C2- and C4- positions; Type II, such as fucoidan from Ascophyllum nodosum, is composed of alternate α (1\\u2009→\\u20093)- and α (1\\u2009→\\u20094)-linked α-L-fucopyranose, with fucose-linked sulfate groups in C2-, C3-, and C4-positions (13). The sulfate groups are important determinants for the bioactivities of fucoidan, depending on the sulfate content and position of the groups (14).       Research in a rat model of breast cancer has indicated that fucoidan intervention can enhance the diversity of the intestinal microbiota and change the microbiota composition, including a decrease in the abundance of Collinsella, Coprococcus, Oscillospira, and Sporosarcina as well as an increase in the abundance of Parabacteroides and Blautia (15). In an ulcerative colitis mice model, fucoidan was also found to significantly increase the abundance of Ruminococcaceae and Clostridium-IV (16). Moreover, fucoidan has been demonstrated to increase several beneficial bacterial taxa and decrease several harmful taxa in mice intervened with ciprofloxacin-metronidazole (17), indicating a potential capacity to mitigate dysbiosis caused by antibiotics, which is common during the eradication therapy of H. pylori infection. Nevertheless, data on the effects of fucoidan on the human gut microbiota are limited. In patients with chronic superficial gastritis, a combination of wheat peptides and fucoidan was reported to relieve symptoms, reduce gastric mucosal damage, and regulate gut microbiome, promoting the growth of Bifidobacterium pseudocatenulatum, Eubacterium siraeum, and Bacteroides intestinali (18), but the isolated effects of fucoidan were not evaluated.       Both in vitro and in vivo studies have shown that fucoidan can prohibit H. pylori adhesion to host cells, reduce cytotoxic effects in H. pylori, and decrease the infection rate of H. pylori in a dose-dependent manner (19). However, clinical data regarding the effects of fucoidan on H. pylori infection remains limited. Thus, we conducted a prospective, randomized, open-label trial in patients with H. pylori infection. The aim was to assess the effects of fucoidan supplementation, either alone or in combination with synbiotics supplementation, on the eradication rate and the maintenance of gut microbiota homeostasis in these patients.'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11244529&rettype=xml\n",
            "{'pmcid': '11244529', 'doi': '10.1126/sciadv.adn1745', 'introduction': 'N/A'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11243892&rettype=xml\n",
            "{'pmcid': '11243892', 'doi': '10.3390/s24134266', 'introduction': '1. Introduction       Nucleic acid amplification tests (NAATs) are key tools for the screening, risk assessment, diagnosis, therapy selection, prognosis, and monitoring of many diseases [1]. With the recent COVID-19 pandemic, the need for rapid and accurate nucleic acid testing has been shown in recent years [2,3,4]. While conventional real-time polymerase chain reaction (PCR) [5] methods have demonstrated tremendous success for qualitative analysis as well as semi-quantitative analysis, achieving the absolute quantification of target sequences is challenging. The impact of sample matrices and PCR inhibitors often undermines the reliability of copy number quantification. To overcome this limitation, digital PCR was developed, in which the bulk volume is divided into many smaller portions. Each of these portions containing either 0 or 1 copy of the target sequence is then amplified and read individually to determine the “negative” and “positive” portions [6,7,8]. Absolute quantification is then achieved based on the Poisson distribution. To date, digital PCR has been used in diverse applications, including pathogen detection, rare sequence detection, and liquid biopsy.       Despite the tremendous success in laboratory-based tests, developing point-of-care (POC) digital PCR technology has been challenging. To simplify the amplification process and eliminate the need for a thermal cycler, many isothermal amplification methods have been used for digital amplification. Notably, loop-mediated isothermal amplification (LAMP) [9] is the most widely performed reaction due to the high specificity and sensitivity of using loop structures to encourage the amplification of the target. To date, many digital LAMP (dLAMP) methods have been reported depending on different mechanisms for achieving sample compartmentalization. The first class of methods utilizes pre-formed or prefabricated structures for distributing bulk samples into small volumes. The use of gel arrays with LAMP reagents embedded inside for amplification has been reported as a possible method for dLAMP [10,11,12,13]. Similarly, dLAMP has been performed in commercially available membranes using the pores as a microwell array [14,15,16]. In addition, slipchip technology [17,18,19,20,21,22,23] and self-digitizing microfluidic chips [24,25,26] have been utilized to integrate additional fluid operations to distribute the bulk solution for dLAMP. In addition to these microwell-based dLAMP methods, water-in-oil emulsions (droplets) have also been used for dLAMP due to their advantage of generating a much larger number of individual amplifications with lower cost, termed droplet digital LAMP (ddLAMP). Microfluidic flow focusing and T-junction are the most used methods for generating monodisperse droplets for ddLAMP [27,28,29,30]. Although microfluidic-based methods work well for droplet generation, they often require multiple fluid pumps and microfabricated devices, making them difficult to use in POC systems. Non-microfluidic strategies, including centrifugal force-based [31,32,33,34,35], inkjet printing [36,37,38,39], and electrowetting methods [40], have also been reported to simplify the droplet generation process. However, sophisticated external equipment is still required for operation.       In recent years, droplet generation methods that are simple and portable have been reported. Yuan and colleagues reported a ddLAMP method utilizing a hand-powered droplet generator [41], in which manual syringe-based suction pumping is used to form droplets in a microfluidic chip. The droplets are collected in a container and heated with a water bath, and then they are transferred to a well plate for fluorescence imaging. The authors reported a limit of detection of 10 copies per microliter with the system that can be used in POC; however, the size of the droplets cannot be tuned, so only one average droplet size is possible. Similarly, Wu and colleagues reported a ddLAMP method utilizing microfluidic step-emulsion of droplets through the manual pumping of a syringe [42], in which the bacterial DNA targets were distributed into a chip, amplified using a water bath, and underwent fluorescence imaging for duplex detection. The authors reported a limit of detection of 19.8 copies per microliter. Mao and colleagues reported an open droplet array consisting of hydrophilic and hydrophobic patterning, which was sealed with mineral oil. The chip was then incubated, and smartphone-based fluorescent imaging and app-based analysis were performed [43]. The authors reported a limit of detection of 1 copy of lambda DNA per microliter; however, the method is limited by fixed dimensions of the pattern. Although these simplified droplet generation methods are more POC-friendly, the performance of these methods in terms of throughput, monodispersity, and droplet size flexibility is compromised due to the lack of sophisticated control equipment, which leads to reduced analytical performance for digital assays.       Recently, our lab reported a simple and compact droplet generation method based on a vibrating sharp-tip capillary [44]. This system eliminates the need for external pumps and achieves on-demand droplet generation with a portable signal generation, which shows great potential for POC digital nucleic acid tests. In this work, we combine the vibrating sharp-tip capillary droplet generation with LAMP to achieve a simple digital droplet assay for detecting bacteria pathogen sequences. We generated a large range of stable droplet sizes (~0.7 pL to 22 nL) with a single device. We optimized the conditions for the digital LAMP reactions and employed a simple DNA fragmentation method to ensure uniform sample distribution throughout the experiments. Finally, we detected common waterborne pathogens, Vibrio cholerae and Shigella sonnei, with a linear dynamic range of approximately 2 to 6000 copies/μL. This method eliminates the need for bulky equipment for generating monodisperse droplets and a thermal cycler, which paves the way for a portable and high dynamic range ddLAMP system.'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11243103&rettype=xml\n",
            "{'pmcid': '11243103', 'doi': '10.3390/nu16131988', 'introduction': '1. Introduction       Autism spectrum disorder (ASD) is a behaviorally defined neurodevelopmental disorder. ASD lacks specific clinical biomarkers and has seen an evolving conceptualization through the decades since it was first described. Over the past four decades, there has been dramatic increase in the number of individuals diagnosed with ASD [1]. In general, ASD is diagnosed by 3 years of age in most of those children experiencing it, although roughly 40% of such children are not first evaluated until 4 years of age [2]. A psychiatric diagnosis of ASD, which has behavior as its basis of definition, relies heavily upon precise observation and clinical expertise because the condition lacks standardized biomarkers [3].       ASD is one of the most common and challenging neurodevelopmental disorders in children. Its prevalence rate worldwide now exceeds 1%. A small number of these children appear to develop normally in their first year and then go through a period of regression between 18 and 24 months of age. ASD is characterized by deficits in communication and social interaction, as well as a presence of repetitive and restrictive behaviors. Moreover, ASD often manifests with a wide range of comorbidities that include morphological, physiological, and psychiatric conditions. ASD’s most commonly proposed causes are physiological and metabolic disorders involving immunity, oxidative stress, and mitochondrial dysfunction [4]. Co-occurrence of two or more disorders in the same individual has been observed, with comorbidities including anxiety, depression, attention deficit/hyperactivity disorder (AD/HD), epilepsy, gastrointestinal symptoms/problems, sleep disorders, learning disabilities, obsessive–compulsive disorder, intellectual disability, sensory problems, and immune disorders. The most prevalent comorbidity, at roughly 50%, is intellectual disability [5]. At least one comorbidity exists in about 70% of children with ASD, while 41% have two or more [6]. An estimated 20% of individuals diagnosed with ASD also have epilepsy [7].       Although ASD’s etiology remains largely unexplained, a recent finding identifies specific gut microbiota composition in ASD patients. Post-mortem examination of ASD subjects’ brain tissue and small intestines has revealed that the blood–brain barrier and gut barrier were disrupted, with significant neuroinflammation evidenced by increased expression of genes and markers associated with brain inflammation. It has further been inferred that the gut–brain axis disruption may be associated with non-self antigens that trigger a neuroinflammatory reaction by crossing the damaged gut barriers, thus leading to ASD in genetically susceptible subjects [8]. In a study involving 192 twins, however, genetic factors accounted for only 38% of ASD risk, whereas the remaining 68% was attributed to environmental factors [9]. A significant role was ascribed to the gut microbiota [10]. Microbiota is shaped by diet, lifestyle, and microbial exposure in the early developmental phase and infection, as well as by genetic makeup, metabolites, and immunological and hormonal aspects [11].       Analysis of gut microbiota is currently a growing area of research linked to neuropsychological disorders, including depression [12], metabolic disorders such as obesity [13], and gastrointestinal disorders, including inflammatory bowel disease or irritable bowel syndrome [14,15]. Many studies have identified that microbiota composition in ASD patients differs significantly from that in healthy controls [16,17,18,19]. Nutritional intervention, prebiotics, probiotics, and symbiotics, including fecal microbiota transplantation as a remedy to modulate the species composition of the intestinal microbiota of patients with ASD, already have been tested [20,21,22,23,24]. These studies generally have concluded, however, that their findings should be taken with caution because there still exist only limited data from studies examining different regimens of different remedy applications and there have not yet been double-blind studies demonstrating clinical significance of the effects of those remedies used. Here, therefore, we present the results of a double-blind, placebo-controlled study utilizing the biological response modifier Juvenil for influencing the gut microbiota of children suffering from ASD and for modulating their ASD symptoms.'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11242183&rettype=xml\n",
            "{'pmcid': '11242183', 'doi': '10.3390/ijms25137375', 'introduction': '1. Introduction       The genus Plesiomonas, represented by a single species, Plesiomonas shigelloides, is a Gram-negative bacillus associated with gastrointestinal and extraintestinal diseases in humans [1], such as an invasive shigellosis-like disease, cholera-like illness, pseudoappendicitis, meningitis, acute secretory gastroenteritis, and bacteremia [2,3,4,5,6,7]. Many pathogenic bacterial species express flagella on their surfaces, which are their primary means of locomotion or motility, and flagellar motility is necessary for them to reach the site of infection, which is the initial step in the establishment of a bacterial infection [8]. Meanwhile, flagellum motility provides a significant advantage for bacteria to move toward favorable conditions or away from detrimental environments and to effectively compete with other microorganisms [9].       The flagellum consists of three parts: the basal body, hook, and filament [10]. Flagellar motility, an energetically expensive process, requires the coordinated expression of more than 50 genes encoding regulatory proteins, structural components of the secretion and assembly apparatus, and chemo-sensor machinery [8]. The coordinated expression of these promoters clusters gene transcription at three or four levels of hierarchy: classes I to III or I to IV [11]. Peritrichous flagellated bacteria, such as Salmonella and Escherichia coli, have three levels of hierarchy, and transcription of class I and II genes requires the housekeeping sigma factor 70 (σ70) [12]. However, some bacteria, such as Vibrio cholerae [13,14,15], Vibrio parahaemolyticus [16], and Aeromonas hydrophila [11,17], have four levels of flagella regulatory hierarchy. Furthermore, several of the aforementioned bacteria contain both polar and lateral flagella. Bacteria with polar flagellation exhibit four transcriptional levels: class I promoters encode a σ54-dependent activator (FlrA) and activate class II σ54-dependent promoters (FlrBC), which encode a two-component signal-transducing system whose regulator activates class III σ54-dependent promoters [13,14,15,16,17]. Moreover, class II promoters also encode the σ28 factor (FliA), which activates the transcription of class IV genes. The lateral flagellar system, such as that of V. parahaemolyticus and A. hydrophila, which have three levels of flagella regulatory hierarchy, is encoded by more than 30 lateral flagella genes and regulated by the LafK master regulator [18]. Furthermore, LafK may compensate for FlaK’s role in enabling swimming in some bacteria [19]. In the expression of the lateral flagellar system, the master regulator LafK activates the class II genes, and then the expression of class III genes is activated by σ28, which is encoded by fliAL [16,20,21]. In a word, in a spatiotemporal-dependent manner, the expression of flagellar genes is strictly regulated by the respective regulators and sigma factors.       The primary function of flagella is to provide motility to bacteria; nevertheless, they are also related to bacterial pathogenicity. Flagellum-mediated motility plays a critical role in interactions between Vibrio species and their hosts, and these bacteria colonization and lethality in their respective hosts decrease when motility is restricted [22]. Vibrios have therefore been shown to be valuable models for researching flagellar control and its role in colonization. Motility has been identified as a virulence determinant of V. cholerae. Additionally, non-motile mutants in competition assays using an infant mouse model system show no significant defect in their capacity to colonize the small intestine [23]. A Na+ gradient across the membrane drives V. cholerae flagellar rotation while also regulating transcription of the toxT gene, which is necessary for CT and TCP expression, revealing a possible mechanism for connecting virulence factor expression to motility [24,25]. Coster and Kenner found that the human reactogenicity of live attenuated V. cholerae vaccines is also lowered in non-motile mutants [26]. In addition to being more frequently documented in V. cholerae, the correlation between flagella and virulence has also been investigated in other bacteria. Flagella motility, like that of V. cholerae, is critical in the pathogenesis of V. campbellii, and inhibiting motility greatly reduces host mortality during infection [27]. Previous studies have also reported the synergistic action of SPI-1 gene expression in Salmonella enterica serovar typhimurium through transcriptional cross-talk with the flagellar system [28]. Zhao et al. found that the type III secretion system intersects with the lateral flagellar system in A. hydrophila [29]. In a burned mouse model of infection, non-flagellated mutants of P. aeruginosa have been demonstrated to exhibit reduced penetration of cultured corneal epithelial cells and to be deficient in pathogenicity [30,31]. As a result, the significance of flagella in pathogenicity is obvious.       In addition to granting motility to bacteria and influencing adherence and virulence, flagellar regulators and flagellar genes are also involved in the physiological metabolism of bacteria. The relationship between the ability to survive periods of acid shock and the fine-tuned expression of motility genes and fitness is currently unknown; however, Schumacher et al. found an inverse correlation between the expression of FlhDC, the master regulator of flagellation, and acid shock survival in E. coli [32]. According to Hoque et al., the ∆flrA mutation in V. cholerae increases iron utilization and oxidative stress tolerance since it significantly upregulates most of the genes involved in heme absorption, siderophore uptake, and vibriobactin uptake [33]. Sinha-Ray et al. found that a mutation in V. cholerae’s flrA gene is inversely involved in the vps-independent biofilm driving bacteria toward nutrients in lake water [34]. Rodríguez-Herva et al. suggested that in addition to a number of motility and chemotaxis genes, the fliA gene product is also necessary for the expression of some genes potentially involved in amino acid utilization or stress responses [35]. Li et al. reported that flagellum hook-associated protein (FlgK) increases the resilience of C. sakazakii to dehydration and regulates formate dehydrogenase and nitrate reductase [36]. It has also been reported in studies that FliZ plays a key role in controlling lipase and hemolysin activities [37,38].       P. shigelloides is a special member of the Enterobacteriaceae family that can produce both polar and lateral flagella, and Merino et al. showed that P. shigelloides has two distinct gene clusters: one for lateral flagella biosynthesis and another for biosynthetic polar flagella [39]. Merino and co-workers reported that although P. shigelloides lacks a FlrB ortholog, P. shigelloides FlrC (FlaM) contains the PAS and His Kinase A domains found in the FlrC proteins of Vibrio species and A. hydrophila, indicating that P. shigelloides FlaM may activate transcription from σ54-dependent promoters of class III genes [39]. Furthermore, the P. shigelloides polar flagella gene regions show greater similarity to those reported in Vibrio or Aeromonas than the regions in Enterobacteriaceae [40]. Though a number of studies have implicated motility as being important for bacteria, the transcriptional regulation network of polar and lateral flagella in P. shigelloides has yet to be revealed. In this work, we identified FlaK, FlaM, FliA, and FliAL as the four regulators responsible for polar and lateral flagellar regulation in P. shigelloides. Through transcriptome sequencing analysis, we confirmed the polar and lateral flagellar regulatory hierarchy and further revealed the cross-talk between the flagellar hierarchy system and P. shigelloides’ virulence and physiological metabolism.'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11241808&rettype=xml\n",
            "Failed to fetch data for PMCID 11241808. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11240908&rettype=xml\n",
            "Failed to fetch data for PMCID 11240908. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11240787&rettype=xml\n",
            "{'pmcid': '11240787', 'doi': '10.3390/ani14131978', 'introduction': '1. Introduction       Currently, neonatal porcine diarrhea caused by bacteria is the main problem in some large-scale breeding farms free of enteroviruses [1], with diarrhea induced by E. coli being the most common form [2,3,4]. This disease causes high mortality or low weight gain in suckling piglets, leading to severe economic losses for the pig industry [5,6,7]. Reducing the pathogen load of bacteria in the environment and the sow body surface through disinfection, washing, and enhancing the immunity of piglets against pathogens through vaccination are the main prevention and control measures [8,9]. However, it is difficult to synthesize a strain-specific vaccine with efficient cross-protection because of the serotype diversity of E. coli strains [1,10]. Therefore, sows are often fed a homogenate of intestinal tissues and contents from piglets who succumbed to diarrhea to stimulate their immune system and pass specific antibodies to suckling piglets through colostrum and milk when E. coli-induced diarrhea occurs in pig farms, which is also known as feedback feeding [9,11,12]. However, the individual differences in the immune response of the same batch of sows to feedback feeding make it impossible to achieve consistent results from this procedure.       The gastrointestinal tract is one of the largest immunological organs of the body [13]. Feedback feeding is a kind of gastrointestinal mucosal immune strategy. Mucosal immunity provides the first line of defense against exogenous pathogens by secreting SIgA and IgM to inhibit surface colonization by microorganisms [13,14,15]. B cells in mucosa-associated lymphoid tissue are the cells that give rise to secretory IgA and IgM. During the perinatal period, in sows, maternal IgA-secreting cells migrate to the mammary gland to increase the levels of maternal antibodies in milk [15]. The authors of a previous study verified the involvement of intestinal microorganisms in the number of IgA plasma cells in the mammary glands (but not the small and large intestines), as well as the production of maternal IgA in milk [16].       B. subtilis is a kind of probiotic with excellent properties that can be widely used as a feed additive in livestock and poultry farming [17,18,19]. It has been shown that B. subtilis can increase the levels of IgG, IgM, and SIgA in serum or milk [20,21]. In this study, we aimed to enhance the immune response of sows against E. coli by combining B. subtilis with feedback feeding to increase the levels of specific maternal antibodies to improve intestinal function and reduce E. coli-caused diarrhea in suckling piglets. Our study provides a new resolution for the prevention and control of E. coli-caused diarrhea in suckling piglets.'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11240761&rettype=xml\n",
            "Failed to fetch data for PMCID 11240761. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11240451&rettype=xml\n",
            "{'pmcid': '11240451', 'doi': '10.3390/ani14131959', 'introduction': '1. Introduction       Chlamydial infections caused by widespread obligate intracellular Gram-negative bacteria from the Chlamydiaceae family are associated with a wide spectrum of clinical symptoms in livestock, companion animals, wildlife, and various exotic species including several birds species [1,2,3]. In addition, some chlamydial species, such as Chlamydia abortus and Chlamydia psittaci, are capable of causing zoonotic infections [4,5].        In humans, Chlamydia psittaci (C. psitacci) causes a severe disease, known as psittacosis, which, in most cases, can lead to pneumonia and significant mortality if left untreated [6,7,8,9]. Psittacosisis is a significant threat to public health and is considered an occupational zoonosis of pet shop owners, zookeepers, veterinarians, abattoir workers, bird breeders, and pet bird enthusiasts [2,10,11].       At least 465 bird species belonging to 30 orders can have chlamydiosis. However, psittacidae (lories, parakeets, parrots, and cockatoos) and columbiformes (pigeons) are the most frequently affected. Primary reservoirs for zoonotic psittacosis are psittacine pets, as well as wild and racing pigeons [12].       C. psittaci replicates using a biphasic developmental cycle within cytoplasmic inclusion vacuoles of eukaryotic cells. This cycle involves two morphologically different forms, as follows: the infectious extracellular form, termed the elementary body (EB), and the reproductive reticulated body (RB). The EB enters a host cell through receptor-mediated endocytosis. This process, which lasts several hours, results in the conversion of the EB into a reticulate body (RB). The RB divides via binary fission to produce more EBs. This cycle continues for several hours after infection, at which time the host cell lyses, releasing several hundred EBs, RBs, and intermediates [13].       Chlamydiosis causes a range of clinical signs in birds, including respiratory difficulty, diarrhea, and lethargy, and it can also lead to chronic or asymptomatic infections. Some infected birds may appear healthy, but shed the organism intermittently. Shedding may be exacerbated by internal factors such as the bird’s immune status, in addition to external factors such as stress, reproductive activities, relocation, shipping, crowding, and cold temperature. The severity of clinical signs varies significantly, influenced by the species and age of the birds, as well as the specific strains of the pathogen involved [2].       In psittacine birds, gross lesions include splenomegaly, hepatomegaly, enteritis, sinusitis, air sacculitis, pericarditis, and conjunctivitis. Microscopic lesions of chlamydiosis in birds include necrotizing changes in several organs associated with chlamydial intracytoplasmic elementary bodies and the infiltration of heterophils, macrophages, and plasma cells [14,15,16].       Chlamydia psittaci infection in birds has been reported previously in many countries [1,17,18,19,20]. In recent years, there has been a global increase in the number of reported cases of avian chlamydiosis in both birds and humans [9,21,22,23].       The analysis of C. psittaci ompA gene sequences is crucial for genotype determination [24]. In the studied psittacidae population, the circulating genotypes are identified as A and B. Initially, C. psittaci was categorized into nine genotypes—A-F, E/B, M56, and WC, with the later proposal of eight new genotypes [1,24]. Genotype A is prevalent among psittacine birds, but is also found in turkeys, ducks, pigeons, and Passeriformes. Genotype B is endemic in pigeons but can infect various bird species including chickens, turkeys, ducks, Psittacidae, and Passeriformes. Waterfowl are commonly infected with genotype C and E/B strains, while turkeys can harbor genotypes D and C. Genotype E is found in ducks, pigeons, ostriches, and rheas, and genotype F is in psittacine and turkey isolates [1,25,26,27].       This article describes a large outbreak of psittacosis in a commercial breeding aviary in Buenos Aires, Argentina, and emphasizes the need to monitor and quickly identify the disease to minimize its impact in birds and prevent the spread to people at risk.'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11239581&rettype=xml\n",
            "Failed to fetch data for PMCID 11239581. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11238125&rettype=xml\n",
            "{'pmcid': '11238125', 'doi': '10.1016/j.heliyon.2024.e33152', 'introduction': 'N/A'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11237755&rettype=xml\n",
            "Failed to fetch data for PMCID 11237755. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11237395&rettype=xml\n",
            "Failed to fetch data for PMCID 11237395. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11343591&rettype=xml\n",
            "{'pmcid': '11343591', 'doi': '10.1172/jci.insight.174776', 'introduction': 'Introduction       Axial spondyloarthritis (axSpA) is a chronic inflammatory disorder that damages the spine, with occasional involvement of peripheral joints and tendons (1). The greatest genetic risk factor for both radiographic axSpA (r-axSpA) and nonradiographic axSpA (nr-axSpA) is the HLA-B*27 allele; more than 80% of patients are HLA-B*27+ compared with less than 10% of the general population (2–4). Many patients with axSpA also have acute anterior uveitis (AAU), and HLA-B*27 increases the risk for developing isolated AAU; 50% of individuals with AAU are HLA-B27+ (5, 6). HLA-B*27–associated AAU (B27AAU) presents as sudden onset, unilateral ocular inflammation, with resolution after several weeks (7, 8). Patients with B27AAU are also at high risk (55%–90%) of developing axSpA (6, 9). The shared genetic risk allele of HLA-B*27 and clinical overlap of axSpA and B27AAU suggest a common pathophysiologic mechanism.       HLA-B*27 has been suggested to increase the risk of developing axSpA and B27AAU through various mechanisms. One hypothesis is that HLA-B*27 indirectly causes loss of immune tolerance via its deleterious effect on immune homeostasis and promotion of intestinal inflammation (10–12). Investigators have identified asymptomatic, microscopic inflammation in intestinal biopsies of patients with either r-axSpA or reactive arthritis, the latter being clinically associated with specific intestinal pathogens, including Shigella and Salmonella (13–15). However, which immune cells or microbial products are responsible for transferring enteric inflammation to the joints in axSpA or eye in B27AAU remain unclear (16). Another hypothesis is that HLA-B*27 presents antigens to pathogenic CD8+ T cells in the form of molecular mimicry, whereby CD8+ T cells initially respond to foreign antigens, such as from bacteria, but subsequently are stimulated by self-antigens due to cross-reactivity of their T cell receptors (TCRs) (16). This “arthrogenic peptide” hypothesis is supported by our recent identification of axSpA-associated public TCRs that are enriched in the joint in axSpA and eye of B27AAU and can recognize both bacterial and self-peptides presented by HLA-B*27 (17). Moreover, elimination of these public TCRs via TRBV9-targeted depletion leads to disease improvement, at least in some cases (18, 19). One bacterial peptide recognized by these public TCRs is derived from yeiH, a gene present in multiple enteric microbes. Whether these expanded CD8+ T cells in axSpA and B27AAU have a signature of mucosal immunity remains unknown.       CD161 (KLRB1) is expressed on 2 subsets of T cells that are associated with enteric immunity. CD161hiCD8+ T cells are mucosal-associated invariant T (MAIT) cells that are enriched in enteric organs, such as the liver and intestine; participate in immunity to microbiota; and may play a role in axSpA and B27AAU (20–22). These unconventional T cells recognize 5-OP-RU from bacterial riboflavin biosynthesis in the context of the MHC class I–like molecule MR1 (23). In contrast to MAIT cells, CD8+ T cells expressing intermediate levels of CD161 (CD161int) are conventional polyclonal αβ T cells that recognize foreign peptides presented by canonical MHC molecules (24). Despite this distinction, CD161intCD8+ T cells share several properties with MAIT cells. For instance, CD161intCD8+ T cells are enriched in enteric organs, particularly the colon (24), and contain a distinct transcriptional program that is shared with MAIT cells (25). In addition, CD161 is expressed by a population of CD8+ T cells that establish tissue residency within the small intestine (26). However, whether antigen-specific CD161intCD8+ T cells play a role in axSpA and B27AAU remains unknown.       Here, we report that B27AAU is characterized by an influx YeiH.232-240–specific CD161intCD8+ T cells into the eye. Moreover, these YeiH.232-240–specific CD8+ T cells expressed an intestinal CD8+ T cell transcriptional program, were more abundant in axSpA and B27AAU compared with individuals acting as healthy controls, and expressed surface molecules associated with mucosal trafficking and immunity. Collectively, our data suggest that, in HLA-B27–associated inflammation, the early activation of pathogenic T cells may occur in enteric organs.'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11342760&rettype=xml\n",
            "Failed to fetch data for PMCID 11342760. Status code: 429\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11341846&rettype=xml\n",
            "{'pmcid': '11341846', 'doi': '10.1038/s41598-024-70654-6', 'introduction': 'N/A'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11340806&rettype=xml\n",
            "{'pmcid': '11340806', 'doi': '10.3389/fmicb.2024.1439204', 'introduction': 'N/A'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11340127&rettype=xml\n",
            "{'pmcid': '11340127', 'doi': '10.1186/s13062-024-00517-7', 'introduction': 'N/A'}\n",
            "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=11339456&rettype=xml\n",
            "{'pmcid': '11339456', 'doi': '10.1038/s41598-024-69923-1', 'introduction': 'N/A'}\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "print(records)\n",
        "print(len(records))"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "CST7tQXxORjy",
        "outputId": "8ce1e6be-a22d-4339-a3e6-5577a934e555"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "[{'pmcid': '11342756', 'doi': '10.1016/j.psj.2024.104099', 'introduction': 'N/A'}, {'pmcid': '11342206', 'doi': '10.1016/j.bioactmat.2024.07.032', 'introduction': 'N/A'}, {'pmcid': '11341105', 'doi': '10.7759/cureus.65185', 'introduction': 'Introduction       Pseudomonas aeruginosa is an opportunistic pathogen responsible for various healthcare-related infections [1]. These infections are challenging to treat due to the bacteria\\'s intrinsic and acquired resistance mechanisms [2]. The improper and unnecessary use of β-lactam antibiotics exacerbates bacterial resistance, leading to the selection of extended-spectrum β-lactamases (ESBL), AmpC β-lactamases, and metallo-β-lactamases (MBL), which pose significant challenges in antimicrobial therapy [3]. Pseudomonas aeruginosa, typically susceptible to ceftazidime, aztreonam, and carboxypenicillins, can develop resistance to third-generation cephalosporins through the hyperproduction of AmpC beta-lactamase [2]. AmpC β-lactamases hydrolyze cephalosporin antibiotics and are resistant to clavulanic acid. Unlike ESBLs, AmpC β-lactamases can deactivate a range of antimicrobials, including monobactams, aminopenicillins, cephamycins, and cephalosporins [4]. Both chromosomal and plasmid-mediated genes can produce these enzymes [5].       AmpC β-lactamases are class C enzymes according to the Ambler classification, they have serine residues that facilitate catalysis in their active site [6]. There\\xa0are three types of resistance mechanisms by AmpC β-lactamases can be distinguished: i) inducible resistance\\xa0by chromosomally encoded AmpC genes, accountable for resistance in bacteria like Serratia marcescens, Enterobacter cloacae, Citrobacter freundii, Pseudomonas aeruginosa,\\xa0etc.\\xa0ii) non-inducible chromosomal resistance caused by mutations in the promoter and attenuator\\xa0mechanisms (e.g., Acinetobacter baumannii, Shigella species, and Escherichia coli),\\xa0iii) or resistance mediated by plasmids (e.g., Salmonella species, Klebsiella pneumonia, E. coli, etc.) [7].       Even infections produced by originally susceptible isolates can develop considerable AmpC synthesis and β-lactam resistance as a result of exposure to β-lactams. Treatment considerations are complicated by species and β-lactam variations in the possibility of generating AmpC formation [8].\\xa0Three proteins, AmpG, AmpD, and AmpR, are involved in the regulation of AmpC β-lactamase induction. Mutations in the structural AmpD genes result in hyper-inducibility and overproduction of the AmpC enzyme [9]. The main resistance against β-lactam drugs in Pseudomonas aeruginosa\\xa0involves mutations boosting chromosomal AmpC production, and selecting resistant mutants. Pseudomonas aeruginosa also produces chromosomal cephalosporinases (\\u200b\\u200b\\u200b\\u200b\\u200b[Pseudomonas-derived cephalosporinases (PDC)] with extended-spectrum activity. Plasmid-mediated AmpC emergence adds to the challenge of treating Pseudomonas aeruginosa\\xa0infections [10].       Plasmid-mediated class C enzymes are named according to various criteria. Some names for them include Ambler class C (ACC) or AmpC type (ACT), which are β-lactamase types; cefoxitin (FOX), or cephamycins (CMY), which are antibiotics they resist; and Dhahran Hospital (DHA) or Miriam Hospital (MIR-1), which are the names of the hospitals where they were found. In rare cases, they are even given patient names, like Bilal (BIL-1) [11]. South Korea (CMY-1) is where the first pAmpC was found in 1989. Many other pAmpCs (FOX, CIT, MOX, DHA, EBC, and ACC) have since been characterized; the most frequent subtype is the CMY-2 enzyme (CIT-type) [12].       The study investigated beta-lactamase-mediated resistance in hospital isolates of Pseudomonas aeruginosa. \"The primary objective of this study is to investigate the prevalence and distribution of AmpC beta-lactamase-producing Pseudomonas aeruginosa in a tertiary care hospital setting in Karad, Maharashtra, India.\" \"The secondary objectives include phenotypic and genotypic characterization of AmpC beta-lactamase production and identification of the specific genes (blaPDC and blaCMY) responsible for this resistance.       The prevalence of multidrug-resistant (MDR) infections is rapidly increasing in hospital settings, largely attributed to the extensive utilization of broad-spectrum cephalosporins, which hampers the efficacy of control measures. In this regard, this study aimed to explore the production of AmpC β-lactamase in the strains of Pseudomonas aeruginosa\\xa0isolated from a tertiary care hospital in Karad, Maharashtra, India.'}, {'pmcid': '11340603', 'doi': '10.1016/j.csbj.2024.07.017', 'introduction': 'N/A'}, {'pmcid': '11340327', 'doi': '10.1128/jb.00049-24', 'introduction': \"INTRODUCTION       The gram-negative bacterial cell envelope confers protection from a variety of toxic agents, many of which are encountered by Escherichia coli and other enteric bacteria in their natural environments, such as solvents, acid, detergents, bile salts, and antibiotics (1). The envelope is also important for cellular responses to external stresses with alterations in membrane lipid and membrane protein composition playing critical roles in maintaining physiological homeostasis. Small RNAs (sRNAs) are well-characterized regulators that participate in maintaining cell envelope homeostasis through modulation of membrane protein synthesis (2–7). Several sRNAs are integral parts of important stress regulons, including the σE, Cpx, and OmpR responses. The transcription factor σE maintains outer membrane homeostasis by inducing the transcription of ~100 genes, comprised mainly of genes encoding proteins involved in membrane repair as well as three sRNAs, RybB, MicA, and MicL (3–5). RybB and MicA downregulate the synthesis of abundant membrane porins, including OmpA, OmpC, OmpD, and OmpF, upon entry into stationary phase (4, 5, 8, 9). MicL represses the translation of outer membrane lipoprotein Lpp, a highly abundant protein (3). Together, MicA, RybB, and MicL repress the synthesis of all of the most abundant outer membrane proteins in response to stress (3). While σE is responsible for maintaining homeostasis of the outer membrane, the Cpx response monitors inner membrane integrity (10). The Cpx response controls the production of an sRNA called CpxQ, which is processed from the 3' UTR of cpxP mRNA (2). Synthesis of CpxQ is triggered by membrane-damaging stresses, such as excessive synthesis of membrane proteins, alkaline pH, ethanol, and cationic antimicrobial peptides (2). Upon induction, CpxQ represses the production of numerous inner membrane proteins (2). Both CpxQ and the Cpx pathways are required for cell survival during conditions that lead to the dissipation of membrane potential (2). Lastly, the sRNAs OmrA and OmrB are induced by high osmolarity and regulated by the OmpR response regulator (6). Most mRNA targets are shared between OmrA and OmrB and encode outer membrane proteins. For example, OmrA and OmrB repress the translation of ompT, which encodes an outer membrane protease, along with cirA, fecA, and fepA, all encoding different outer membrane porins specific for iron–siderophore complexes (6).       Importantly, while changes in the structure of the fatty acids in phospholipids can alter critical membrane properties such as fluidity and permeability (1), the roles of sRNAs in modulating membrane lipids are just beginning to be explored. RydC is the first sRNA shown to regulate membrane fatty acid composition and plays a role in the alteration of cell membrane properties and resistance to stress (11, 12). RydC is a 65-nt sRNA that requires the RNA chaperone Hfq for stability and base-pairing interactions with target mRNAs (11, 13–15). Like other Hfq-dependent sRNAs, RydC base pairing with target mRNAs has different outcomes depending on the site of interaction. RydC’s best-characterized target is cfa mRNA, which codes for cyclopropane fatty acid (CFA) synthase, an enzyme that converts unsaturated fatty acids to cyclopropanated fatty acids (11, 12, 16). RydC base pairs at a site within the long 5' untranslated region (UTR) of cfa mRNA and modulates RNase E-dependent degradation and Rho-dependent transcription termination (11, 17). We showed previously that RydC-mediated activation of cfa promotes survival during acid stress (12). In addition to regulating fatty acid modification through activation of cfa, RydC has also been shown to impact mRNAs involved in sugar utilization (yejABEF), biofilm formation (csgD), and aromatic amino acid biosynthesis (pheA and trpE) (13, 15, 18).       Previous work identified two additional sRNAs, ArrS and CpxQ, that also directly regulate cfa (12). Like RydC, ArrS acts by reducing cfa mRNA turnover, leading to increased translation of cfa mRNA (12). In contrast, CpxQ binds cfa mRNA upstream from the site bound by activating sRNAs. We recently showed that like RydC, ArrS and CpxQ also regulate cfa through the modulation of Rho-dependent transcription termination in the long cfa 5' UTR (17). While the direct signals responsible for promoting sRNA regulation of cfa are not currently known, both CpxQ and ArrS were previously linked to cell envelope (2) and acid stress responses (19), respectively.       To better understand the physiological function of RydC and its role in cellular homeostatic or stress response pathways through cfa regulation, we sought to identify regulators that control RydC production. A candidate-based approach revealed that none of the known membrane stress-responsive transcription factors regulate rydC transcription. Instead, a transposon screen identified a GntR-like regulator, YieP, as a repressor of RydC. We showed that YieP auto-activates its own transcription and that the sRNA regulator FnrS represses yieP mRNA translation. Additionally, we demonstrate that the osmotic stress-responsive sRNA OmrB inhibits RydC activity on cfa, suggesting that OmrB acts as a sponge for RydC. Collectively, these results demonstrate that RydC is regulated directly at both transcriptional and post-transcriptional levels within a regulatory network associated with membrane stress signals.\"}, {'pmcid': '11326835', 'doi': '10.1093/infdis/jiad585', 'introduction': 'N/A'}, {'pmcid': '11324020', 'doi': '10.1128/msphere.00060-24', 'introduction': 'INTRODUCTION       Enterobacter cloacae are saprophytic environmental bacteria and are part of the human gut microbiota (1). This Gram-negative bacterium is an important opportunistic pathogen in healthcare-associated infections (HAIs) in patients hospitalized in the intensive care unit. These infections encompass the lower respiratory tract, urinary tract, and meninges (2). E. cloacae virulence factors associated with biofilm formation and cytotoxic activity on eukaryotic cells have been described (3–6).       One of the virulence factors recently studied in E. cloacae is the type VI secretion system (T6SS). This nanomachine was discovered in 2006 as a complex macromolecular apparatus found in more than 25% of sequenced genomes of Gram-negative bacteria (7–9). At the molecular level, the T6SS is assembled via interactions between 13 canonical proteins, called Tss proteins. Several of these proteins share structural homologies with components of the T4 bacteriophage contractile tail, such as the major tail tube protein (Hcp), the cell puncturing device (VgrG), the sheath (TssB-C), and at least one component of the baseplate (TssE) (8, 10, 11). The genomic analysis of E. cloacae ATCC 13047 revealed that it possesses two T6SS clusters (2), which our group named T6SS-1 and T6SS-2 (12). The functional characterization of both systems showed that these two T6SSs are not expressed under the same environmental conditions (12). However, the regulation of the expression of these T6SSs in E. cloacae has not been studied.       The biogenesis and function of T6SS are paramount as it is energetically costly to bacterial cells, necessitating tight control over its gene expression to adapt its expression and assembly to changing environmental conditions. A plethora of environment modulators and regulatory systems have been reported in bacteria to control the transcription of T6SSs either directly or indirectly. Furthermore, beyond the transcription mechanisms, several T6SSs are post-translationally activated by a threonine phosphorylation pathway in response to cell damage or envelope stress.       Nucleoid-associated proteins (NAPs) are abundant proteins in bacterial cells involved in many important cellular processes such as genome architecture, physiology, metabolism, stress response, and virulence (13–15). One of the first NAPs to be described in detail was HU (Histone-like protein from Escherichia coli strain U93) (16). In bacteria that belong to the Enterobacteriaceae and Vibrionaceae families (13, 17), HU, the histone-like protein most abundant on the bacterial nucleoid, is a heterodimer formed of two subunits, HupA (HUα) and HupB (HUβ) (18). HU protein has three naturally occurring forms: the HUα2 and HUβ2 homodimers and the HUαβ heterodimer. The heterodimer binds DNA in a non-specific manner, contributing to DNA flexibility by bending the duplex, which is essential for the structural integrity of the chromosome (19–24) and regulation of virulence factors in several enterobacterial species (25–28). However, two genome analyses identified HU protein-binding motifs revealing A/T- and T/G-rich sequences in E. coli and Francisella tularensis, respectively (29, 30).       Currently, the role of the HU protein as a global regulator in the biology of E. cloacae is unknown. Nevertheless, in its close relative, E. coli HU regulates the expression of 353 genes that respond to anaerobiosis, acid stress, high osmolarity, and SOS response (17, 31).       In this study, we demonstrated that HU positively regulates the T6SS gene expression in E. cloacae by binding directly to their promoter regions in E. cloacae. Deleting either hupA or hupB genes results in a significantly reduced transcription of gene clusters encoding the T6SS-1 and T6SS-2. Consistently, virulence phenotypes such as inter-bacterial competition, biofilm formation, and adherence to epithelial cells were severely affected in the hupA or hupB mutants but restored when hupA or hupB genes were overexpressed in trans from plasmids. To our knowledge, this is the first study that addresses the transcriptional regulation of the T6SS gene cluster, in which HU acts as a direct activator, turning on the T6SS-associated phenotypes.'}, {'pmcid': '11323402', 'doi': '10.1186/s40001-024-02007-y', 'introduction': 'N/A'}, {'pmcid': '11322368', 'doi': '10.3389/fmicb.2024.1437660', 'introduction': \"1       Introduction       Vibrio parahaemolyticus is one of the most important zoonotic pathogens and is mainly associated with the consumption of raw or undercooked contaminated aquatic products (Campbell et al., 2022; Yang et al., 2022). Antimicrobial agents are widely used for the prevention and treatment of bacterial infections in the fields of agriculture, aquaculture, and clinical medicine, and their main mechanisms are as follows: (I) inhibition of cell wall synthesis (such as β-lactams); (II) inhibition of protein synthesis (such as phenicols, tetracyclines, aminoglycosides, and macrolides); (III) inhibition of nucleic acid synthesis (such as quinolones); and (IV) inhibition of folate synthesis (such as sulfonamides and trimethoprim) (Zou et al., 2017). The misuse and overuse of antimicrobials have facilitated the rapid development and spread of antimicrobial resistance (Heffernan, 2022; Sagar et al., 2023). Bacterial infections caused by multidrug-resistant organisms are often responsible for increased morbidity, mortality, and medical costs. Globally, an estimated 4.95 million deaths were associated with antimicrobial resistance in 2019, of which 1.27 million deaths were directly attributable to antimicrobial resistance (Murray et al., 2022). The review on antimicrobial resistance by O'Neill (2014, 2016) reported that antimicrobial resistance could cause 10 million deaths every year by 2050.       The dissemination of antimicrobial resistance is largely attributed to the transfer of antimicrobial resistance genes (ARGs) among different bacterial hosts, including by vertical gene transfer (VGT) and horizontal gene transfer (HGT) (Li et al., 2019). HGT is the dominant mechanism for the transmission of ARGs, and consists of the following three primary pathways: conjugation, transformation, and transduction, with conjugation being the most important route (Li et al., 2021; Zhang et al., 2021). In general, these ARGs are located on mobile genetic elements (MGEs), such as plasmids, insertion sequences (ISs), transposons (Tns), and integrons (Ins) (Partridge et al., 2018; Zhang et al., 2019). Plasmids are extrachromosomal DNA molecules that can be gained and lost within and between bacterial populations, thereby playing a significant role in HGT (Ares-Arroyo et al., 2023). Additionally, it was discovered that plasmids of incompatibility groups IncF, IncI, IncA/C, IncH, IncN, and IncX were considered major carriers of ARGs and conferred a multidrug-resistance (MDR) phenotype, resulting in the widespread dispersion of many classes of ARGs, including β-lactamases (blaTEM, blaCTX-M, blaCMY, and blaOXA), quinolone resistance genes (PMQR), colistin resistance genes (mcr-1), enzyme-modifying genes of aminoglycosides (aadA, aphA1, aacC, strA, and strB), sulfonamides (sul1, sul2), and trimethoprim (dfrA), among others (Rozwandowicz et al., 2018).       At present, there are many reports on different types of resistance plasmids and their horizontal transmission in Enterobacteriaceae (Sun et al., 2019; Stosic et al., 2021; Lefèvre et al., 2023; Supa-amornkul et al., 2023) and Vibrio spp. (e.g., Vibrio cholerae and Vibrio alginolyticus) (Wang et al., 2018; Zheng et al., 2018), but there are still few studies on V. parahaemolyticus (Liu et al., 2013). In this study, we identified a conjugative plasmid recovered from a V. parahaemolyticus strain of seafood origin that carried 12 different ARGs. The objective of this study was to determine the structural features and transferability of the plasmid, and MDR phenotypes for the tested antimicrobials.\"}, {'pmcid': '11319344', 'doi': '10.1038/s41598-024-69566-2', 'introduction': 'N/A'}, {'pmcid': '11318556', 'doi': '10.1016/j.psj.2024.104043', 'introduction': 'N/A'}, {'pmcid': '11316331', 'doi': '10.1186/s12879-024-09709-y', 'introduction': 'N/A'}, {'pmcid': '11315073', 'doi': '10.1016/j.heliyon.2024.e34384', 'introduction': 'N/A'}, {'pmcid': '11313837', 'doi': '10.3390/nu16152553', 'introduction': '1. Introduction       A mushroom is the fruiting body of a fungus, typically grown in soil or its food materials [1]. Mushrooms have been widely consumed as a food item for centuries due to their nutritional value, delicious taste, and health-promoting benefits. The global production of cultivated edible and medical mushrooms has increased more than 30-fold since 1978 [2]. Thus, studies on the health benefits of mushrooms have been gaining interest.       The mushroom cell wall contains β-glucan, a polysaccharide consisting of D-glucose monomers linked by β-1,3 glycosidic bonds as the backbone, with β-1,6 branch linkages [3]. β-glucan is not digested in the human gastrointestinal tract because the digestive enzymes secreted by the pancreas and brush-border epithelial cells cannot hydrolyze β-glycosidic bonds. Hence, β-glucan is resistant to digestion in the upper gastrointestinal tract and remains in the large intestine, where it can be utilized by the gut microbiota [4].        The gut microbiota comprise a complex community of 100 trillion microbes, with more than 1000 microbial species colonizing the human intestines. Most gut microbiota are obligate anaerobes, including Firmicutes (65%), Bacteroidetes (25%), Proteobacteria (8%), and Actinobacteria (5%) [5,6]. These gut microbes can ferment non-digestible polysaccharides because they have multiple metabolic genes encoded by the genome and produce several beneficial metabolites for the host [7]. The gut microbiota and their metabolites play important roles in various health benefits, including nutrient metabolism, glucose homeostasis, lipid metabolism, inhibition of pathogens, gut barrier integrity, regulation of immune responses, balance in mood disorders, and alleviation of neuroinflammation [8,9]. One class of metabolites produced by the gut microbiota are the SCFAs, which are volatile fatty acids, among which acetic acid, propionic acid, and butyric acid are the most abundant. Several studies have demonstrated that SCFAs are one of the most important microbe metabolites and are especially related to gut health. For instance, acetic acid plays an important role in inhibiting pathogenic bacteria. Propionic acid is involved in gut hormone stimulation. Furthermore, butyric acid increases mucin production, enhances epithelial barrier function, and is the main energy source of colonocytes [9,10].       The composition and abundance of colonic bacteria are affected by several factors, including genetics, the mode of delivery, medication, lifestyle, and diet patterns [11]. Therefore, one’s diet, especially dietary fiber, impacts the gut microbial community and gut homeostasis. β-glucan from mushrooms is one type of dietary fiber and is well-known for its prebiotic potential that shapes the gut microbial composition and contributes to health benefits, particularly in gut health [12]. Overall, previous findings have revealed that mushroom β-glucan-supplementation in a colitis mouse group increased the abundance of beneficial bacteria and inhibited the growth of pathogenic bacterial groups. Taken together, the production of SCFAs, including acetic acid, propionic acid, and butyric acid, was promoted, resulting in a reduction in intestinal inflammation-related disorders [13,14,15]. These studies have demonstrated that β-glucan from typically consumed mushrooms could alleviate gut inflammation via the modulation of intestinal microbes. Thus, further explorations of new edible mushrooms and their stages could expand knowledge about their positive effects on gastrointestinal health.       Phallus atrovolvatus (Syn. Dictyophora duplicata (Bosc.) E. Fisch.) is found in the southern forests of Thailand and has been recently identified as a new Thai strain. This mushroom has been histologically consumed by locals for decades without reports on any toxicity or discomfort after consumption. Although Phallus spp. are common edible wild mushrooms and regularly used in many cuisines [16,17], it is worth noting that not all wild mushrooms are edible; some of them can cause life-threatening issues. Nowadays, P. atrovolvatus can be commercially cultivated through the support of the Department of Agriculture, the Ministry of Agriculture and Cooperatives, Thailand. P. atrovolvatus is scientifically close to P. indusiatus, mostly found and widely consumed in China. These two mushrooms are commonly called ‘bamboo mushroom’. The Chinese strain of the bamboo mushroom P. indusiatus is regarded as the queen of mushrooms due to its nutritional value, taste, and unique appearance, and it also exhibits several health-promoting benefits, including antioxidative, anti-cancer, anti-inflammatory, neuroprotective, immunomodulatory, and gut microbiota-modulating effects [18].       The biological properties of the Thai strain of bamboo mushroom, P. atrovolvatus, have recently been reported. Crude polysaccharide, extracted from the fruiting bodies of P. atrovolvatus using hot water extraction, showed strong antioxidative activities in vitro and alleviated gut inflammation via the inhibition of myeloperoxidase activity and pro-inflammatory cytokines in dinitrobenzene sulfonic acid-induced colitis mice [19,20]. In Thailand, P. atrovolvatus is commonly consumed as the mature fruiting body; however, consumption at the egg stage has attracted interest due to its unique and specific texture. The growth of the bamboo mushroom’s fruiting body is divided into four maturity stages, starting from primordia and developing into ball-shaped, peach-shaped, and mature stages [21]. The immature stage, which is peach-shaped, is also called the egg stage. Different maturity stages of mushrooms can affect their taste, flavor, chemical compounds, polysaccharide contents, and biological properties [22]. A previous study reported that the total phenolic and flavonoid contents of Ganoderma lucidum stipes were highest in the spore maturity stage. In contrast, the fruiting body maturity stage contained the highest content of ganoderic acid, one of the triterpenoids found in Ganoderma spp. mushrooms [23]. The chemical components and anti-tumor activity of Pleurotus eryngii fruiting bodies increased with development to the mature stages [24].       Although the impacts of mushroom polysaccharides—in terms of molecular weight, polysaccharide composition, and structure—on gut microbiota modulation have been established in a review by Zhao et al. [25], the effects of the different stages and the link to the chemical compositions and consumption forms of mushroom have not yet been elucidated. Therefore, the current study aimed (1) to determine the chemical composition of the P. atrovolvatus fruiting body in both the egg and mature stages, which are commercially cultivated and consumed; and (2) to investigate gut microbiota-modulating activity, comparing both maturity stages (egg and mature fruiting body) and mushroom consumption forms (cooked mushroom and aqueous extract) to expand the current knowledge and promote the consumption of this unique and edible mushroom.'}, {'pmcid': '11313783', 'doi': '10.3390/plants13152128', 'introduction': '1. Introduction       Leymus chinensis is a perennial rhizomatous grass widely distributed in northern China, Mongolia, the Korean peninsula, and Siberia (GBIF), with high leaf volume, rich in nutrients, and good palatability, and it is the main high-quality forage grass in northeastern China [1,2,3]. However, it is susceptible to infection by pathogens, which affects its economic and ecological values [1,4]. Grassland plant diseases, such as leaf spot, are the most diverse and widely distributed disease groups. Leptosphaeria avenaria and Parastagonospora nodorum could lead to the leaf spot disease of L. chinensis [1,5]. Thus, the plant biomass, nutrients, as well as growth rate decrease obviously, affecting its yield and quality [4]. Pathogens parasitizing perennial plants also have long-term cumulative impacts on grassland communities, even endangering other plants around the host and changing the direction of vegetation succession in the whole community [6,7,8].       Grazing is the main utilization method of natural grasslands; animal feeding, trampling, and feces may either reduce the incidence of disease or increase the chances of disease transmission and infestation. Animal feeding has the ability to eliminate pathogens by removing pathogen-infected plant tissues, reducing the primary source of infection, and mitigating disease incidence. Additionally, grazing increases the infection of certain pathogens and exacerbates the spread of disease by creating new wounds on plant surfaces [9,10,11]. The movement of herbivores and mechanical damage caused by trampling can also accelerate the spread of some pathogens across grasslands [11]. In addition, animal feces can increase soil nitrogen and phosphorus concentrations, significantly alter soil chemistry, and increase the risk of plants infecting themselves with fungal pathogens [12]. Damage caused by grazing can also cause plant tissues to activate jasmonic acid-dependent defense responses more quickly and efficiently, therefore reducing pathogen invasion [13,14]. At the same time, grazing can also indirectly affect the occurrence of plant diseases by altering the plant community. Studies have shown that due to the selective feeding on plants by herbivores can lead to changes in species composition, canopy characteristics, and the phylogenetic structure of plant communities [15,16]. Microenvironmental factors, such as humidity, temperature, and wind speed, closely linked to disease incidence and pathogen spread are also influenced by grazing [17]. Thus far, many studies have proved that moderate grazing effectively controls certain diseases in natural grasslands by reducing the occurrence of diseases, such as leaf spot diseases, through direct or indirect factors [11,18].       It is well known that microbial communities are closely related to plant health, and beneficial micro-organisms can protect plants from pathogens by antagonizing, competing with pathogens, or establishing a reciprocal, symbiotic relationship with the host plant [19,20]. Whereas, some micro-organisms are potential phytopathogens, and growth and reproduction under suitable environmental conditions can jeopardize plant health [21,22]. Several studies have focused on the relationship among soil, inter-root microbial communities, and plant health, and reported that plant diseases strongly influence microbial community structure, and in turn, microbes influence disease development through complex interactions as well. Jiao et al. (2022) discussed that the diversity of microbial communities in soil samples of healthy Fritillaria ussuriensis was higher than that in soil samples of F. ussuriensis infected with Fusarium sp. [23]. The probable reason could be that the root secretions of healthy plants can provide more nutrients to soil micro-organisms, thus increasing the species richness and diversity of the microbial community. Fernandez-Gonzalez et al. (2020) reported that susceptibility/tolerance to soil-borne pathogens in olive was correlated with changes in the inter-root microbial network [24]. Like soil micro-organisms, phyllospheric micro-organism have similar effects on plant health. The plant leaf provides a vast living space and nutrients for micro-organisms, which can inhibit the growth of plant pathogens through mechanisms, such as nutrient competition and antagonism [25,26,27]. It can also affect host plants through the production of volatile organic compounds (VOCs), produced by the Poaceae endophytic fungi, Epichloë sp. It has fungitoxic effects at ecologically significant concentrations, where Chokol K inhibits spore germination of plant pathogenic fungi [28]. Thus, plants can maintain health by regulating the phyllospheric micro-organism composition and diversity, as well as preventing flora dysbiosis. Relevant studies have mainly focused on crops and model plants, with greenhouse pot experiments, and little has been reported in complex natural grassland ecosystems [29,30,31,32,33].       Few other studies suggest that animal trampling, selective feeding, and defecation alter the abundance, composition, and diversity of soil micro-organisms by affecting soil nutrient cycling and soil structure, which are dependent on the intensity and duration of grazing [34,35]. Jing et al. (2023) found that yak grazing altered the structure of soil microbial communities [36], however, the network complexity of bacteria and fungi responded differently to grazing. Eldridge and Delgado-Baquerizo (2018) showed that sheep grazing had direct and indirect negative effects on the abundance of plant pathogens [37]. Existing studies of grazing-affected microbial community changes and grazing-affected plant diseases have been conducted independently, with few studies linking the two parts. Consequently, there is a paucity of research on the effects of grazing on microbial communities and plant health, focused mainly on soil micro-organisms. However, the destruction of plant above-ground tissues by feeding and trampling by grazing livestock creates favorable conditions for the colonization of phyllospheric micro-organism. Simultaneously, it may also disrupt the interactions between micro-organisms that have already been effectively colonized, compromising the normal growth and development of the plant. Excreta from domestic animals could also inevitably affect plant phyllospheric micro-organisms through direct contact with plant above-ground tissues or by altering the physiological and biochemical properties of the soil, thereby affecting the normal growth of the plant.       Theoretically, the pathogens that cause leaf spot disease in L. chinensis mainly infect the wounds on plant leaves caused by livestock feeding and trampling. Livestock causes wounds more conducive to the horizontal transmission of micro-organisms, and livestock feeding may reduce the load of pathogens and the source of primary infestation in the field. The saliva and excreta of livestock may change the phyllospheric micro-organism community, and some of them may also act as antagonists to the pathogen, parasitism, and other complex relationships. In summary, the microecological balance composed of grazing livestock, pasture, and phyllospheric micro-organism is important for the healthy growth of L. chinensis in natural grasslands.       Exploring the interactions and synergistic development of livestock-pasture diseases and phyllospheric micro-organism under grazing conditions could be a meaningful study. Therefore, this study investigated the effects of grazing intensity on the phyllospheric micro-organism community and took a typical leaf spot disease in L. chinensis, as the research object. The study was aimed at elucidating the interactions between livestock-quality pasture leaf disease and phyllospheric micro-organism in natural grassland ecosystems. The current research could provide a theoretical basis for the sustainable utilization of natural grassland and the biological prevention and control of high-quality pasture diseases.'}, {'pmcid': '11312530', 'doi': '10.1101/2024.07.30.605899', 'introduction': 'N/A'}, {'pmcid': '11312258', 'doi': '10.3390/foods13152343', 'introduction': '1. Introduction       Cholesterol is an important factor in the maintenance of cellular homeostasis, and it is synthesized in the liver (about 700–900 mg/day) and ingested through diet (about 300–500 mg/day) [1]. An excessive consumption of foods rich in saturated and trans fats will increase cholesterol levels and lead to hypercholesterolemia. Hypercholesterolemia is characterized by high levels of total cholesterol (TC), low-density lipoprotein cholesterol (LDL-C), or triglycerides (TG), as well as a decrease in high-density lipoprotein cholesterol (HDL-C) [2]. Hypercholesterolemia often leads to atherosclerosis, coronary artery disease, and a range of cardiovascular and cerebrovascular diseases. Statins are commonly used as cholesterol-lowering drugs, but they may cause adverse reactions, such as liver and kidney damage, rhabdomyolysis, elevated blood sugar-induced diabetes, etc., and they are not considered to be an ideal long-term treatment [3]. Therefore, it is important to find new effective drugs and new treatment strategies for preventing or alleviating hypercholesterolemia.       Systemic cholesterol homeostasis is a strictly regulated process involving de novo biosynthesis, dietary cholesterol absorption, biliary clearance, and excretion, and the liver is the main organ to maintain cholesterol homeostasis [4]. The metabolism of cholesterol in the body mainly through the following ways: 1. Cholesterol is transformed into cholesterol derivatives with important physiological functions, such as Vitamin D, sex hormones, and adrenocortical hormones; 2. Cell-membrane formation; 3. Esterification; 4. Excretion in feces; 5. Cholesterol is hydroxylated and degraded to produce bile acids (BAs) [5]. BAs are steroid substances synthesized by the liver from cholesterol. Various bile salts (BS) formed by BAs are the main components of bile, and the conversion of cholesterol into BAs accounts for 80% of total metabolism. There are two main pathways for BA synthesis in hepatocytes: The classical pathway with cholesterol 7α hydroxylase (CYP7A1), and the alternative pathway with sterol 27 hydroxylase (CYP27A1) [6]. When cholesterol levels are high, cholesterol is broken down into BAs and stored in the gallbladder. They are then released into the duodenum and proximal jejunum after cholecystokinin stimulation. After entering the intestine, a part of the BAs is absorbed by the mucosa and returned to the liver to complete the hepatoenteric circulation. In contrast, the unabsorbed parts are degraded into fecal matter by intestinal microorganisms and expelled [7]. Promoting the conversion of liver cholesterol to BAs is an important method to reduce cholesterol, which is of great significance for the prevention and treatment of hypercholesterolemia.       Intestinal flora is related to digestion, nutrient metabolism, immune function, etc., and is important in body health and physiological metabolism. Diet is one of the most critical factors in the change of gut microbiota structure, and the functional components in food can regulate the growth and metabolic activities of gut microbiota, thus affecting the microbial composition. Under normal circumstances, the gut flora and the host benefit from a mutual dynamic ecological balance. Once this balance is disrupted, pathogenic bacteria can cause infectious diseases, metabolic syndromes (such as hyperlipidemia and high blood pressure), and certain neurological disorders [8,9]. Studies have shown that a high-fat diet disrupts the body’s original microbial balance and induces more conditioned pathogens, such as Betaproteobacteria, Clostridium_bolteae, Desulfovibrio, and Enterobacter_cloacae [10,11,12]. In addition, intestinal flora participates in and regulates host metabolism by producing metabolites and short-chain fatty acids (SCFAs) to alleviate hypercholesterolemia [13]. SCFAs can stimulate the proliferation and differentiation of intestinal epithelial cells and help maintain the mineral balance of intestinal epithelial cells and the absorption of iron, calcium, and magnesium. Common SCFAs include acetic acid, propionic acid, and butyric acid, while other SCFAs such as isobutyric acid, valeric acid, and isovaleric acid are relatively low in the colon [14].       Polysaccharides are essential biomacromolecules in living organisms, playing important roles in life activities such as cell–cell connections, signaling pathways, cell adhesion, and immune system molecular recognition. In recent years, more and more studies have reported the isolation and identification of bioactive polysaccharides from natural sources, indicating that polysaccharides have a variety of biological functions, including anti-hypercholesterolemic activity [15,16,17]. Cyclocarya paliurus is a species native only to China, and its leaves are used in medicinal preparations in traditional Chinese medicine, as well as an ingredient in functional foods [18,19]. Previous studies have shown that polysaccharides from Cyclocarya paliurus leaves can regulate lipid metabolism, relieve hyperlipidemia, and relieve diabetes, but its effect on hypercholesterolemia has not been reported [18,19,20]. Therefore, this study attempted to explore the effect of Cyclocarya paliurus polysaccharides on hypercholesterolemia through in vivo and in vitro experiments to explore the mechanism of action.'}, {'pmcid': '11312129', 'doi': '10.3390/ijms25158347', 'introduction': '1. Introduction       Antibiotic-resistant Enterobacterales, especially E. coli and Salmonella spp., are considered major enteric pathogens, posing a threat to global public health. They can occupy multiple niches, including human and animal hosts [1,2]. The United States Center for Disease Control and Prevention (CDC) has designated foodborne drug-resistant Salmonella spp. as a significant public health hazard.       Infective multidrug-resistant (MDR) Enterobacterales in the poultry industry have been linked to increased morbidity and mortality. The transmission of these pathogens via food may be responsible for causing foodborne illnesses and deaths.       Plasmids serve as reservoirs for multiple antibiotic-resistance genes among closely related bacterial species. Conjugative incompatibility F (IncF) plasmids have received much attention among 27 incompatibility (Inc) groups previously recognized in Enterobacterales alone. Plasmid incompatibility refers to multiple plasmids within one cell having the same replicon or partitioning system [3]. IncF plasmids were sub-grouped into replicons FII, FIA and FIB, which are the most frequently encountered among Enterobacterales, especially E. coli. They represent 23.1% of plasmids [4] that have been deposited in the plasmid multilocus sequence typing (pMLST) database (http://pubmlst.org/plasmid, accessed on 20 December 2023). Conjugative plasmids (CPs) transmitting antibiotic resistance genes are the major direct cause of outbreaks of antibiotic-resistant Enterobacterales. Actually, Gram-negative bacilli have a remarkable capacity to transfer conjugative antibiotic resistance genes [5]. Transfer of a single plasmid to a bacterium allows it to be resistant to multiple antibiotics. Therefore, plasmids harboring antimicrobial resistance genes may result in plasmid endemics [6,7]. Despite the relatively low abundance of conjugative plasmids, many resistance genes of sulfonamide (sulI and sulII), chloramphenicol (cat), trimethoprim (dfr), colistin (mcr), New Delhi metallo-beta-lactamase (blaNDM) and extended-spectrum β-lactamase (ESBL; blaTEM, blaCTX-M and blaSHV) were significantly located in these plasmids [8,9,10].       The widespread nature of antimicrobial resistance (AMR) genes mediates plasmid epidemics among different members of Enterobacterales [11]. The bacterial-resistant phenotypes observed during epidemics can persist for months even in the absence of ongoing specific selection with risk of prolonged carriage [12]. Plasmid epidemics arise from the massive exchange of AMR genes (more than 86%) not only across other distant plasmid categories but also between bacterial chromosomes [13]. The mobility of CPs is a strong factor facilitating the accumulation of AMR genes as these plasmids possess the ability to pass across phylogenetically distant bacteria [14,15]. Transfer and stabilization of mutations in insertion sequence (IS)-associated AMR genes are affected by the use of antibiotics, which also promote bacterial conjugation [16,17,18], support bacterial fitness and persistence in adverse environmental conditions, providing virulence (bacteriocins, siderophores, cytotoxins, and adhesion factors) and antimicrobial resistance determinants [19]. Antibiotic selective pressure on commensals and pathogenic AMR Salmonella spp. and E. coli leads to clonal expansion of these strains [20]. Using oral antibiotics during the broiler production period for therapeutic, prophylactic, or subtherapeutic growth promotion purposes modifies gut microbiota [21]. Lateral transfer of CPs to recipient cells confers genes encoding antimicrobial resistance, exerts strong selective pressure, and facilitates the spread of genetic elements of insertion sequences (IS26 and IS1294), integrons, and transposons [22,23].       In Egypt, IncF grouping studies have focused only on the association of resistance to an antibiotic with a specific replicon using whole-genome sequencing. However, there is a need to investigate the molecular subtyping of conjugative IncF plasmids [24]. In the present work, PCR-based replicon grouping was applied using conserved IncF primers following the GenBank entries, and the replicon type was subjected to allele subtyping [25].'}, {'pmcid': '11311926', 'doi': '10.1186/s13099-024-00633-9', 'introduction': 'N/A'}, {'pmcid': '11311727', 'doi': '10.3390/diagnostics14151598', 'introduction': '1. Introduction       In the realm of molecular biology and diagnostics, the emergence of digital polymerase chain reaction (dPCR) has sparked a revolution in the way we perceive and manipulate nucleic acids. This novel technique offers unparalleled precision and sensitivity, enabling researchers and clinicians to detect and quantify nucleic acids with unprecedented accuracy. As we delve into the intricate world of dPCR, it becomes evident that its applications extend far beyond traditional PCR methodologies. dPCR represents a technological leap from conventional polymerase chain reaction methods, offering an accurate absolute quantification of target nucleic acid molecules present in a sample [1]. This advanced technique partitions the sample into numerous compartments within which nucleic acids undergo stochastic partitioning followed by clonal amplification [2]. Concentrations are determined by analyzing the proportion of nonfluorescent particles according to the Poisson distribution. Recent advancements have led to the development of several commercial PCR droplet platforms, aimed at expediting the clinical adoption of digital PCR.       This review aims to provide an overview of dPCR, focusing on its applications in infectious disease diagnostics and environmental surveillance. We will delve into the fundamental principles underlying dPCR, highlighting its unique ability to partition nucleic acids into individual reactions, thereby facilitating absolute quantification. Beyond its technical intricacies, the true essence of dPCR lies in its transformative potential across various fields. From precision medicine to environmental monitoring, dPCR offers a versatile toolkit for addressing complex biological questions and tackling real-world challenges.       As we navigate through the potential applications, it is imperative to address the challenges, the limitations, and the future perspective in dPCR technology. From optimizing experimental conditions to standardizing data analysis workflows, there are several obstacles that must be overcome to fully realize the potential of dPCR in clinical and research settings. By critically evaluating these challenges, we can chart a course toward harnessing the full power of dPCR for advancing science and improving healthcare outcomes.'}, {'pmcid': '11304630', 'doi': '10.1186/s12934-024-02484-0', 'introduction': 'N/A'}, {'pmcid': '11304591', 'doi': '10.1093/g3journal/jkae119', 'introduction': \"Introduction       Underpinning every aspect of an organism's biology is its evolutionary history. Phylogenetic methods strive to reconstruct these histories using genetic data. With the advent of “Next-” and “Third-gen” sequencing methodologies, the amount of genetic data in public databases has increased dramatically. At writing, there are currently 2,018,201 bacterial genome assemblies available through NCBI alone (ncbi.nlm.nih.gov/assembly/?term = bacteria). The wealth of available genetic data has transformed the fields of bacterial phylogenetics and taxonomy (Konstantinidis and Tiedje 2007; Varghese et al. 2015; Jain et al. 2018). Unfortunately, most whole genome phylogenetic analysis methods require specialized bioinformatic data processing and expertise (Smith 2013; Lozano-Fernandez 2022). There are few very large-scale trees to place these sequences into an evolutionary framework. The available large-scale trees are under-resolved and built by reconciling many disparate phylogenetic studies, such as NCBI taxonomy (Schoch et al. 2020) or are generated with methods unsuited for narrow and broad evolutionary distances (Hördt et al. 2020) reducing their utility at multiple evolutionary scales. This flood of genomic data necessitates an easy-to-use phylogenetic analysis pipeline to help reveal the evolutionary context for the myriad of sequenced bacteria.                Analysis options                    Single and multi-locus           A classic way to generate bacterial trees is to compare single homologous loci (e.g. 16S ribosomal, recA, gyrA, rpoB, or dnaK genes). While largely replaced by other methods, single gene trees have some advantages, which include the ability to use conserved primer sites to amplify sequences, very low computational intensity, and the availability of curated databases (SILVA, RDP, NCBI GenBank, etc.). Using genes other than rDNA allows for greater resolution at certain evolutionary distances but still lacks broad resolution (i.e. Near resolution for swiftly evolving genes, far for conserved) (Yang 1998). However, these trees are prone to misleading topologies due to horizontal gene transfers, incomplete lineage sorting, paralogous gene conversions, and limited phylogenetic information (Huerta-Cepas et al. 2007). Using multi-locus alignments to build phylogenetic trees increases the total amount of phylogenetic information (more sequence used), reduces the effects of incongruent gene tree topologies due to recombination or horizontal gene transfer, and widens the breadth of evolutionary resolution (Gontcharov et al. 2004). However, in multi-locus analyses of taxa without well-vetted multi-locus methods, locus selection is non-trivial and the effort and cost of primer design and amplification optimization scales with the number of sequences used, which can rapidly become prohibitive for many labs. Additionally, strains of many taxa, like Brucella, are not differentiated well by available multi-locus methods, leading to unknown evolutionary histories (Sankarasubramanian et al. 2019).                             Whole genome           Classically, whole genome alignments were required to identify Single Nucleotide Polymorphisms (SNPs) of assemblies (see Shakya et al. 2020; Treangen et al. 2014 for method details). These methods add considerable phylogenetic data for tree building and can produce different topologies than multi-locus alignments, illustrating their potential utility (Baltrus et al. 2014). A major drawback of whole genome alignments is that they tend to produce short alignments or fail outright, for more distantly related genomes (Darling et al. 2004; Angiuoli and Salzberg 2011; Chung et al. 2018; Shakya et al. 2020). Thus, phylogenetic information drops quickly as a function of evolutionary distance. The practical result is a lack of resolution at even moderate evolutionary distances. For alignments to a reference, having the “missing” data enriched for the more divergent sites will possibly lead to an underestimation of the sequence to reference evolutionary distances for more divergent sequences (Spencer et al. 2007; Shavit Grievink et al. 2013; Bertels et al. 2014). Alternatively, one can use all-vs-all whole genome alignments, however, they scale poorly and are limited in the number of input taxa that can be aligned (Angiuoli and Salzberg 2011).                             K-mer overlap           An emergent method to quickly estimate evolutionary histories from genome assemblies is to use k-mer content. This involves identifying subsequences of length k within each assembly and calculating the shared k-mer content (for details see Bussi et al. 2021). The advantage of this class of methods is that it is extremely fast relative to alignment-based methods. The comparison of k-mer overlap between sequences naturally leads to pairwise distance matrices, thus neighbor-joining trees can be generated directly from the results (Saitou and Nei 1987). Recent work has integrated more complicated distance statistics with success (Tang et al. 2023). Crucially, k-mer distances become non-linearly associated with true percent identity, leading to erroneous estimated branch lengths (Jain et al. 2018).           An alternative k-mer-based approach is kSNP (Gardner and Hall 2013; Gardner et al. 2015; Hall and Nisbet 2023), which identifies SNPs by flanking conserved k-mers. This approach allows the use of tree-building methods including parsimony, maximum likelihood, and Bayesian. Like whole genome alignments and k-mer overlap methods, kSNP suffers from a rapid decline in phylogenetic information as sequences diverge. Additionally, the signal/noise ratio degrades quickly because non-homologous k-mers are increasingly common as sequences diverge (Gardner and Hall 2013).                             Predicted gene alignments           To use the wealth of data generated by NGS methods and simultaneously simplify the problem of whole genome alignments, CDS or protein sequences derived from annotations or transcriptome sequencing can be compared. This method breaks the problem of identifying homologous loci into two tractable parts: gene identification, then identification of homologous sequences. This approach identifies many phylogenetically informative sites (like whole genome), but alignments are more tractable (like single loci). However, gene-based alignments are negatively affected by inferred paralogs, i.e. duplicated genes or contamination. Thus, most methods filter out gene families with paralogs. However, see ASTRAL-Pro for a method that is “paralog aware” (Zhang et al. 2020). A widely used pipeline that accomplishes this task is GToTree, which annotates assemblies, uses precomputed hidden Markov models to identify single copy orthologs, aligns predicted amino acid sequences, then generates trees with FastTree2 or IQTree2 from concatenated alignments (Lee 2019). While focused on protein alignments, GToTree can also be run in nucleotide mode to use CDS alignments for phylogenetic inference instead.           A trade-off arises while using predicted coding or protein sequences to build species trees. Coding sequences (CDS) have the most phylogenetic information due to codon degeneracy but are hard to align at high divergencies (States et al. 1991). Proteins remain alignable at high divergences but lack the information content of CDS alignments. This can be addressed by converting protein alignments to corresponding codon alignments, leveraging nucleotide phylogenetic information and protein alignment accuracy (Wernersson and Pedersen 2003; Bininda-Emonds 2005). However, at increasing evolutionary distances, even with accurate alignments, mutational saturation in nucleotide data can negatively affect phylogenetic estimates. Evolutionary model selection, avoiding compositional bias, and dense taxon sampling can largely alleviate these issues, see (Kapli et al. 2023) for details.                                Benefits of an automated workflow         The skills required to generate bacterial phylogenetic trees from whole genomes are extensive. Some of the many steps include gathering and annotating assemblies, identifying and filtering orthologs, sequence alignment, trimming and concatenation, and finally, tree inference (Ashford et al. 2020; Lozano-Fernandez 2022). Each one of these steps, especially for large data sets, requires expertise in picking parameters, file management, and data format manipulation and filtering. Many of these steps also require familiarity with UNIX command line.         Beyond taxonomic studies, the phylogenetic context of organisms is becoming increasingly important for standard molecular and evolutionary studies. Even with the available data and a clear need for the phylogenetic placement of focal study species, technical barriers preclude many researchers from inferring their evolutionary histories. An easy-to-use, accurate phylogenomic pipeline will encourage wide adoption of phylogenomic methods to complement ongoing environmental, evolutionary, and clinical microbiological research, and perhaps help standardize the estimation of phylogenetic trees within and between research labs.         To meet this demand, we present OrthoPhyl, a phylogenomic pipeline that takes bacterial genomes as input, annotates them, identifies orthologs, converts protein to nucleotide alignments, and builds species trees with both concatenated alignments and gene tree to species tree reconciliation. The workflow accepts an arbitrarily large number of input genomes (tested up to 1,200 here). To accelerate analysis, a subset of samples representing the whole dataset's diversity are identified, their proteomes are used to identify orthologs and build hidden Markov models, which are then expanded to the full input dataset by iterative searches. This strategy allows the generation of trees for 689 Brucella assemblies in ∼48\\u2005hrs using 30 cpus and 58.3GB memory with no hands-on time required. This pipeline is designed to be an easy-to-install, scalable, turn-key solution for generating high-resolution bacterial trees from diverse clades.\"}, {'pmcid': '11303641', 'doi': '10.1186/s11671-024-04075-9', 'introduction': 'N/A'}, {'pmcid': '11302663', 'doi': '10.1128/spectrum.00052-24', 'introduction': 'INTRODUCTION       Retinal vein occlusion (RVO) is a prevalent blinding disorder within the realm of ophthalmology, ranking as the second most common retinal vascular disease globally (1, 2). Annually, approximately 16.4 million individuals develop RVO (3). The etiology of RVO is multifaceted, with specific pathogenic mechanisms and causative factors yet to be fully elucidated. Associated risk factors for RVO encompass advancing age, diabetes mellitus, hypertension, arteriosclerosis, cardiovascular diseases (characterized by augmented blood viscosity), obesity (accompanied by hyperlipidemia), hematological disorders (leukemia, lymphoma, and polycythemia), dysproteinemias, systemic vasculitis or autoimmune diseases (causing endothelial injury), malignancies, tobacco use, and anemia, among others (4–8). RVO primarily manifests through either partial or complete occlusion of the retinal veins, attributed to these diverse factors. This blockage disrupts venous blood return, leading to venous dilation and subsequent retinal hemorrhage. These initial pathological changes frequently progress to macular edema, which significantly deteriorates visual acuity and can lead to blindness in severe cases (9). Although the incidence of RVO is slightly lower than that of the most prevalent diabetic retinopathy (DR), its prevalence increases gradually with progressive aging, imposing significant economic burdens on both seniors and society (10). Consequently, investigating the pathogenesis underlying RVO holds vital societal and economic values.       Our bodies harbor many ever-evolving microbial communities, known as microbiotas, whose intricate configuration and functionality play crucial roles in our well-being and the onset of diseases (11). The harmonious coexistence of microbial commensals exerts an extensive influence over the intricate web of immune development and homeostasis, shaping the delicate equilibrium between well-being and illness. A growing body of evidence supports the hypothesis that these gut-dwelling allies not only play a role in intestinal disorders but also extend their impact to remote tissues. Emerging scientific evidence has uncovered an intriguing phenomenon termed the “gut–eye axis,” demonstrating how the dynamic presence of gut microbiota influences ocular health, including the retinal structure (12–14).       Various retinal disorders, such as age-related macular degeneration, DR, or retinitis pigmentosa, may fall under the influence of the gut microbiota, hinting at the existence of a direct gut–retina axis (15, 16). However, current research does not directly link RVO onset with gut microbiota imbalance; in other words, whether RVO is related to the gut–retinal axis is still unknown. Systemic diseases that alter blood flow, blood viscosity, coagulation, or vascular structure are known to lead to RVO (17–21), yet the specific pathogenesis remains poorly understood. Therefore, we speculate whether gut dysbiosis could prompt vascular alterations and affect vascular health (i.e., changes in blood viscosity and abnormal blood coagulation), thereby initiating the onset of RVO.       Gut microbes profoundly influence the physiology of their hosts, encompassing vital aspects such as metabolism, immune system homeostasis, nervous system regulation, and resistance against pathogen colonization (22, 23). With the vigorous advancement of multiomics technologies, there has been a shift from meticulous reductionist investigations toward holistic systemic research. These technologies allow for the cross-validation of findings, enhancing the robustness and applicability of our conclusions about gene functionality. However, genomic analysis of gut microbiomes merely deciphers microbial composition and potential functionalities, without delving into their actual activities at a viable state. In contrast, metabolomics represents a discipline that explores the variations in metabolites of a community of organisms resulting from external perturbations and unveils patterns of temporal changes—a field more closely aligned with phenotypic studies (24). Thus, our study endeavors to unravel potential mechanisms underlying the onset of RVO by integrating various omics technologies, including genomics and metabolomics, offering significant implications for guiding clinical therapeutic development.'}, {'pmcid': '11302304', 'doi': '10.1128/spectrum.00759-24', 'introduction': 'INTRODUCTION       Recent meta-analyses of semen parameters from 1973 to 2019 indicate a significant global decline in sperm concentration (SC) and total sperm count (TSC), with the rate of reduction accelerating (1). Inferior sperm quality has been strongly associated with adverse reproductive outcomes (2). Nonetheless, the predictive capacity of routine semen analysis for assessing male fecundity is constrained (3). The DNA fragmentation index (DFI), a measure of sperm DNA fragmentation, has gained clinical relevance (4). Factors, such as reactive oxygen species (ROS) (5) and inflammatory agents (6), have been identified as contributors to sperm DNA damage. High levels of sperm DNA fragmentation are associated with reduced fertilization rates, poor embryo quality, lower pregnancy success, and increased miscarriage rates (7).       Current treatments for high DNA fragmentation index (HDFI) include antioxidant therapy, infection management, and lifestyle changes (8), but these approaches often prove inadequate, especially in cases of idiopathic HDFI (9). This has led to an increased reliance on assisted reproductive technologies (ARTs) (5), yet the success rates vary, underscoring the need for alternative treatment strategies.       The human microbiome, encompassing the collective genetic material of microbiota in various body niches, plays a vital role in physiological homeostasis (10). Imbalances in the microbiome can lead to various pathologies (11). Recent research has begun to explore the connection between the semen microbiome and semen quality (12). Notably, a study by Sergio et al. using 16S rRNA gene sequencing revealed a negative correlation between certain bacteria and sperm DNA fragmentation in a Western Mediterranean cohort (13). Thus, our study aims to further investigate the role of the semen microbiome in HDFI using both 16S rRNA and metabolome sequencing. We hypothesize that the seminal microbiota could significantly influence sperm DNA integrity, potentially offering new avenues for HDFI treatment.'}, {'pmcid': '11300865', 'doi': '10.1038/s41598-024-69162-4', 'introduction': 'N/A'}, {'pmcid': '11237486', 'doi': '10.1128/spectrum.04278-23', 'introduction': 'INTRODUCTION       Bacteria, which function as social organisms, engage in continual interactions with adjacent cells to outcompete their nonkin counterparts for vital nutrients and space (1). In this microbial competition, they have evolved an array of strategic mechanisms aimed at surpassing neighboring species, including the production of diffusible molecules, the release of phage or phage-like particles, and the use of contact-dependent antibacterial weaponry (2, 3). Among these strategies, the Type VI secretion system (T6SS) is a lethal weapon that functions as a crossbow and effectively delivers toxic effectors to target cells (4, 5). The T6SS ejects a spear-like tube carrying toxic effectors outward within milliseconds, possessing sufficient force to puncture through the formidable barrier of the cell wall and two cellular membranes of Gram-negative cells (6, 7). Intriguingly, the T6SS has unveiled a myriad of effectors across a diverse spectrum of bacterial species, and antibacterial toxins are the most extensively studied among these effectors (8, 9). Notably, each antibacterial toxin is encoded in tandem with a cognate immune gene within a precise proximity, a defensive mechanism shielding the bacterium from self-intoxication or T6SS-mediated killing by neighboring sister cells (4). The bactericidal efficacy of the T6SS introduces a novel approach for modulating the composition of the intestinal and rhizosphere flora in practical applications. This discovery holds promise prospects for addressing challenges related to drug-resistant bacteria, as well as for the prevention and control of plant pathogens and environmental ecological management.       Many species have evolved to produce various antimicrobial compounds that inhibit growth of or kill rival species, encompassing an array of defenses such as broad-spectrum antibiotics, proteinaceous exotoxins, lysozymes, and bacteriocins (10). Bacteriocins, a group of potent antibacterial protein, have been identified across almost all bacterial lineages (11). Remarkably, these bacteriocins are peptides synthesized by bacterial ribosomes, closely phylogenetically related species are selected as target cells, and producers have specialized immune mechanisms to counteract the effects of these peptides (12, 13). Bacteriocins are narrow-spectrum antibacterial agents that have biological effects on closely related species, and their encoding genes are found on transposable elements, plasmids, or the producer’s genome (10). The multifaceted modes of action of bacteriocins include the depolarization of lipid bilayer membranes, disruption of cell wall production (14, 15), suppression of protein synthesis (16, 17), and destruction of host nucleic acids (18). A noteworthy aspect of bacteriocins is the dependence of cellular import mechanisms on the specific target organism.       The Gram-negative bacterium Yersinia pseudotuberculosis is widely distributed in the natural environment, and as an intestinal pathogen, it can infect many animal species (19, 20). The genome of Y. pseudotuberculosis encompasses four complete sets of T6SSs that play pivotal roles in bacterial quorum sensing, host defense, and antibacterial activities (21, 22). The third Type VI secretion system (T6SS-3) gene cluster contains several effector-immune protein pairs of unknown function. Previous studies have shown that T6SS-3 secretes a novel Ca2+- and Mg2+-dependent DNA hydrolase effector protein, YPK_0954, which exhibits a contact-independent bactericidal function (23). Considering that YPK_0952 is also an effector protein secreted by T6SS-3, it could be inferred that YPK_0952 may possess similar antibacterial properties.       Here, YPK_0952 was identified as a pyocin-like effector, demonstrating its ability to be secreted via the T6SS-3, a mechanism crucial for its pronounced toxicity. Our results demonstrated that YPK_0952 relies on various bivalent metal ions to exert DNase activity and that this activity is inhibited by the immune protein YPK_0953. Furthermore, YPK_0952 exhibits intra-species and inter-species bactericidal activity through a T6SS-mediated contact-dependent and contact-independent mechanisms. Our data provide insight into important bacterial tactics that enable cells to establish a foothold in competitive niches, not only through the secretion of specialized metabolites and bacteriocins that antagonize microbial competitors but also through contact-dependent and contact-independent suppression of rival bacteria by toxins. The cross-species bactericidal action of pyocin-like effectors released by the T6SS in Y. pseudotuberculosis makes bacteriocins appealing therapeutic agents, particularly in the fight against multidrug-resistant diseases.'}, {'pmcid': '11293832', 'doi': '10.1099/acmi.0.000830.v4', 'introduction': 'Introduction       In the realm of environmental sustainability and bioremediation, biosurfactants have emerged as valuable tools due to their eco-friendly nature and versatile applications. Derived from microbial sources, particularly bacteria, biosurfactants exhibit unique surface-active properties that enable them to reduce interfacial tension and enhance the solubilization and emulsification of hydrophobic compounds, making them promising agents for oil spill remediation. The increasing frequency of oil spills in marine environments is a serious threat to biodiversity and ecosystems. According to the Statista Research Department report, between 2010 and 2019, there was an average of 1.8 significant oil spills due to tanker incidents annually. Over 700 metric tons of oil was reported to have leaked in four separate oil spills in 2022 [1]. These incidents highlight the ongoing risk and environmental impact of oil spills. Despite efforts to improve safety measures and regulations, the frequency and magnitude of such accidents remain a concern. According to the International Tanker Owners Federation (ITOPF), between 1970 and 2019, approximately 5.86\\u2009million tons of oil was spilled into marine environments due to tanker transportation alone [1]. Every year, transoceanic shipping carries about 35\\u2009million barrels of petroleum, and there are many ship accidents, most of which result in a significant spill of petroleum into marine environments. Large-scale, toxic oil spills typically occur at marine sites. Spills and leaks of crude oil have the potential to cause terrible harm to aquatic environments [2]. There is a possibility that these hazardous substances will enter the marine environment during the extraction, transportation and refining of crude oil, which would have negative effects on all biotic life [3]. Conventional techniques for cleaning up oil spills involve the use of mechanical tools like oil booms and skimmers, but these are labor- and money-intensive procedures [1]. Chemicals can be used as oxidants, reductants, polymers, or precipitants in chemical remediation processes [4]. Chemical surfactants, on the other hand, have toxic environmental effects. Biosurfactants are surface active biomolecules produced by microbial (bacteria, yeast and fungi) metabolism, containing both hydrophilic group and lipophilic group [5]. Evaluable biotechnology technique known as ‘bioremediation’ is the process of using micro-organisms to mitigate and remove pollutants from a projected contaminated environment [6]. Because biosurfactants are more environmentally friendly, less toxic, biodegradable and have lower critical micelle concentration (CMC) values than chemical surfactants, they can be used as substitutes for chemical surfactants [7]. The critical micelle concentration refers to the minimum amount of surfactant required to achieve the lowest surface tension [8]. Biosurfactants are a type of surface-active compounds produced by micro-organisms, including bacteria, yeast and fungi. Biosurfactants are amphiphilic compounds that contain both hydrophilic (polar head) and hydrophobic (non-polar tail) domains. Surface-active compounds are categorized based on their molecular weight. Low-molecular-weight (LMW) surfactants lower the surface tension between two immiscible liquids, and high-molecular-weight (HMW) emulsifiers allow water-in-oil or oil-in-water emulsions [910]. The effectiveness of a surfactant is based on the extent to which it can lower surface tension. A highly effective biosurfactant can decrease the surface tension of water from 72 to below 35 mN/m [10]. Low molecular weight glycolipids (rhamnolipids, sophorolipids, etc.) are among the most widely used biosurfactants, and they have been thoroughly and extensively studied for their use in biotechnological applications. A glycolipid is a compound comprising a carbohydrate section and fatty acid chains connected by a glycosidic bond, acting as hydrophilic and hydrophobic links, respectively [11]. Biosurfactants can be produced by a number of bacterial species, including Bacillus, Pseudomonas and Enterobacter. Rooney et al. (2009) [12] reported the isolation of E. hormaechei from the biodiesel-contaminated soil, which was able to produce rhamnolipid-type biosurfactant [12]. Some other studies also reported the production of biosurfactants from Enterobacter spp. [13]. The delve of this study is to explore and characterize a newly isolated bacterium capable of producing biosurfactants, with the goal of assessing its potential for use in oil remediation processes.'}, {'pmcid': '11293673', 'doi': '10.1371/journal.pntd.0012348', 'introduction': 'Introduction       Borrelia spp. are globally distributed, vector-borne bacteria that cause diverse diseases, rendering these pathogens an immense public health concern [1,2]. Borrelia can be divided into two distinct groups, relapsing fever (RF) and Lyme disease (LD) Borrelia, based on associated disease courses, vector/enzootic cycles, and additional phylogenetic differences [1–3]. Infection with RF spirochetes is characterized by recurring severe, febrile episodes that coincide with repeated bouts of high-level bacteremia (up to 108 bacteria/ml of blood) [1,4–8]. Furthermore, infection by RF Borrelia can lead to more severe symptoms and outcomes, such as acute respiratory distress syndrome, jaundice, meningitis, splenomegaly, and perinatal mortality [4,9,10]. Alternatively, mammalian infection with LD Borrelia results in lower levels of bacteremia (103−104 bacteria/ml of blood) that is restricted to early stages of infection, after which it disseminates into multiple host tissues [11]. Therefore, symptoms of RF are primarily associated with acute bacteremia, whereas most LD symptoms occur because of chronic tissue colonization (e.g., arthritis, neuroborreliosis, and carditis) [2]. In addition, arthropod vectors differ greatly between RF and LD Borrelia. RF bacteria are transmitted by the human body louse [louse-borne relapsing fever (LBRF)] or Ornithodoros or Ixodes spp. ticks [tick-borne relapsing fever (TBRF)], while LD spirochetes are only transmitted by Ixodes spp. ticks [12–17]. The different disease courses resulting from infection by LD and RF Borrelia and their disparate enzootic cycles involving distinct vectors suggest that these two groups of bacteria have likely evolved to encode different factors required for vector acquisition and colonization, transmission, and vertebrate infection. Because most molecular research has focused primarily on LD Borrelia, a significant gap exists in our understanding of the biology of RF spirochetes [18,19].       RF Borrelia can be further subdivided into New World (found in the Americas) and Old World (found in Africa, Asia, and Europe) species [4,5,16,20]. New World RF spirochetes include Borrelia turicatae, which is found in the arid climate of the southwestern United States, and Borrelia hermsii and Borrelia parkeri, the causes of RF in the mountainous regions of the western United States and southwestern Canada [4,5,12,13,21–23]. Old World RF spirochetes include Borrelia crocidurae and Borrelia duttonii, which are endemic to western and eastern Africa, respectively, Borrelia persica, a causative agent of TBRF in Eurasia, and Borrelia recurrentis, the LBRF spirochete responsible for epidemic RF in eastern Africa [24–30]. RF spirochetes were historically classified into New World and Old World species based solely on geographic separation, but phylogenetic analyses have revealed genomic differences in these bacteria, supporting the separation of the groups into two distinct clades [26,31–34].       Although RF is globally distributed, the public health impact varies throughout the world. While RF is a relatively rare and possibly underreported disease in the Americas, this infection is endemic and highly prevalent in parts of Africa [1,4,5,20,25–27,32]. In fact, RF has been reported as the most common bacterial infection in Senegal, the most prevalent cause of fever in rural Zaire, and a top 10 cause of death in children under the age of 5 in Tanzania [1,25,27,35–37]. Furthermore, cases are often unreported or are misdiagnosed as another disease, such as malaria, in endemic regions of Africa, implying that the actual incidence is higher than previously reported [28,38]. Despite the clear public health threat that Old World RF spirochetes represent, very little is known regarding the molecular pathogenesis of these bacteria.       Use of molecular Koch’s postulates provides the most direct way to determine a causal relationship between a bacterial gene, the protein it encodes, and a requirement during the bacterial lifecycle [39,40]. To apply molecular Koch’s postulates, the gene of interest is inactivated, and the phenotype of the mutant is compared to that of the wild-type parent. Additionally, expression of the gene must be restored in the mutant to assess if the phenotype of interest is reversed. While a limited number of studies have applied genetic manipulation in RF Borrelia to investigate proteins potentially required during the bacteria’s enzootic cycle, these studies have only been conducted with the New World RF spirochetes, B. hermsii and B. turicatae [41–50]. Extensive geographic separation and phylogenetic differences between New World and Old World RF spirochetes suggests that these two groups likely evolved distinct regulatory pathways and proteins required for mammalian virulence, as well as vector acquisition, colonization, and transmission. Therefore, there is a significant need to develop molecular techniques for the study of Old World RF Borrelia.       Herein, we aimed to develop a genetic system for use in the Old World TBRF spirochete, B. duttonii. First, we developed shuttle vector- and integration-based constructs and methods for electroporation with B. duttonii, creating green fluorescent protein (gfp)-expressing B. duttonii strains to validate the approach. Allelic exchange mutagenesis was then used to inactivate the gene encoding the B. duttonii homolog of P66, a known virulence factor of the LD spirochete, Borrelia burgdorferi [51,52]. Using a murine model of RF, we demonstrated that the B. duttonii p66 mutant was attenuated. Finally, this p66 mutant was complemented using shuttle vector- and cis integration-based approaches to restore production of P66 and murine infectivity of the p66 mutant strains. The techniques described in this manuscript represent a foundation for the application of molecular tools to study Old World RF Borrelia and will ultimately guide research to better understand these understudied pathogens.'}, {'pmcid': '11291472', 'doi': '10.3389/fvets.2024.1436807', 'introduction': '1       Introduction       With the continuous improvement of the scale and intensity of poultry farming, the production performance of poultry has been gradually improved. However, broilers are susceptible to external factors such as disease, nutrition, and environment due to the imperfect development of intestinal function and low immunity, which frequently result in poor health, enhanced stress response, and intestinal imbalance (1). In the era of a complete ban on antibiotics as feed additives and a focus on healthy breeding, the development and utilization of feed additives like medicinal plant extracts and probiotics have garnered much attention (2, 3).       Glycyrrhiza uralensis Fisch is a traditional medicinal and edible plant with a long history of dietary and pharmacological applications, and its edible and medicinal parts are the root and rhizome (4, 5). The primary active ingredients in Glycyrrhiza uralensis extract (GUE) include glycyrrhiza polysaccharides, triterpene saponins (glycyrrhizic acid, glycyrrhizinic acid, etc.), and flavonoids (chalcone, isoflavone, etc.) (6). GUE exhibits various pharmacological effects, such as anti-inflammatory (7), antioxidant (8), antiviral (9), immune regulation (10) and improvement intestinal microbiota (11). Lactobacillus acidophilus is a dominant microbiota in the gastrointestinal tract (GIT) of humans and animals, often used as a probiotics due to its health-promoting properties (12). Probiotics Lactobacillus is typically employed as Direct-Fed microbiota to poultry and other livestock to enhance intestinal health (13), enhancing immunity (14, 15) and reducing colonization of pathogens in the GIT (16).       The intestinal tract is a crucial organ for digestion and absorption of feed nutrients in animals, closely linked to a range of physiological and biochemical processes, contributing to the animal’s nutrition and overall health (17). The intestinal microbiota is critical for maintaining host intestinal health, acting as the organism’s “second brain,” preventing pathogen colonization, modulating the intestinal epithelial barrier and inflammatory response (18, 19). The colonization of gut microbiota is relatively stable, and their composition and structure are influenced by factors such as diet, age, and feeding methods in animals (20). Changes in the GIT microbial community impact feed efficiency, productivity, and the health of chickens (21). The cecum, with the highest microbial colonization and the most abundant and diverse microbial community in the broiler gut, is crucial for the overall performance in poultry (22, 23). Generally, the chyme stays in the cecum for an extended period, allowing thorough decomposition of many feed ingredients by microbiota to improve nutrient utilization (24). Therefore, the balance of cecum microbiota is essential for enhancing overall performance of poultry.       Several recent studies have confirmed that GUE (25, 26) and probiotics (27, 28) promote growth and improve intestinal health in broilers. However, the potential effects of the combination of GUE and Lactobacillus on growth performance and intestinal health are not yet fully understood. Thus, the purpose of the present study was to investigate the impact of GUE, Lactobacillus acidophilus and their combination on growth performance and intestinal health in Liangfenghua broiler chickens, a medium-growing broiler strain known for its popularity due to the excellent meat quality. The findings of this study would be beneficial in comprehending the intestinal microecology of chickens and offering novel perspectives on the upkeep of intestinal wellness in broiler chickens that are fed antibiotic-free diets.'}, {'pmcid': '11289428', 'doi': '10.1002/hsr2.2275', 'introduction': 'N/A'}, {'pmcid': '11288818', 'doi': '10.3389/fonc.2024.1416806', 'introduction': '1       Introduction       Colorectal cancer (CRC), which includes cancer of the colon and rectum, has a high incidence and mortality rate, which seriously affects people’s physical health. According to the latest epidemiological data in 2020, the number of new cases of CRC in the world is approximately 1.9 million, and 935,000 deaths, ranking as the third and second among all malignant tumors (1). Both the incidence and mortality of CRC are still increasing year by year. It is estimated that by 2030, the global burden of CRC is expected to increase by 60%, with more than 2.2 million new cases and 1.1 million cancer deaths (2).       The incidence of CRC is on the rise in developing countries, while it has stabilized or declined in developed countries where the incidence remains high (2, 3). This is due to the early implementation of population-specific CRC early screening and lifestyle management in developed countries. As we all know, the occurrence and development of CRC mostly follow the “adenoma–cancer” sequence, and the whole development process generally lasts 5–10 years (4). Timely detection of precancerous lesions and intervention treatment can effectively prevent the occurrence of CRC and improve the prognosis (4).       The increasing number of CRC cases over the past decade indicates that in addition to genetic factors, unhealthy dietary habits, such as high intake of red meat, processed meats, refined grains, sugary beverages, desserts and potatoes (5); lack of fiber intake; and a high-fat diet, are also important factors in the development of CRC (6). In addition, intestinal flora is also a specific environmental risk factor for CRC (7). Relevant clinical studies have found that the intestinal flora colonized in the colorectal directly or indirectly participate in the body’s digestion and absorption, substance metabolism, immune regulation, and other physiological processes (8). Moreover, it can maintain the stability of intestinal microecosystem (8). The status of intestinal flora is closely related to the occurrence and development of CRC. A large number of animal models and clinical studies have proven that changes in the composition and quantity of intestinal flora promote the occurrence and development of CRC (9).       In recent years, the research on intestinal microecology has gradually increased, and more and more researchers have paid attention to this field. Studies on the relationship between intestinal flora and CRC are also increasing. This paper reviewed the correlation between intestinal flora and CRC, as well as the application of intestinal microecology in CRC screening and diagnosis to provide evidence for early screening and treatment of CRC.'}, {'pmcid': '11285316', 'doi': '10.1186/s12864-024-10621-7', 'introduction': 'N/A'}, {'pmcid': '11281493', 'doi': '10.3390/vetsci11070324', 'introduction': '1. Introduction       Research on the microbiota of the female reproductive tract in horses using high-throughput sequencing is a developing area of study, and little is known about it compared to other mammals. This niche has received limited attention in the equine species, particularly in comparison with other body regions, such as the gastrointestinal tract [1,2,3,4]. However, there is increasing evidence that the reproductive tracts of mammals harbour distinctive bacterial communities [5,6,7]. High-throughput sequencing studies have supported those using culture-dependent methods, suggesting that the uterus may not be a sterile environment [7]. However, the results of the culture-dependent method studies were limited, particularly in underestimating the abundance of certain bacteria, but the newer sequencing technologies allow researchers a broader identification of the bacteria present in this body system [8]. Despite this, few studies have characterized the microbiota of the uterus and vagina of healthy mares, particularly comparing both organs within the same animal. Most published studies currently use 16s rRNA sequencing to identify the bacterial microbiota, technical limitations or confounding factors can lead to discrepancies in the results, especially when determining which bacterial communities and at what proportion they are present in the uterus and vagina. It is difficult to establish a consensus on the normal microbiota of the reproductive tract in healthy mares unless sequencing is performed at a very deep level (shotgun sequencing), which is yet to be described in mares. In addition, determining a normal or core microbiota present in the reproductive tract of mares can be challenging because relevant factors such as the sampling site, the stage of the estrous cycle, age, nutritional regime, systemic disease, previous administration of antimicrobial drugs, season, and geographic location among other factors may impact the reproductive host’s health and therefore the microbiota. Therefore, this narrative literature review aims to summarize the current information regarding the microbial communities present in the reproductive tract of mares, the factors associated with their development and establishment, and their association with fertility, health, and disease.'}, {'pmcid': '11279407', 'doi': '10.3390/pathogens13070583', 'introduction': '1. Introduction       When gastrointestinal pathogens are detected in stool samples of patients with infectious gastroenteritis living in regions where such pathogens as well as associated infectious gastroenteritis are frequent, it can be difficult to say whether the detected pathogen is really the cause of the observed clinical symptoms. This is particularly true if more pathogens than just a single one are detected in the same stool sample. The problem that most enteric pathogens in regions with high prevalence for infectious gastroenteritis can both cause enteric disease or just persist as harmless colonizers is called “facultative pathogenicity” [1,2,3,4,5]. Previously, other researchers have tried to link the quantity of pathogens in stool samples with their likelihood of causing infectious gastroenteritis in such patients. In the case of diagnostic real-time PCR, so-called cycle threshold (Ct) values are an indirect option for target quantification because low Ct values indicate high quantities of the PCR target and vice versa. However, such attempts at linking pathogen quantity in stool samples with the likelihood of this pathogen causing clinical disease were only partly successful [2,3,6]. Consequently, generally accepted cut-offs for a Ct-value-based assignment of etiological relevance of enteric pathogens do not exist.       Obviously, more factors influence the likelihood of an association between clinically observed infectious gastroenteritis and the detection of an enteric pathogen in a human stool sample. One of these factors is “semi-immunity”, which means immunological adaptation of the human gut to rapid cycles of repeated pathogen exposure under poor hygiene conditions. This has been observed repeatedly in resource-limited tropical regions [7,8,9]. Although it is not yet completely understood how enteric semi-immunity works on a cellular or molecular level, some well-defined hypotheses regarding host–pathogen coexistence have been proposed, particularly for enteric helminth infections, as summarized elsewhere [10]. Furthermore, the semi-immunity concept is already in preventive medical use. Oral typhoid fever vaccination is the most commonly known example of induced short-term immunity on enteric mucous membranes [11].       Next to immunological adaptations, the enteric microbiome’s composition is believed to play a role in the degree of susceptibility towards the virulence of enteric pathogens. For both laboratory animals and human individuals, favorable microbiome compositions have been demonstrated to mitigate colonization resistance towards enteric pathogens, thus making gastroenteric infections less likely [12,13,14]. In addition, there is an increasing body of evidence suggesting likely interaction between enteric pathogens and commensals, which affects the clinical outcome [5,15].       In order to contribute to deciphering such microbial interactions in the gut of individuals from high-endemicity settings for infectious gastroenteritis, our study group recently conducted a cluster analysis assessing gastroenteric pathogens in stool samples of a Colombian indigenous population [16]. Within this population, a cluster consisting of Blastocystis hominis, Campylobacter spp., and Giardia duodenalis was shown to interact with Dientamoeba fragilis and Ascaris lumbricoides in a microbial-density-dependent way [16]. During the interpretation of this finding [16], however, it remained uncertain whether this observation just represented a regional peculiarity or a general pattern of microbial interaction in the human gut.       As such, it seemed promising to repeat the analysis with populations from other tropical high-endemicity settings for gastroenteric infections. Ghana, in the West African region, is an example of a country where enteric pathogens can be detected both in association with diarrheal disease and in asymptomatic individuals [4,17]. Especially high detection rates of gastroenteric pathogens can be expected in Ghanaian children [18,19,20], and in older studies, when diagnostic approaches with poor diagnostic accuracy, like Widal testing, were still in use [21], even underestimations of the true prevalence were likely. In such populations, co-occurrence of various microbial agents in stool samples as well as rapid pathogen acquisition cycles have been reported previously [22,23]. Furthermore, the common co-occurrence of genetic resistance determinants in Ghanaian stool samples [24,25] bears the risk of resistance transmission to bacterial enteric pathogens via mobile genetic elements. For Campylobacter spp. detections in Ghana, resistance was pronounced in cases of HIV-positive patients [26]. Enteric protozoan parasite infections of the gut have been reported to be particularly frequent in Ghanaian patients with diabetes [27]. Ghanaian farm environments provide reservoirs for several enteric pathogens like, e.g., salmonellae [28] and Cryptosporidium spp. [29]. Fecal contamination of regional environments is common [30], and food-borne or water-borne transmissions of infectious gastroenteritis are frequent events, particularly for poor Ghanaians [31,32,33]. Consequently, evidence of long-term efficiency of water filtration has recently been confirmed for Ghana [34], and, in line with the abovementioned information, colonized food vendors play a relevant role for such food-borne transmission events in Ghana [35].       Based on such previous experiences, Ghana was chosen as a suitable candidate region for cross-checking the experience from Colombia [16] in another tropical high-endemicity setting. To do so, cluster analyses were performed both with the set of microbial parameters chosen for the Colombian assessment [16] alone as well as in comparison with the Colombian results published elsewhere [16] and also with a broadened dataset available for the Ghanaian samples only. Furthermore, the dataset on Ghanaian individuals was sub-divided into subsets comprising children under 2 years of age and HIV-positive individuals. The rationale of these analytical steps is as follows. If microbial interactions are stable, comparable clustering should appear despite regionally different populations and despite interindividual differences, including factors like medical conditions and environmental factors. In summary, the study aimed at identifying stable clusters as markers of microbial interaction with potential importance for etiological relevance assignment in cases of multiple enteric pathogen detections, and the inclusion of different subpopulations, including children and HIV-positive individuals, was performed to further challenge the stability of potentially observed clustering.'}, {'pmcid': '11278787', 'doi': '10.3390/microorganisms12071328', 'introduction': '1. Introduction       Water is commonly contaminated by enteric bacteria, which inhabit the human gastrointestinal tract and are excreted in fecal matter [1]. Consuming water contaminated with these microorganisms can lead to intestinal infections [2]. Total coliforms are a key indicator group for assessing water quality [3].       Traditional methods for detecting and enumerating coliform bacteria typically involve lengthy processes, including enrichment with culture media in the laboratory [4]. However, advancements in molecular techniques have introduced faster, more sensitive, and more specific methods for microbiological water analysis [5], many of which can be automated. These new techniques enable the identification of pathogens, including novel and emerging strains [6]. Consequently, portable biosensors are now being discussed for their high sensitivity in detecting low coliform bacteria concentrations in water samples [2].       Conventional enumeration of viable pathogens in the water relies on fecal indicator bacteria culture methods using selective media. Research has shown that bacteria, including pathogens like Escherichia coli, can enter a viable but non-culturable (VBNC) state under environmental stress and starvation conditions [7,8,9]. This state affects the accuracy of traditional enumeration methods for viable bacteria in water supplies and wastewater treatment systems. RT-PCR also quantifies the target organism’s DNA in real time, providing information on the concentration of pathogens in the sample. However, these methods can overestimate viable bacteria counts due to the persistence of DNA after cell death [10,11]. To address this, mRNA detection via RT-PCR has been developed to test the viability of VBNC bacteria [12,13].       Water quality testing often focuses on detecting Escherichia coli, considered the best indicator of fecal contamination [14,15,16,17,18]. According to WHO guidelines, water poses an intermediate risk when E. coli counts range from 10 to 100 CFU per mL and a high risk when counts are between 100 and 1000 CFU per mL [14].       Preferred methods for microbiological water quality testing include membrane filtration and colony counts. E. coli confirmation typically employs immunoassays (ELISA) and PCR with species-specific DNA primers [17]. Traditional culture-based methods allow for sufficient bacterial growth for detection and enumeration for at least 24 h [19]. These methods involve culturing the bacteria on selective media, followed by colony counting and biochemical testing to determine the concentration of viable cells in the sample [20]. Although practical and low-cost, these methods are time-consuming, less sensitive, and less specific.       Slow and time-consuming traditional methods can delay disease diagnosis and treatment [20]. They are also expensive and require highly trained personnel. Nucleic-acid-based strategies, particularly PCR, offer high sensitivity and rapid detection but are labor-intensive, time-consuming, and costly [21,22]. Despite these drawbacks, nucleic-acid-based methods are powerful tools for pathogen detection in food, overcoming long enrichment stages due to their ease of use and automation potential.       The current molecular methods based on microarrays, nucleic acid sequence amplification, PCR, PCR hybridization, and real-time PCR allow for the detection of various pathogens in the same test. These techniques enable high specificity and sensitivity rates, ensuring rapid and accurate identification of multiple pathogens [23]. In addition, using PCR for pathogen detection requires knowledge of target DNA sequences to design specific primers [24], which limits the technique’s detection capacity.       Multiplex PCR enables the detection of various pathogens, including E. coli strains and their verotoxins. Additionally, this technique can offer high sensitivity and specificity, making it a valuable tool for food safety testing [25]. However, inhibitors in samples can affect the technique’s sensitivity and specificity, leading to false negatives or positives. Additionally, multiplex PCR requires careful validation for each pathogen and toxin.       ELISA is a widely used technique in immunology and diagnostics due to its ability to detect and quantify specific E. coli antigens or antibodies in biological samples. ELISA typically takes about 24 h from sample preparation to results [26,27], though this can vary depending on sample preparation, incubation times, and detection methods [28,29]. ELISA quantitatively determines antigen concentrations in samples, with signal intensity from labeled antibodies correlating with the antigen amount. This quantitative aspect makes ELISA powerful in clinical diagnostics, immunology research, and biotechnology [30,31]. The method involves immobilizing the target antigen onto a solid substrate, typically a plastic surface, followed by detection using a labeled selective antibody [32]. The sensitivity of ELISA can be influenced by various factors, including the quality of antibodies used, the efficiency of antigen capture and detection, and the presence of interfering substances in the sample matrix. Therefore, validating the ELISA method thoroughly and comparing its performance against reference methods is essential to ensure reliable results.       Capture ELISA (cELISA) is a widely used technique for detecting E. coli. Through this method, antibodies that specifically target E. coli are fixed to a plate. When a sample containing E. coli antigens is introduced, these antibodies capture and bind the antigens, allowing for their detection. After capturing the antigens, primary antibodies specific to E. coli are added, followed by secondary antibodies. These secondary antibodies are linked to an enzyme, such as horseradish peroxidase (HRP), or directly conjugated to the detection molecule. Although cELISA offers rapid detection, it may sacrifice some sensitivity compared to more time-consuming methods, balancing speed and accuracy for various applications [33].       Direct ELISA (iELISA) simplifies assays by requiring only one antibody—the primary antibody binding to the target antigen—compared to cELISA, which involves two antibodies [30]. This approach streamlines the experimental setup and is suitable when a specific antibody is available for the antigen of interest.       Biosensors provide advantages over conventional detection methods, including speed and sensitivity [34,35]. They offer high specificity, detecting specific pathogens in water and food samples with precision [36]. Biosensors offer multiplexing. Additionally, biosensors can target various pathogens or contaminants simultaneously, reducing the timing of analysis and providing detection accuracy by providing comprehensive information about the sample’s microbial or contaminant profile [37], making them valuable tools in food safety and environmental monitoring.       Optical biosensors provide rapid and efficient responses for ensuring food safety, particularly in scenarios where timely detection is crucial to prevent outbreaks or minimize economic losses [38]. They can be integrated into automated systems, enabling real-time monitoring of low levels of target analytes and providing high sensitivity and specificity in food samples. In addition, optical biosensors offer portability and ease of use, making them well-suited for deployment in field environments [38].       Integrating photonic biosensors onto silicon integrated circuits or PICs revolutionizes point-of-care diagnostics by offering compact, sensitive, and multiplexed detection platforms highly compatible with CMOS technology [39]. Sensitivity is crucial for biosensor performance, as indicated by limits of detection (LoD) and quantification (LOQ). Enhanced sensitivity improves biosensor efficacy in detecting and measuring specific substances [40]. The sensor’s sensitivity is determined by how effectively the evanescent field interacts with the sample molecules [41,42].       Microfluidic-based biosensors with microchannels for fluidic samples are becoming increasingly relevant due to their on-chip immunoassay capabilities [43]. These systems perform various laboratory processes concurrently on a single chip, including detection, sampling, separation, and mixing [44,45]. Besides being extremely sensitive [46], biosensors can be incorporated into portable sensing systems [47]. Portable biosensors can determine spatio-temporal variations in water quality by deploying sensors in water sources or installing them at the point of source [47].       Ultra-sensitive biosensors with low LoD can detect single microbial cells. While this sensitivity is ideal for assessing water quality, the uneven spread of microbial cells in large volumes of water (e.g., lakes, rivers, wells) can affect accurate cell count. This criterion must be considered when designing biosensors for water quality assessment.       While optical microfluidic biosensors demonstrate competitive sensitivities, they still face challenges that must be addressed, including high costs and complex procedures for integrating optical components with microfluidic channels and functionalized surfaces [48]. Current E. coli biosensors still involve labor-intensive manual procedures, which can limit their practical use in laboratory settings.       This research aims to optimize a photonic immunosensor capable of detecting both viable and non-viable forms of E. coli in water samples for human consumption. The detection method uses ring resonator transduction within functionalized Photonic Integrated Circuits (PICs) coated with specific antibodies targeting lipopolysaccharides (LPSs) from the outer membrane of E. coli or E.-coli-specific β-galactosidase, a peptide carrier protein with biological activity. By enabling early detection and quantification of viable and non-viable bacteria, this technology promises to predict pathogen contamination in water samples, addressing current limitations in pathogen detection systems within the agrifood industry and environmental sector.'}, {'pmcid': '11273848', 'doi': '10.3390/antibiotics13070613', 'introduction': '1. Introduction       Staphylococcus aureus, a member of the ESKAPE (Enterococcus faecium, S. aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa, and Enterobacter species) group of pathogens, is a leading culprit of nosocomial infections [1]. The emergence of biofilm-forming, multidrug-resistant S. aureus strains, particularly methicillin-resistant S. aureus (MRSA), poses a formidable challenge in healthcare and agricultural settings [2]. S. aureus can cause a wide range of infections and exhibit a remarkable ability to develop resistance against conventional antibiotics [3,4]. Its pathogenicity is largely attributed to its ability to form robust biofilms, which are microbial communities encased in an extracellular matrix that enables evasion of host immune responses, antimicrobial resistance, and tenacious surface adhesion [5,6]. Additionally, the multilayered biofilms are facilitated by the icaADBC operon, which facilitates the synthesis of an extracellular matrix [7]. S. aureus employs quorum-sensing (QS) mechanisms to coordinate the expression of virulence factors, exacerbating its pathogenicity. Combating biofilm formation and disrupting QS signaling have emerged as critical objectives in mitigating the impact of S. aureus infections, underscoring the urgent need for novel therapeutic approaches tailored for healthcare and veterinary applications [8]. The QS mechanism in S. aureus is governed by two main cascades: the staphylococcal accessory regulator (Sar) and the accessory global regulator (Agr). Within the Agr QS system, oligopeptides serve as signaling molecules to modulate the expression of various virulence factors. SarA (Staphylococcal accessory regulator A) is a pivotal regulatory protein in the S. aureus QS cascade, significantly impacting bacterial pathogenicity and resistance mechanisms [9,10]. It controls the expression of numerous virulence factors, influencing the transcription of gene-encoding toxins, enzymes, and surface proteins, thus playing a crucial role in the bacteria’s ability to cause infections ranging from minor skin conditions to severe diseases such as pneumonia and sepsis. SarA promotes biofilm formation by upregulating genes responsible for extracellular polysaccharide production and repressing proteases that could degrade the biofilm matrix, thereby enhancing bacterial resistance to antibiotics and host immune responses. Furthermore, SarA contributes to antibiotic resistance by regulating stress response and cell wall synthesis genes, including the methicillin resistance gene mecA. Given its central role in virulence and biofilm formation, SarA is a promising target for new therapeutic strategies aimed at attenuating S. aureus pathogenicity and enhancing susceptibility to antibiotics [8,11].       Bioactive phytoconstituents have emerged as promising agents for combating infectious diseases caused by bacteria that form biofilms. These compounds, derived from plants, exert their therapeutic effects by targeting genes responsible for disease progression, particularly by disrupting virulence factors and inhibiting biofilm formation associated with quorum sensing (QS). Traditional medicine has a rich history of utilizing plants to combat microbial infections [12,13]. Among these botanical remedies, Paederia foetida (known as Sembukan in Indonesian and King’s Tonic in English), a climbing herb belonging to the Rubiaceae family, stands out as one of Asia’s indigenous medicinal plants, thriving in both temperate and tropical regions. Despite its bitter taste and unpleasant odor, the leaves of this plant are traditionally employed to address a range of ailments, including diarrhea, rheumatism, inflammation, piles, dysentery, and stomachache [14,15,16]. This traditional wisdom serves as a valuable resource for uncovering potential drugs. Notably, compounds such as quercetin, catechin, rosmarinic acid, limonoid, ichangin, apigenin, kaempferol, and naringenin have demonstrated efficacy against biofilm-associated infections [17]. Consequently, there is a pressing need to identify potent QS inhibitors (QSIs), preferably sourced from natural reservoirs. The secondary metabolites found in plants have shown promise in effectively treating various infections [18].       Exploring anti-quorum-sensing and anti-biofilm agents from natural sources, such as plant extracts and biologically synthesized agents, offers promising avenues for controlling S. aureus infections [19,20]. Molecular docking simulations predict binding modes and affinities between bioactive compounds and key proteins in bacterial QS pathways, guiding the design of novel, potent QS inhibitors [21,22]. This interdisciplinary approach aligns with the One Health initiative, recognizing the interconnectedness of animals, humans, and the environment [23,24]. The present study investigated the antimicrobial, anti-QS, and anti-biofilm formation abilities of P. foetida Linn. leaf extract against S. aureus, focusing on managing infections in healthcare and veterinary settings while embracing the One Health approach to combat this global threat.'}, {'pmcid': '11272550', 'doi': '10.3389/fmed.2024.1428432', 'introduction': '1       Introduction       The treatment of fracture-related infections (FRI) is challenging. Particularly, the involvement of severe bone defects and multi-resistant bacteria frequently leads to complex situations. The treatment of FRIs mainly consists of two key aspects: First, control of the infection and causative microorganisms must be achieved. Second, bone reconstruction with subsequent successful bone healing needs to be obtained (1).       Both local and systemic antibiotic therapy are hallmarks of fracture-related infection treatment. However, there are often only limited antibiotic treatment options when multi-drug resistant bacteria are involved. An alternative or complementary approach for the treatment of complex infections with multi-drug resistant bacteria is bacteriophages (short phages). The bactericidal effect of phages was first identified more than 100\\u2009years ago by Felix d’Herelle (2). Since then, phages have been studied and used mainly in the former Soviet Union during the second half of the 20th century. In recent years, bacteriophage therapy has also gained interest in the Western world due to the rise of multi- and pan-drug-resistant bacteria, including difficult-to-treat bone and joint infections (3). However, there are only a few published case series or case reports on the treatment of musculoskeletal infections with bacteriophages (4). A recent review identified 33 bone and joint infection cases treated with phages, out of which notably 29 (87%) achieved either microbiological or clinical success. Furthermore, 8 of the 33 cases (24%) experienced mild and temporary adverse effects, but there were no reports of serious complications (5). In addition, underlining the efficacy of phage therapy in resolving difficult-to-treat infections, Pirnay et al. have just recently published clinical outcomes of their first 100 cases treated with personalized phage therapy. Bone infections were the three most common indications. Overall, in 77.2% of targeted infections, patients experienced clinical improvement. In 61.3% of infections where relevant bacteriological follow-up data were available, eradication of the targeted bacteria was observed (6). Ongoing clinical trials aim to refine phage application mode and advance our understanding of how to effectively manage bone infections. Currently there are a few clinical trials registered, which already entered the phase of recruiting patients for phage treatment.       Hitherto, however, the optimal administration of bacteriophages has not yet been elucidated. Phages can be administered intravenously or locally. Different approaches with varying quantity, frequency, and type of application have been described to date (4, 7). Recently, Ferry et al. published a case report with the use of hydrogel as carrier material for the application of bacteriophages in a patient with a mega prosthetic hip infection (8). This study showed a rapid release of the phages from the DAC hydrogel in an in vitro setting. However, no in vivo data for the release of the phages in the patient were presented.       Therefore, in this study, we present a clinical case involving the loading of bacteriophages into a hydrogel in an FRI. Additionally, we assess the in vivo release kinetics of the phages from the hydrogel in the patient’s wound using high-throughput metagenomic sequencing-based characterization of the administered bacteriophage cocktail, followed by real-time PCR quantification of selected bacteriophages in the drainage fluid.'}, {'pmcid': '11269248', 'doi': '10.3389/fmicb.2024.1423610', 'introduction': 'Introduction       The genus Yersinia contains 28 species, three of which are pathogenic for humans (access: January 2024).1 Yersinia (Y.) pestis is the causative agent of plague, transmitted by the bite of an infected flea. On the other hand, Y. pseudotuberculosis and particularly Y. enterocolitica are foodborne enteropathogenic species, which may cause gastrointestinal diseases termed yersiniosis (Le Guern et al., 2016). In 2022, yersiniosis was the third most commonly reported bacterial foodborne zoonotic disease in the European Union (EU) (European Food Safety Authority and European Centre for Disease Prevention and Control, 2023). Typical symptoms are fever, abdominal pain and diarrhea (Bancerz-Kisiel and Szweda, 2015). While human yersiniosis is mainly caused by Y. enterocolitica, Y. pseudotuberculosis is more often found in wild and zoo animals causing various infections, some of which are lethal and exhibit tuberculosis-like symptoms. By contrast, in Europe and many other countries, the presence of Y. enterocolitica is strongly associated with domestic pigs, which do not get ill, but represent a natural reservoir for the bacteria (Fredriksson-Ahomaa et al., 2006a,b; Bancerz-Kisiel and Szweda, 2015; European Food Safety Authority, 2015). Human infections by this species are predominantly linked to the consumption of raw or insufficiently cooked pork (Laukkanen et al., 2008). However, raw milk, water and vegetables such as mixed salad contaminated with Y. enterocolitica have also been reported as possible sources of infection (Jamali et al., 2015; MacDonald et al., 2016; Terech-Majewska et al., 2016).       Y. enterocolitica is divided into six biotypes (1A, 1B, 2, 3, 4 and 5) and more than 70 serotypes (O) (Bancerz-Kisiel et al., 2018). In Europe, the most clinically important bioserotype is 4/O:3, followed by 2/O:9 and 2/O:5,27. Biotype 1A strains frequently occur in the environment and food and are generally regarded as nonpathogenic, although some reports have isolated this biotype from sick individuals (Joutsen et al., 2016; Bancerz-Kisiel et al., 2018).       Since the use of antibiotics is generally avoided in food production, alternative methods are needed to reduce these bacteria along the food chain. One alternative approach to treating bacterial infections or reducing pathogens in food is the application of virulent (strictly lytic) phages, which can also be effective against multidrug-resistant bacteria (Xu, 2021). A number of phages lysing Y. enterocolitica has been described, most of them are podoviruses and myoviruses (Salem et al., 2015; Gwak et al., 2018; Salem and Skurnik, 2018). Many podoviruses infecting Y. enterocolitica are related to T7 and have a rather narrow host range as they mainly lyse O:3 strains (Pajunen et al., 2001; Leon-Velarde et al., 2014; Liang et al., 2016). An exemption is the T7-like phage vB_YenP_Rambo that lysed 50 out of 62 and 54 out of 62 pathogenic Y. enterocolitica strains belonging to several bioserotypes at 28°C and 37°C, respectively (Hammerl et al., 2022b). Y. enterocolitica myoviruses mostly exhibited a broader host range than podoviruses, because they could additionally lyse strains belonging to the serotypes O:9 and O:5,27. One myovirus (fHe-Yen9-01) that lysed 61.3% of 106 tested Yersinia strains was used for reduction experiments with food samples (pork and milk) and kitchenware and was able to reduce an O:9 strain significantly (Jun et al., 2018). The widest host range of all Yersinia phages described thus far showed the myoviruses PY100 and the very similar vB_YenM_P281. These two phages not only infected various bioserotypes of Y. enterocolitica (61.1% of 72 strains at 28°C and 95.8% at 37°C) but also strains of Y. pseudotuberculosis if the medium was supplemented with calcium and magnesium cations (Hammerl et al., 2022b). A combination of vB_YenM_P281 and vB_YenP_Rambo reduced an O:3 and an O:9 strain efficiently down to a temperature of 17°C. Phage-induced lysis of Enterobacteriaceae at low temperatures generally occurs much more slowly than at higher temperatures, such as 37°C, due to the limited propagation of the host bacteria. Additionally, higher multiplicities of infection (MOIs) are often required to achieve strong lytic activity, as shown for the phages vB_YenM_P281 and vB_YenP_Rambo, where a tenfold higher MOI was required for a significant reduction of the tested strains at 17°C (Hammerl et al., 2022b). The access to host cell receptors, important for binding of a phage may also change with the temperature.       In this study, we isolated and characterized three novel virulent myoviruses from game animals (vB_YenM_P744 and vB_YenM_P778) and sewage (vB_YenM_P8). These phages demonstrated excellent lytic activity at low temperatures and low MOIs, complementing each other well regarding their host range. We further evaluated a cocktail of these three phages, together with vB_YenM_P281 and vB_YenP_Rambo, which efficiently lysed a mixture of pathogenic Y. enterocolitica strains.'}, {'pmcid': '11265448', 'doi': '10.1128/msystems.00520-24', 'introduction': 'INTRODUCTION       The giant panda (Ailuropoda melanoleuca) is a rare and large endemic species in China and is one of the most adored and protected species worldwide. At present, there are more than 600 captive giant pandas according to the studbook of giant pandas of 2020 reported by the Chinese Association of Zoological Gardens. More than 1,800 wild giant pandas are found in the mountainous areas of Sichuan, Shaanxi, and Gansu provinces of China according to the Fourth National Survey of Giant Pandas completed in 2015. The threat level of giant pandas has been downgraded from \"endangered\" to \"vulnerable\" (http://www.forestry.gov.cn). However, giant pandas are often threatened by intestinal diseases, the main reason for mortality caused by ascarid nematodes and bacterial infections, such as Escherichia coli, Klebsiella pneumoniae, Clostridium perfringens (1–4). Thus, it is critical to identify efficient ways to prevent and cure intestinal diseases of this beloved species.       Some lactic acid bacteria (LAB) play a key role for the host animal by providing health benefits. Animal studies have shown that LAB significantly prolonged the life span of Caenorhabditis elegans (5) and enabled blood urate control in mice (6). LAB could potentially help to control some cancers (7, 8) and regulate the normal gastrointestinal microbiota, maintain the microecological balance, inhibit pathogen proliferation, and prevent inflammation (9–11).       Owing to the important physiological role of intestinal LAB, screening functional LAB from giant pandas is a good way to potentially use them as probiotics and to prevent and treat intestinal diseases. As of now, the isolated LAB strains from giant pandas mainly included Bifidobacterium (12), Lactobacillus (11, 13–16), and Weissella (17, 18). In these studies, some authors reported the characteristics and drug sensitivity of these strains (12, 17, 18), their survival at low pH and high bile salt concentrations, their activity against pathogens, and their ability to alleviate the inflammatory response (11, 13–15). However, more knowledge about LAB in giant pandas is needed to aid in conservation of giant pandas. A good way to analyze the diversity of LAB and to screen functional strains in giant pandas is to culture the microbiota with culturomics (19).       Cultivation is necessary to screen strains to experimentally investigate microbial traits, to confirm the activities of novel genes through functional characterization studies, and to identify potential health benefits for the development of probiotics (19–21). Culturomics is a comprehensive culture-based approach that applies multiple cultivation conditions to identify bacteria and genes that contribute to potential functions of interest (22, 23). The culturomics was used to successfully culture lower-abundance bacteria (24) and “uncultivable” bacteria (25) in human gut, and to unveil members of the swine gut microbiota (26).       In addition, symbiotic gut bacteria produce different classes of metabolites, such as short-chain fatty acids (SCFAs), which can not only serve as energy substrates but also as signaling molecules with diverse functional roles, including protection of the gut barrier function, modulation of energy metabolism, alteration of immune mechanisms, regulation of metabolic homeostasis, and neuroprotective effects for combating disease and improving health (27, 28).       Thus, in this study, we used culture-independent and culture-enriched methods to reveal the diversity of LAB in giant pandas, compare the difference between the two methods, and establish a pipeline to screen novel strains with the potential for use as probiotics following the combination of liquid and solid cultures. This study provides guidance for future giant panda gut microbiota cultivation.'}, {'pmcid': '11263920', 'doi': '10.2903/j.efsa.2024.8896', 'introduction': 'N/A'}, {'pmcid': '11262882', 'doi': '10.1155/2024/8772009', 'introduction': 'N/A'}, {'pmcid': '11261454', 'doi': '10.1016/j.psj.2024.103975', 'introduction': 'N/A'}, {'pmcid': '11260827', 'doi': '10.3389/fmicb.2024.1394463', 'introduction': \"1 Introduction       The animal gut houses a wide range of microbes (Bankers et al., 2021; Levin et al., 2021), which play crucial roles in various aspects of the host digestion (Francoeur et al., 2020), detoxification (Levin et al., 2021), and immune system (Xiong et al., 2019). Despite the importance of these functions for most animals (Moran et al., 2019), the processes that shape gut microbiota remain largely unexplored, particularly in the case of freshwater snails. Understanding the mechanisms shaping snail gut microbiota is a crucial first step in developing microbial approaches to control invasive snails and preserve native snails.       After years of research, we have gained some insights into the gut microbiota of freshwater snails, with a primary focus on their diversity, composition, and functions (Liu et al., 2018; Chen et al., 2021). Studies have revealed that the gut microbiota in freshwater snails is shaped by both host-related factors, such as species (Zhou et al., 2022), sexes (Chen et al., 2021), and developmental stages (Chen et al., 2021), and environmental factors like season (Li et al., 2022b), temperature (Li et al., 2022a), and toxins (Liu et al., 2023). These studies have systematically uncovered the variations in the gut microbiota of freshwater snails. Among these, the invasive apple snail Pomacea canaliculata attracts attention not only as a pest of rice (Halwart, 1994; Attademo et al., 2016) but also for its ecological advantages in invaded habitats (Fang et al., 2016; Liu et al., 2021b). Research on this snail indicates that its gut microbiota may facilitate adaptation to new environments (Li et al., 2022b; Zhou et al., 2022), thereby providing a basis for developing invasive management strategies that leverage snail gut microbiota. However, there is still much to be explored to harness the potential of gut microbiota, such as understanding the assembly processes of freshwater snail gut microbiota.       According to the current consensus of community assembly, both deterministic and stochastic processes jointly govern the assembly of communities (Zhou and Ning, 2017; Ning et al., 2019, 2020). The neutral theory highlights the stochastic processes, like random birth or death events, immigration, and ecological drift (Chave, 2004), while the niche-based theory assumes that deterministic processes, including environmental filtering and biological interactions (Chesson, 2000). Quantifying the importance of these two processes in governing microbiota assembly has sparked significant scientific interest (Liu et al., 2020; Wang et al., 2020; Ge et al., 2021). In the study of gut microbiota, these processes have also been extensively investigated (Burns et al., 2016; Zhu et al., 2016; Xu et al., 2023) and are considered ecology and evolution significant (Kohl, 2020; Ge et al., 2021). For instance, researchers have found that changes in the importance of these processes can effectively explain the differences in the composition of gut microbiota from different honeybee species and geographical sites (Ge et al., 2021). However, there has been limited research focusing on the potential association between community assembly and gut microbiota diversity. The diversity of animal gut microbiota is thought to be influenced by host filtering, which includes factors such as the immune system and stomach pH (Reese and Dunn, 2018). Host filtering functions as the host's selection for specific microbes (Ley et al., 2006; Kohl, 2020), corresponding to deterministic processes (selection) in community assembly (Xiong et al., 2017; Xiao et al., 2021). It can be inferred that stronger host filtering may lead to a more deterministic assembly, ultimately limiting the diversity of gut bacterial communities. One example of this is that a lower stomach pH can act as a filter, potentially reducing the diversity of gut microbiota (Reese and Dunn, 2018).       Thus, the core objectives of this study are to elucidate how assembly processes shape freshwater snail gut microbiota. We investigated the gut bacterial communities of two freshwater snails, the invasive snail (Pomacea canaliculata) and the native snail (Viviparidae), co-existing across different sites and thus exposed to the same pool of potential microbial colonizers. Although previous studies have reported on the gut bacterial communities of these two snails, their assembly patterns remain unknown, which is crucial for understanding how bacteria coexist in snail gut. To achieve this, we first comprehensively compared the differences in diversity and composition of the gut microbiota between these two snails. Subsequently, we attempted to decipher the observed differences, especially in diversity, using community assembly theory.\"}, {'pmcid': '11257890', 'doi': '10.3389/froh.2024.1425937', 'introduction': \"1       Introduction       Periodontitis is one of the most prevalent microbially-induced disorders worldwide (1). The primary cause of this inflammatory disease, which affects the tissues supporting the teeth, is the presence of a dysbiotic microbial community in the subgingival crevice (2). In a healthy state, subgingival biofilms are formed predominantly by commensal microbes; however, periodontal pathogens may elevate the pathogenicity of the whole community and trigger an exacerbated inflammatory response that leads to tissue destruction and ultimately tooth loss (3, 4). Periodontitis is linked to numerous systemic conditions, such as rheumatoid arthritis, cardiovascular disease, diabetes, cancer and Alzheimer's disease (5–7). The microbial aetiology of periodontal disease has long been established, and among the periodontal pathogens, three species are most often associated with disease development—Porphyromonas gingivalis, Tannerella forsythia and Treponema denticola (8, 9). These bacteria employ an array of virulence factors to establish their pathogenicity within the oral cavity (10, 11), including several proteolytic enzymes. In the case of P. gingivalis, gingipains—aggressive cysteine proteases (RpA, RgpB, and Kgp)—are attaining increasing interest for their wide-reaching effects on the host immune system and systemic diseases (12). For example, they exert proteolysis at the cell surface of human gingival epithelial cells (13) and enhance blood-brain barrier permeability (14). Similarly, for T. forsythia, a plethora of proteases has been linked to the bacterium's pathogenicity, including, e.g., PrtH proteases transferring attachment loss (15), a trypsin-like cysteine protease with both arginine- and citrulline-specific activities (16), and secretory KLIKK proteases exhibiting activity towards diverse protein substrates, including collagen, gelatine, elastin and casein (17).       A less-investigated class of proteases in oral pathogens are HtrA (high-temperature requirement A) enzymes. For pathogens, HtrA family members are generally crucial for survival, adaptation to environmental changes, and tolerance of harsh conditions such as elevated temperatures (heat-shock response), extreme pH and oxidative or osmotic stress (18). In several cases, loss of HtrA function correlates with reduced virulence and restricted bacterial growth under external stresses (19). Thus, bacterial HtrAs are regarded as promising candidates for developing novel antibacterial strategies (20). Mechanistically, HtrAs have ATP-independent dual chaperone-protease activity and mediate protein quality control (19, 21). Members of the HtrA protein family consist of a conserved chymotrypsin-like serine protease domain with the catalytic triad composed of a histidine, aspartate and serine residue (22), and at least one C-terminal regulatory PDZ (post synaptic density of 95\\u2005kDa, discs large, and Zonula Occludens 1) domain for substrate recognition, binding and oligomerization (21, 22). The two major structural subdivisions within this protein family are the DegP/O and DegS enzymes, which ensure protein quality control in the periplasm of E. coli (21). In addition to the serine-protease domain, they contain either two PDZ domains and an N-terminal signal peptide for cytoplasmic membrane translocation (DegP/O), or a single PDZ domain preceded by an N-terminal transmembrane segment for membrane anchoring (DegS) (18). Other HtrA proteins can be translocated to the cell surface or secreted, enabling them to participate in bacterial colonization or host invasion (22, 23). It is proposed that the extracellular activity of HtrAs involves their transport through outer membrane vesicles (22).       HtrA enzymes are crucial for the virulence of several important pathogens due to their diverse functionalities. These pathogens include E. coli (24), Listeria monocytogenes (25), Legionella pneumophila (26), Borrelia burgdorferi (27), Helicobacter pylori (28), Campylobacter jejuni (29), and Bacillus anthracis, among others. In H. pylori, HtrA acts as a secreted virulence factor that cleaves the ectodomain of the cell-adhesion protein E-cadherin leading to E-cadherin shedding and disruption of the epithelial barrier functions, allowing H. pylori to access the intercellular space (30). In C. jejuni, secreted HtrA facilitates transepithelial migration by degrading E-cadherin (31). A C. jejuni deletion mutant lacking HtrA was shown to induce lower levels of apoptosis and reduced secretion of the pro-inflammatory cytokines monocyte chemoattractant protein (MCP)-1, interleukin (IL)-6, tumour necrosis factor (TNF)-α, and interferon (IFN)-γ in a mouse infection model, underscoring the protease's role in the bacterium's virulence (32). Recently, it was found that surface-bound HtrA of C. jejuni, but not secreted HtrA, disrupts epithelial cell-to-cell junctions (33). In B. anthracis, HtrA plays a regulatory role, influencing expression of more than 1,000 genes under stress (34). A B. anthracis ΔhtrA deletion mutant was unable to proliferate in macrophages and impaired their lysis (35). In the context of periodontitis, in P. gingivalis, HtrA was shown to regulate gingipain activity and adaptation to oxidative and long-term heat stress (36, 37). The protease also influenced bacterial survival in a mouse model, and a P. gingivalis ΔhrtA-deletion mutant displayed cell-dependent effects on invasiveness compared to the P. gingivalis W83 parent wild-type strain, with increased invasion of epithelial cells, while invasion of endothelial cells was unaffected (37). These findings suggest a prominent role of HtrA in the interaction of this periodontal pathogen with the host.       In this study, we obtained evidence that T. forsythia ATCC 43037 produces a proteolytically active HtrA ortholog. We identified the htrA gene in T. forsythia ATCC 43037, produced the predicted HtrA recombinantly in E. coli cells, and demonstrated proteolytic activity of the T. forsythia enzyme on different substrates in vitro. Furthermore, we constructed a T. forsythia ΔhtrA deletion mutant. Using this mutant along with the T. forsythia parent strain and recombinant HtrA, we conducted infection studies to assess the enzyme’s effect on cellular viability and pro-inflammatory cytokine production in human gingival fibroblasts and macrophages.\"}, {'pmcid': '11249912', 'doi': '10.34133/research.0415', 'introduction': 'N/A'}, {'pmcid': '11246856', 'doi': '10.3389/fnut.2024.1407736', 'introduction': '1       Introduction       Helicobacter pylori (H. pylori) infection is a global health problem, resulting in chronic inflammation and the increased risk of peptic ulcer disease, atrophic gastritis, gastric carcinoma, and mucosa-associated lymphoid tissue (MALT) lymphoma (1). The global pooled prevalence of H. pylori infection has been reported to be 48.5%, ranging from 70.1% in Africa to 24.4% in Oceania (2). A recent meta-analysis reported that the prevalence of H. pylori infection in mainland China was 40.0% during the period 2015–2019, highlighting China as one of the countries with a substantial burden of H. pylori infection, despite a declining trend from 2000 to 2019 (3). Besides, considering the high resistance rates of clarithromycin and metronidazole (4), bismuth quadruple therapy is recommended as the first-line treatment in China (5), including 2 antibiotics (with five different regimens), a standard dose of a proton pump inhibitor (PPI), and bismuth (6). Both antibiotics and PPIs are known to disrupt the gut bacterial equilibrium, and the bacterial shifts caused by eradication therapy may contribute to the disturbance of carbohydrate and lipid metabolism (7). Consequently, it is necessary to develop a therapeutic regimen for H. pylori infection that not only achieves a satisfactory eradication rate but also minimizes the impact on the gut microbiota.       Fucoidan is a complex sulfated polysaccharide extracted from various species of brown seaweed (8). There is a growing body of evidence suggesting that fucoidan possesses various physiological properties, including high antioxidant activity (9), anti-tumor effect (10), and anti-allergic property (11). Besides, fucoidan can exert anti-inflammatory effects by modulating neutrophil migration and the expression of genes related to COX-2 and the NF-κB signaling pathway (12). Therefore, fucoidan is regarded as a promising component in functional food or adjuvant therapy. Fucoidan has two primary structural backbones: Type I, such as fucoidan from Laminaria japonica, is composed of repeated α (1\\u2009→\\u20093)-linked α-L-fucopyranose, with fucose-linked sulfate groups in C2- and C4- positions; Type II, such as fucoidan from Ascophyllum nodosum, is composed of alternate α (1\\u2009→\\u20093)- and α (1\\u2009→\\u20094)-linked α-L-fucopyranose, with fucose-linked sulfate groups in C2-, C3-, and C4-positions (13). The sulfate groups are important determinants for the bioactivities of fucoidan, depending on the sulfate content and position of the groups (14).       Research in a rat model of breast cancer has indicated that fucoidan intervention can enhance the diversity of the intestinal microbiota and change the microbiota composition, including a decrease in the abundance of Collinsella, Coprococcus, Oscillospira, and Sporosarcina as well as an increase in the abundance of Parabacteroides and Blautia (15). In an ulcerative colitis mice model, fucoidan was also found to significantly increase the abundance of Ruminococcaceae and Clostridium-IV (16). Moreover, fucoidan has been demonstrated to increase several beneficial bacterial taxa and decrease several harmful taxa in mice intervened with ciprofloxacin-metronidazole (17), indicating a potential capacity to mitigate dysbiosis caused by antibiotics, which is common during the eradication therapy of H. pylori infection. Nevertheless, data on the effects of fucoidan on the human gut microbiota are limited. In patients with chronic superficial gastritis, a combination of wheat peptides and fucoidan was reported to relieve symptoms, reduce gastric mucosal damage, and regulate gut microbiome, promoting the growth of Bifidobacterium pseudocatenulatum, Eubacterium siraeum, and Bacteroides intestinali (18), but the isolated effects of fucoidan were not evaluated.       Both in vitro and in vivo studies have shown that fucoidan can prohibit H. pylori adhesion to host cells, reduce cytotoxic effects in H. pylori, and decrease the infection rate of H. pylori in a dose-dependent manner (19). However, clinical data regarding the effects of fucoidan on H. pylori infection remains limited. Thus, we conducted a prospective, randomized, open-label trial in patients with H. pylori infection. The aim was to assess the effects of fucoidan supplementation, either alone or in combination with synbiotics supplementation, on the eradication rate and the maintenance of gut microbiota homeostasis in these patients.'}, {'pmcid': '11244529', 'doi': '10.1126/sciadv.adn1745', 'introduction': 'N/A'}, {'pmcid': '11243892', 'doi': '10.3390/s24134266', 'introduction': '1. Introduction       Nucleic acid amplification tests (NAATs) are key tools for the screening, risk assessment, diagnosis, therapy selection, prognosis, and monitoring of many diseases [1]. With the recent COVID-19 pandemic, the need for rapid and accurate nucleic acid testing has been shown in recent years [2,3,4]. While conventional real-time polymerase chain reaction (PCR) [5] methods have demonstrated tremendous success for qualitative analysis as well as semi-quantitative analysis, achieving the absolute quantification of target sequences is challenging. The impact of sample matrices and PCR inhibitors often undermines the reliability of copy number quantification. To overcome this limitation, digital PCR was developed, in which the bulk volume is divided into many smaller portions. Each of these portions containing either 0 or 1 copy of the target sequence is then amplified and read individually to determine the “negative” and “positive” portions [6,7,8]. Absolute quantification is then achieved based on the Poisson distribution. To date, digital PCR has been used in diverse applications, including pathogen detection, rare sequence detection, and liquid biopsy.       Despite the tremendous success in laboratory-based tests, developing point-of-care (POC) digital PCR technology has been challenging. To simplify the amplification process and eliminate the need for a thermal cycler, many isothermal amplification methods have been used for digital amplification. Notably, loop-mediated isothermal amplification (LAMP) [9] is the most widely performed reaction due to the high specificity and sensitivity of using loop structures to encourage the amplification of the target. To date, many digital LAMP (dLAMP) methods have been reported depending on different mechanisms for achieving sample compartmentalization. The first class of methods utilizes pre-formed or prefabricated structures for distributing bulk samples into small volumes. The use of gel arrays with LAMP reagents embedded inside for amplification has been reported as a possible method for dLAMP [10,11,12,13]. Similarly, dLAMP has been performed in commercially available membranes using the pores as a microwell array [14,15,16]. In addition, slipchip technology [17,18,19,20,21,22,23] and self-digitizing microfluidic chips [24,25,26] have been utilized to integrate additional fluid operations to distribute the bulk solution for dLAMP. In addition to these microwell-based dLAMP methods, water-in-oil emulsions (droplets) have also been used for dLAMP due to their advantage of generating a much larger number of individual amplifications with lower cost, termed droplet digital LAMP (ddLAMP). Microfluidic flow focusing and T-junction are the most used methods for generating monodisperse droplets for ddLAMP [27,28,29,30]. Although microfluidic-based methods work well for droplet generation, they often require multiple fluid pumps and microfabricated devices, making them difficult to use in POC systems. Non-microfluidic strategies, including centrifugal force-based [31,32,33,34,35], inkjet printing [36,37,38,39], and electrowetting methods [40], have also been reported to simplify the droplet generation process. However, sophisticated external equipment is still required for operation.       In recent years, droplet generation methods that are simple and portable have been reported. Yuan and colleagues reported a ddLAMP method utilizing a hand-powered droplet generator [41], in which manual syringe-based suction pumping is used to form droplets in a microfluidic chip. The droplets are collected in a container and heated with a water bath, and then they are transferred to a well plate for fluorescence imaging. The authors reported a limit of detection of 10 copies per microliter with the system that can be used in POC; however, the size of the droplets cannot be tuned, so only one average droplet size is possible. Similarly, Wu and colleagues reported a ddLAMP method utilizing microfluidic step-emulsion of droplets through the manual pumping of a syringe [42], in which the bacterial DNA targets were distributed into a chip, amplified using a water bath, and underwent fluorescence imaging for duplex detection. The authors reported a limit of detection of 19.8 copies per microliter. Mao and colleagues reported an open droplet array consisting of hydrophilic and hydrophobic patterning, which was sealed with mineral oil. The chip was then incubated, and smartphone-based fluorescent imaging and app-based analysis were performed [43]. The authors reported a limit of detection of 1 copy of lambda DNA per microliter; however, the method is limited by fixed dimensions of the pattern. Although these simplified droplet generation methods are more POC-friendly, the performance of these methods in terms of throughput, monodispersity, and droplet size flexibility is compromised due to the lack of sophisticated control equipment, which leads to reduced analytical performance for digital assays.       Recently, our lab reported a simple and compact droplet generation method based on a vibrating sharp-tip capillary [44]. This system eliminates the need for external pumps and achieves on-demand droplet generation with a portable signal generation, which shows great potential for POC digital nucleic acid tests. In this work, we combine the vibrating sharp-tip capillary droplet generation with LAMP to achieve a simple digital droplet assay for detecting bacteria pathogen sequences. We generated a large range of stable droplet sizes (~0.7 pL to 22 nL) with a single device. We optimized the conditions for the digital LAMP reactions and employed a simple DNA fragmentation method to ensure uniform sample distribution throughout the experiments. Finally, we detected common waterborne pathogens, Vibrio cholerae and Shigella sonnei, with a linear dynamic range of approximately 2 to 6000 copies/μL. This method eliminates the need for bulky equipment for generating monodisperse droplets and a thermal cycler, which paves the way for a portable and high dynamic range ddLAMP system.'}, {'pmcid': '11243103', 'doi': '10.3390/nu16131988', 'introduction': '1. Introduction       Autism spectrum disorder (ASD) is a behaviorally defined neurodevelopmental disorder. ASD lacks specific clinical biomarkers and has seen an evolving conceptualization through the decades since it was first described. Over the past four decades, there has been dramatic increase in the number of individuals diagnosed with ASD [1]. In general, ASD is diagnosed by 3 years of age in most of those children experiencing it, although roughly 40% of such children are not first evaluated until 4 years of age [2]. A psychiatric diagnosis of ASD, which has behavior as its basis of definition, relies heavily upon precise observation and clinical expertise because the condition lacks standardized biomarkers [3].       ASD is one of the most common and challenging neurodevelopmental disorders in children. Its prevalence rate worldwide now exceeds 1%. A small number of these children appear to develop normally in their first year and then go through a period of regression between 18 and 24 months of age. ASD is characterized by deficits in communication and social interaction, as well as a presence of repetitive and restrictive behaviors. Moreover, ASD often manifests with a wide range of comorbidities that include morphological, physiological, and psychiatric conditions. ASD’s most commonly proposed causes are physiological and metabolic disorders involving immunity, oxidative stress, and mitochondrial dysfunction [4]. Co-occurrence of two or more disorders in the same individual has been observed, with comorbidities including anxiety, depression, attention deficit/hyperactivity disorder (AD/HD), epilepsy, gastrointestinal symptoms/problems, sleep disorders, learning disabilities, obsessive–compulsive disorder, intellectual disability, sensory problems, and immune disorders. The most prevalent comorbidity, at roughly 50%, is intellectual disability [5]. At least one comorbidity exists in about 70% of children with ASD, while 41% have two or more [6]. An estimated 20% of individuals diagnosed with ASD also have epilepsy [7].       Although ASD’s etiology remains largely unexplained, a recent finding identifies specific gut microbiota composition in ASD patients. Post-mortem examination of ASD subjects’ brain tissue and small intestines has revealed that the blood–brain barrier and gut barrier were disrupted, with significant neuroinflammation evidenced by increased expression of genes and markers associated with brain inflammation. It has further been inferred that the gut–brain axis disruption may be associated with non-self antigens that trigger a neuroinflammatory reaction by crossing the damaged gut barriers, thus leading to ASD in genetically susceptible subjects [8]. In a study involving 192 twins, however, genetic factors accounted for only 38% of ASD risk, whereas the remaining 68% was attributed to environmental factors [9]. A significant role was ascribed to the gut microbiota [10]. Microbiota is shaped by diet, lifestyle, and microbial exposure in the early developmental phase and infection, as well as by genetic makeup, metabolites, and immunological and hormonal aspects [11].       Analysis of gut microbiota is currently a growing area of research linked to neuropsychological disorders, including depression [12], metabolic disorders such as obesity [13], and gastrointestinal disorders, including inflammatory bowel disease or irritable bowel syndrome [14,15]. Many studies have identified that microbiota composition in ASD patients differs significantly from that in healthy controls [16,17,18,19]. Nutritional intervention, prebiotics, probiotics, and symbiotics, including fecal microbiota transplantation as a remedy to modulate the species composition of the intestinal microbiota of patients with ASD, already have been tested [20,21,22,23,24]. These studies generally have concluded, however, that their findings should be taken with caution because there still exist only limited data from studies examining different regimens of different remedy applications and there have not yet been double-blind studies demonstrating clinical significance of the effects of those remedies used. Here, therefore, we present the results of a double-blind, placebo-controlled study utilizing the biological response modifier Juvenil for influencing the gut microbiota of children suffering from ASD and for modulating their ASD symptoms.'}, {'pmcid': '11242183', 'doi': '10.3390/ijms25137375', 'introduction': '1. Introduction       The genus Plesiomonas, represented by a single species, Plesiomonas shigelloides, is a Gram-negative bacillus associated with gastrointestinal and extraintestinal diseases in humans [1], such as an invasive shigellosis-like disease, cholera-like illness, pseudoappendicitis, meningitis, acute secretory gastroenteritis, and bacteremia [2,3,4,5,6,7]. Many pathogenic bacterial species express flagella on their surfaces, which are their primary means of locomotion or motility, and flagellar motility is necessary for them to reach the site of infection, which is the initial step in the establishment of a bacterial infection [8]. Meanwhile, flagellum motility provides a significant advantage for bacteria to move toward favorable conditions or away from detrimental environments and to effectively compete with other microorganisms [9].       The flagellum consists of three parts: the basal body, hook, and filament [10]. Flagellar motility, an energetically expensive process, requires the coordinated expression of more than 50 genes encoding regulatory proteins, structural components of the secretion and assembly apparatus, and chemo-sensor machinery [8]. The coordinated expression of these promoters clusters gene transcription at three or four levels of hierarchy: classes I to III or I to IV [11]. Peritrichous flagellated bacteria, such as Salmonella and Escherichia coli, have three levels of hierarchy, and transcription of class I and II genes requires the housekeeping sigma factor 70 (σ70) [12]. However, some bacteria, such as Vibrio cholerae [13,14,15], Vibrio parahaemolyticus [16], and Aeromonas hydrophila [11,17], have four levels of flagella regulatory hierarchy. Furthermore, several of the aforementioned bacteria contain both polar and lateral flagella. Bacteria with polar flagellation exhibit four transcriptional levels: class I promoters encode a σ54-dependent activator (FlrA) and activate class II σ54-dependent promoters (FlrBC), which encode a two-component signal-transducing system whose regulator activates class III σ54-dependent promoters [13,14,15,16,17]. Moreover, class II promoters also encode the σ28 factor (FliA), which activates the transcription of class IV genes. The lateral flagellar system, such as that of V. parahaemolyticus and A. hydrophila, which have three levels of flagella regulatory hierarchy, is encoded by more than 30 lateral flagella genes and regulated by the LafK master regulator [18]. Furthermore, LafK may compensate for FlaK’s role in enabling swimming in some bacteria [19]. In the expression of the lateral flagellar system, the master regulator LafK activates the class II genes, and then the expression of class III genes is activated by σ28, which is encoded by fliAL [16,20,21]. In a word, in a spatiotemporal-dependent manner, the expression of flagellar genes is strictly regulated by the respective regulators and sigma factors.       The primary function of flagella is to provide motility to bacteria; nevertheless, they are also related to bacterial pathogenicity. Flagellum-mediated motility plays a critical role in interactions between Vibrio species and their hosts, and these bacteria colonization and lethality in their respective hosts decrease when motility is restricted [22]. Vibrios have therefore been shown to be valuable models for researching flagellar control and its role in colonization. Motility has been identified as a virulence determinant of V. cholerae. Additionally, non-motile mutants in competition assays using an infant mouse model system show no significant defect in their capacity to colonize the small intestine [23]. A Na+ gradient across the membrane drives V. cholerae flagellar rotation while also regulating transcription of the toxT gene, which is necessary for CT and TCP expression, revealing a possible mechanism for connecting virulence factor expression to motility [24,25]. Coster and Kenner found that the human reactogenicity of live attenuated V. cholerae vaccines is also lowered in non-motile mutants [26]. In addition to being more frequently documented in V. cholerae, the correlation between flagella and virulence has also been investigated in other bacteria. Flagella motility, like that of V. cholerae, is critical in the pathogenesis of V. campbellii, and inhibiting motility greatly reduces host mortality during infection [27]. Previous studies have also reported the synergistic action of SPI-1 gene expression in Salmonella enterica serovar typhimurium through transcriptional cross-talk with the flagellar system [28]. Zhao et al. found that the type III secretion system intersects with the lateral flagellar system in A. hydrophila [29]. In a burned mouse model of infection, non-flagellated mutants of P. aeruginosa have been demonstrated to exhibit reduced penetration of cultured corneal epithelial cells and to be deficient in pathogenicity [30,31]. As a result, the significance of flagella in pathogenicity is obvious.       In addition to granting motility to bacteria and influencing adherence and virulence, flagellar regulators and flagellar genes are also involved in the physiological metabolism of bacteria. The relationship between the ability to survive periods of acid shock and the fine-tuned expression of motility genes and fitness is currently unknown; however, Schumacher et al. found an inverse correlation between the expression of FlhDC, the master regulator of flagellation, and acid shock survival in E. coli [32]. According to Hoque et al., the ∆flrA mutation in V. cholerae increases iron utilization and oxidative stress tolerance since it significantly upregulates most of the genes involved in heme absorption, siderophore uptake, and vibriobactin uptake [33]. Sinha-Ray et al. found that a mutation in V. cholerae’s flrA gene is inversely involved in the vps-independent biofilm driving bacteria toward nutrients in lake water [34]. Rodríguez-Herva et al. suggested that in addition to a number of motility and chemotaxis genes, the fliA gene product is also necessary for the expression of some genes potentially involved in amino acid utilization or stress responses [35]. Li et al. reported that flagellum hook-associated protein (FlgK) increases the resilience of C. sakazakii to dehydration and regulates formate dehydrogenase and nitrate reductase [36]. It has also been reported in studies that FliZ plays a key role in controlling lipase and hemolysin activities [37,38].       P. shigelloides is a special member of the Enterobacteriaceae family that can produce both polar and lateral flagella, and Merino et al. showed that P. shigelloides has two distinct gene clusters: one for lateral flagella biosynthesis and another for biosynthetic polar flagella [39]. Merino and co-workers reported that although P. shigelloides lacks a FlrB ortholog, P. shigelloides FlrC (FlaM) contains the PAS and His Kinase A domains found in the FlrC proteins of Vibrio species and A. hydrophila, indicating that P. shigelloides FlaM may activate transcription from σ54-dependent promoters of class III genes [39]. Furthermore, the P. shigelloides polar flagella gene regions show greater similarity to those reported in Vibrio or Aeromonas than the regions in Enterobacteriaceae [40]. Though a number of studies have implicated motility as being important for bacteria, the transcriptional regulation network of polar and lateral flagella in P. shigelloides has yet to be revealed. In this work, we identified FlaK, FlaM, FliA, and FliAL as the four regulators responsible for polar and lateral flagellar regulation in P. shigelloides. Through transcriptome sequencing analysis, we confirmed the polar and lateral flagellar regulatory hierarchy and further revealed the cross-talk between the flagellar hierarchy system and P. shigelloides’ virulence and physiological metabolism.'}, {'pmcid': '11240787', 'doi': '10.3390/ani14131978', 'introduction': '1. Introduction       Currently, neonatal porcine diarrhea caused by bacteria is the main problem in some large-scale breeding farms free of enteroviruses [1], with diarrhea induced by E. coli being the most common form [2,3,4]. This disease causes high mortality or low weight gain in suckling piglets, leading to severe economic losses for the pig industry [5,6,7]. Reducing the pathogen load of bacteria in the environment and the sow body surface through disinfection, washing, and enhancing the immunity of piglets against pathogens through vaccination are the main prevention and control measures [8,9]. However, it is difficult to synthesize a strain-specific vaccine with efficient cross-protection because of the serotype diversity of E. coli strains [1,10]. Therefore, sows are often fed a homogenate of intestinal tissues and contents from piglets who succumbed to diarrhea to stimulate their immune system and pass specific antibodies to suckling piglets through colostrum and milk when E. coli-induced diarrhea occurs in pig farms, which is also known as feedback feeding [9,11,12]. However, the individual differences in the immune response of the same batch of sows to feedback feeding make it impossible to achieve consistent results from this procedure.       The gastrointestinal tract is one of the largest immunological organs of the body [13]. Feedback feeding is a kind of gastrointestinal mucosal immune strategy. Mucosal immunity provides the first line of defense against exogenous pathogens by secreting SIgA and IgM to inhibit surface colonization by microorganisms [13,14,15]. B cells in mucosa-associated lymphoid tissue are the cells that give rise to secretory IgA and IgM. During the perinatal period, in sows, maternal IgA-secreting cells migrate to the mammary gland to increase the levels of maternal antibodies in milk [15]. The authors of a previous study verified the involvement of intestinal microorganisms in the number of IgA plasma cells in the mammary glands (but not the small and large intestines), as well as the production of maternal IgA in milk [16].       B. subtilis is a kind of probiotic with excellent properties that can be widely used as a feed additive in livestock and poultry farming [17,18,19]. It has been shown that B. subtilis can increase the levels of IgG, IgM, and SIgA in serum or milk [20,21]. In this study, we aimed to enhance the immune response of sows against E. coli by combining B. subtilis with feedback feeding to increase the levels of specific maternal antibodies to improve intestinal function and reduce E. coli-caused diarrhea in suckling piglets. Our study provides a new resolution for the prevention and control of E. coli-caused diarrhea in suckling piglets.'}, {'pmcid': '11240451', 'doi': '10.3390/ani14131959', 'introduction': '1. Introduction       Chlamydial infections caused by widespread obligate intracellular Gram-negative bacteria from the Chlamydiaceae family are associated with a wide spectrum of clinical symptoms in livestock, companion animals, wildlife, and various exotic species including several birds species [1,2,3]. In addition, some chlamydial species, such as Chlamydia abortus and Chlamydia psittaci, are capable of causing zoonotic infections [4,5].        In humans, Chlamydia psittaci (C. psitacci) causes a severe disease, known as psittacosis, which, in most cases, can lead to pneumonia and significant mortality if left untreated [6,7,8,9]. Psittacosisis is a significant threat to public health and is considered an occupational zoonosis of pet shop owners, zookeepers, veterinarians, abattoir workers, bird breeders, and pet bird enthusiasts [2,10,11].       At least 465 bird species belonging to 30 orders can have chlamydiosis. However, psittacidae (lories, parakeets, parrots, and cockatoos) and columbiformes (pigeons) are the most frequently affected. Primary reservoirs for zoonotic psittacosis are psittacine pets, as well as wild and racing pigeons [12].       C. psittaci replicates using a biphasic developmental cycle within cytoplasmic inclusion vacuoles of eukaryotic cells. This cycle involves two morphologically different forms, as follows: the infectious extracellular form, termed the elementary body (EB), and the reproductive reticulated body (RB). The EB enters a host cell through receptor-mediated endocytosis. This process, which lasts several hours, results in the conversion of the EB into a reticulate body (RB). The RB divides via binary fission to produce more EBs. This cycle continues for several hours after infection, at which time the host cell lyses, releasing several hundred EBs, RBs, and intermediates [13].       Chlamydiosis causes a range of clinical signs in birds, including respiratory difficulty, diarrhea, and lethargy, and it can also lead to chronic or asymptomatic infections. Some infected birds may appear healthy, but shed the organism intermittently. Shedding may be exacerbated by internal factors such as the bird’s immune status, in addition to external factors such as stress, reproductive activities, relocation, shipping, crowding, and cold temperature. The severity of clinical signs varies significantly, influenced by the species and age of the birds, as well as the specific strains of the pathogen involved [2].       In psittacine birds, gross lesions include splenomegaly, hepatomegaly, enteritis, sinusitis, air sacculitis, pericarditis, and conjunctivitis. Microscopic lesions of chlamydiosis in birds include necrotizing changes in several organs associated with chlamydial intracytoplasmic elementary bodies and the infiltration of heterophils, macrophages, and plasma cells [14,15,16].       Chlamydia psittaci infection in birds has been reported previously in many countries [1,17,18,19,20]. In recent years, there has been a global increase in the number of reported cases of avian chlamydiosis in both birds and humans [9,21,22,23].       The analysis of C. psittaci ompA gene sequences is crucial for genotype determination [24]. In the studied psittacidae population, the circulating genotypes are identified as A and B. Initially, C. psittaci was categorized into nine genotypes—A-F, E/B, M56, and WC, with the later proposal of eight new genotypes [1,24]. Genotype A is prevalent among psittacine birds, but is also found in turkeys, ducks, pigeons, and Passeriformes. Genotype B is endemic in pigeons but can infect various bird species including chickens, turkeys, ducks, Psittacidae, and Passeriformes. Waterfowl are commonly infected with genotype C and E/B strains, while turkeys can harbor genotypes D and C. Genotype E is found in ducks, pigeons, ostriches, and rheas, and genotype F is in psittacine and turkey isolates [1,25,26,27].       This article describes a large outbreak of psittacosis in a commercial breeding aviary in Buenos Aires, Argentina, and emphasizes the need to monitor and quickly identify the disease to minimize its impact in birds and prevent the spread to people at risk.'}, {'pmcid': '11238125', 'doi': '10.1016/j.heliyon.2024.e33152', 'introduction': 'N/A'}, {'pmcid': '11343591', 'doi': '10.1172/jci.insight.174776', 'introduction': 'Introduction       Axial spondyloarthritis (axSpA) is a chronic inflammatory disorder that damages the spine, with occasional involvement of peripheral joints and tendons (1). The greatest genetic risk factor for both radiographic axSpA (r-axSpA) and nonradiographic axSpA (nr-axSpA) is the HLA-B*27 allele; more than 80% of patients are HLA-B*27+ compared with less than 10% of the general population (2–4). Many patients with axSpA also have acute anterior uveitis (AAU), and HLA-B*27 increases the risk for developing isolated AAU; 50% of individuals with AAU are HLA-B27+ (5, 6). HLA-B*27–associated AAU (B27AAU) presents as sudden onset, unilateral ocular inflammation, with resolution after several weeks (7, 8). Patients with B27AAU are also at high risk (55%–90%) of developing axSpA (6, 9). The shared genetic risk allele of HLA-B*27 and clinical overlap of axSpA and B27AAU suggest a common pathophysiologic mechanism.       HLA-B*27 has been suggested to increase the risk of developing axSpA and B27AAU through various mechanisms. One hypothesis is that HLA-B*27 indirectly causes loss of immune tolerance via its deleterious effect on immune homeostasis and promotion of intestinal inflammation (10–12). Investigators have identified asymptomatic, microscopic inflammation in intestinal biopsies of patients with either r-axSpA or reactive arthritis, the latter being clinically associated with specific intestinal pathogens, including Shigella and Salmonella (13–15). However, which immune cells or microbial products are responsible for transferring enteric inflammation to the joints in axSpA or eye in B27AAU remain unclear (16). Another hypothesis is that HLA-B*27 presents antigens to pathogenic CD8+ T cells in the form of molecular mimicry, whereby CD8+ T cells initially respond to foreign antigens, such as from bacteria, but subsequently are stimulated by self-antigens due to cross-reactivity of their T cell receptors (TCRs) (16). This “arthrogenic peptide” hypothesis is supported by our recent identification of axSpA-associated public TCRs that are enriched in the joint in axSpA and eye of B27AAU and can recognize both bacterial and self-peptides presented by HLA-B*27 (17). Moreover, elimination of these public TCRs via TRBV9-targeted depletion leads to disease improvement, at least in some cases (18, 19). One bacterial peptide recognized by these public TCRs is derived from yeiH, a gene present in multiple enteric microbes. Whether these expanded CD8+ T cells in axSpA and B27AAU have a signature of mucosal immunity remains unknown.       CD161 (KLRB1) is expressed on 2 subsets of T cells that are associated with enteric immunity. CD161hiCD8+ T cells are mucosal-associated invariant T (MAIT) cells that are enriched in enteric organs, such as the liver and intestine; participate in immunity to microbiota; and may play a role in axSpA and B27AAU (20–22). These unconventional T cells recognize 5-OP-RU from bacterial riboflavin biosynthesis in the context of the MHC class I–like molecule MR1 (23). In contrast to MAIT cells, CD8+ T cells expressing intermediate levels of CD161 (CD161int) are conventional polyclonal αβ T cells that recognize foreign peptides presented by canonical MHC molecules (24). Despite this distinction, CD161intCD8+ T cells share several properties with MAIT cells. For instance, CD161intCD8+ T cells are enriched in enteric organs, particularly the colon (24), and contain a distinct transcriptional program that is shared with MAIT cells (25). In addition, CD161 is expressed by a population of CD8+ T cells that establish tissue residency within the small intestine (26). However, whether antigen-specific CD161intCD8+ T cells play a role in axSpA and B27AAU remains unknown.       Here, we report that B27AAU is characterized by an influx YeiH.232-240–specific CD161intCD8+ T cells into the eye. Moreover, these YeiH.232-240–specific CD8+ T cells expressed an intestinal CD8+ T cell transcriptional program, were more abundant in axSpA and B27AAU compared with individuals acting as healthy controls, and expressed surface molecules associated with mucosal trafficking and immunity. Collectively, our data suggest that, in HLA-B27–associated inflammation, the early activation of pathogenic T cells may occur in enteric organs.'}, {'pmcid': '11341846', 'doi': '10.1038/s41598-024-70654-6', 'introduction': 'N/A'}, {'pmcid': '11340806', 'doi': '10.3389/fmicb.2024.1439204', 'introduction': 'N/A'}, {'pmcid': '11340127', 'doi': '10.1186/s13062-024-00517-7', 'introduction': 'N/A'}, {'pmcid': '11339456', 'doi': '10.1038/s41598-024-69923-1', 'introduction': 'N/A'}]\n",
            "59\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "for item in records:\n",
        "    pmcid = item.get('pmcid')\n",
        "    if pmcid:\n",
        "        fetch_response = requests.get(f'https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id={pmcid}&retmode=xml')\n",
        "        if fetch_response.status_code==200:\n",
        "            file_name = f'{pmcid}.xml'\n",
        "            with open(file_name, \"w\") as file:\n",
        "                file.write(fetch_response.text)\n",
        "            print(f\"saved {pmcid} to {file_name}\")\n",
        "        else:\n",
        "            print(f\"Failed to fetch data for {pmcid}. Status code: {fetch_response.status_code}\")"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "jyGuG3E-Obos",
        "outputId": "1c0f4964-ad27-4de5-f8a9-df8a30972878"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "saved 11342756 to 11342756.xml\n",
            "saved 11342206 to 11342206.xml\n",
            "saved 11341105 to 11341105.xml\n",
            "Failed to fetch data for 11340603. Status code: 429\n",
            "Failed to fetch data for 11340327. Status code: 429\n",
            "Failed to fetch data for 11326835. Status code: 429\n",
            "Failed to fetch data for 11324020. Status code: 429\n",
            "Failed to fetch data for 11323402. Status code: 429\n",
            "Failed to fetch data for 11322368. Status code: 429\n",
            "saved 11319344 to 11319344.xml\n",
            "saved 11318556 to 11318556.xml\n",
            "saved 11316331 to 11316331.xml\n",
            "Failed to fetch data for 11315073. Status code: 429\n",
            "Failed to fetch data for 11313837. Status code: 429\n",
            "Failed to fetch data for 11313783. Status code: 429\n",
            "Failed to fetch data for 11312530. Status code: 429\n",
            "Failed to fetch data for 11312258. Status code: 429\n",
            "Failed to fetch data for 11312129. Status code: 429\n",
            "saved 11311926 to 11311926.xml\n",
            "Failed to fetch data for 11311727. Status code: 429\n",
            "saved 11304630 to 11304630.xml\n",
            "Failed to fetch data for 11304591. Status code: 400\n",
            "saved 11303641 to 11303641.xml\n",
            "saved 11302663 to 11302663.xml\n",
            "saved 11302304 to 11302304.xml\n",
            "saved 11300865 to 11300865.xml\n",
            "Failed to fetch data for 11237486. Status code: 429\n",
            "Failed to fetch data for 11293832. Status code: 429\n",
            "Failed to fetch data for 11293673. Status code: 429\n",
            "Failed to fetch data for 11291472. Status code: 429\n",
            "saved 11289428 to 11289428.xml\n",
            "Failed to fetch data for 11288818. Status code: 429\n",
            "saved 11285316 to 11285316.xml\n",
            "Failed to fetch data for 11281493. Status code: 429\n",
            "saved 11279407 to 11279407.xml\n",
            "Failed to fetch data for 11278787. Status code: 429\n",
            "saved 11273848 to 11273848.xml\n",
            "Failed to fetch data for 11272550. Status code: 429\n",
            "Failed to fetch data for 11269248. Status code: 429\n",
            "saved 11265448 to 11265448.xml\n",
            "saved 11263920 to 11263920.xml\n",
            "saved 11262882 to 11262882.xml\n",
            "Failed to fetch data for 11261454. Status code: 429\n",
            "Failed to fetch data for 11260827. Status code: 429\n",
            "Failed to fetch data for 11257890. Status code: 429\n",
            "saved 11249912 to 11249912.xml\n",
            "saved 11246856 to 11246856.xml\n",
            "saved 11244529 to 11244529.xml\n",
            "Failed to fetch data for 11243892. Status code: 429\n",
            "Failed to fetch data for 11243103. Status code: 429\n",
            "Failed to fetch data for 11242183. Status code: 429\n",
            "Failed to fetch data for 11240787. Status code: 429\n",
            "saved 11240451 to 11240451.xml\n",
            "Failed to fetch data for 11238125. Status code: 429\n",
            "Failed to fetch data for 11343591. Status code: 429\n",
            "saved 11341846 to 11341846.xml\n",
            "saved 11340806 to 11340806.xml\n",
            "Failed to fetch data for 11340127. Status code: 429\n",
            "Failed to fetch data for 11339456. Status code: 429\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "pmcid_list = []\n",
        "for item in records:\n",
        "    pmcid = item.get('pmcid')\n",
        "    if pmcid:\n",
        "        pmcid_list.append(pmcid)\n",
        "\n",
        "xml_count = len([f for f in os.listdir('/content/') if f.endswith('.xml')])\n",
        "\n",
        "print(f'numbe of pmcid : {len(pmcid_list)}')\n",
        "print(f'number of xml : {xml_count}')"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "yIy2G3MmPe-B",
        "outputId": "599393c9-e1ae-4c55-d1b3-864ac1d25bb3"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "numbe of pmcid : 59\n",
            "number of xml : 25\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# extracted data to tsv file\n",
        "df = pd.DataFrame(records)\n",
        "df.head()"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 206
        },
        "id": "MrtDA-g3RcBW",
        "outputId": "0f975a02-10ae-4ccc-bd8c-0f63a04dea92"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "      pmcid                              doi  \\\n",
              "0  11342756        10.1016/j.psj.2024.104099   \n",
              "1  11342206  10.1016/j.bioactmat.2024.07.032   \n",
              "2  11341105             10.7759/cureus.65185   \n",
              "3  11340603       10.1016/j.csbj.2024.07.017   \n",
              "4  11340327              10.1128/jb.00049-24   \n",
              "\n",
              "                                        introduction  \n",
              "0                                                N/A  \n",
              "1                                                N/A  \n",
              "2  Introduction       Pseudomonas aeruginosa is a...  \n",
              "3                                                N/A  \n",
              "4  INTRODUCTION       The gram-negative bacterial...  "
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-1899487e-1f5f-4460-81f9-5e36888d9f53\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>pmcid</th>\n",
              "      <th>doi</th>\n",
              "      <th>introduction</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>11342756</td>\n",
              "      <td>10.1016/j.psj.2024.104099</td>\n",
              "      <td>N/A</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>11342206</td>\n",
              "      <td>10.1016/j.bioactmat.2024.07.032</td>\n",
              "      <td>N/A</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>11341105</td>\n",
              "      <td>10.7759/cureus.65185</td>\n",
              "      <td>Introduction       Pseudomonas aeruginosa is a...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>11340603</td>\n",
              "      <td>10.1016/j.csbj.2024.07.017</td>\n",
              "      <td>N/A</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>11340327</td>\n",
              "      <td>10.1128/jb.00049-24</td>\n",
              "      <td>INTRODUCTION       The gram-negative bacterial...</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-1899487e-1f5f-4460-81f9-5e36888d9f53')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-1899487e-1f5f-4460-81f9-5e36888d9f53 button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-1899487e-1f5f-4460-81f9-5e36888d9f53');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-b4183f1b-b9ca-4c6c-995d-6fb139a6b69b\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-b4183f1b-b9ca-4c6c-995d-6fb139a6b69b')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-b4183f1b-b9ca-4c6c-995d-6fb139a6b69b button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "variable_name": "df",
              "summary": "{\n  \"name\": \"df\",\n  \"rows\": 59,\n  \"fields\": [\n    {\n      \"column\": \"pmcid\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 59,\n        \"samples\": [\n          \"11342756\",\n          \"11326835\",\n          \"11279407\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"doi\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 59,\n        \"samples\": [\n          \"10.1016/j.psj.2024.104099\",\n          \"10.1093/infdis/jiad585\",\n          \"10.3390/pathogens13070583\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"introduction\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 34,\n        \"samples\": [\n          \"Introduction       Borrelia spp. are globally distributed, vector-borne bacteria that cause diverse diseases, rendering these pathogens an immense public health concern [1,2]. Borrelia can be divided into two distinct groups, relapsing fever (RF) and Lyme disease (LD) Borrelia, based on associated disease courses, vector/enzootic cycles, and additional phylogenetic differences [1\\u20133]. Infection with RF spirochetes is characterized by recurring severe, febrile episodes that coincide with repeated bouts of high-level bacteremia (up to 108 bacteria/ml of blood) [1,4\\u20138]. Furthermore, infection by RF Borrelia can lead to more severe symptoms and outcomes, such as acute respiratory distress syndrome, jaundice, meningitis, splenomegaly, and perinatal mortality [4,9,10]. Alternatively, mammalian infection with LD Borrelia results in lower levels of bacteremia (103\\u2212104 bacteria/ml of blood) that is restricted to early stages of infection, after which it disseminates into multiple host tissues [11]. Therefore, symptoms of RF are primarily associated with acute bacteremia, whereas most LD symptoms occur because of chronic tissue colonization (e.g., arthritis, neuroborreliosis, and carditis) [2]. In addition, arthropod vectors differ greatly between RF and LD Borrelia. RF bacteria are transmitted by the human body louse [louse-borne relapsing fever (LBRF)] or Ornithodoros or Ixodes spp. ticks [tick-borne relapsing fever (TBRF)], while LD spirochetes are only transmitted by Ixodes spp. ticks [12\\u201317]. The different disease courses resulting from infection by LD and RF Borrelia and their disparate enzootic cycles involving distinct vectors suggest that these two groups of bacteria have likely evolved to encode different factors required for vector acquisition and colonization, transmission, and vertebrate infection. Because most molecular research has focused primarily on LD Borrelia, a significant gap exists in our understanding of the biology of RF spirochetes [18,19].       RF Borrelia can be further subdivided into New World (found in the Americas) and Old World (found in Africa, Asia, and Europe) species [4,5,16,20]. New World RF spirochetes include Borrelia turicatae, which is found in the arid climate of the southwestern United States, and Borrelia hermsii and Borrelia parkeri, the causes of RF in the mountainous regions of the western United States and southwestern Canada [4,5,12,13,21\\u201323]. Old World RF spirochetes include Borrelia crocidurae and Borrelia duttonii, which are endemic to western and eastern Africa, respectively, Borrelia persica, a causative agent of TBRF in Eurasia, and Borrelia recurrentis, the LBRF spirochete responsible for epidemic RF in eastern Africa [24\\u201330]. RF spirochetes were historically classified into New World and Old World species based solely on geographic separation, but phylogenetic analyses have revealed genomic differences in these bacteria, supporting the separation of the groups into two distinct clades [26,31\\u201334].       Although RF is globally distributed, the public health impact varies throughout the world. While RF is a relatively rare and possibly underreported disease in the Americas, this infection is endemic and highly prevalent in parts of Africa [1,4,5,20,25\\u201327,32]. In fact, RF has been reported as the most common bacterial infection in Senegal, the most prevalent cause of fever in rural Zaire, and a top 10 cause of death in children under the age of 5 in Tanzania [1,25,27,35\\u201337]. Furthermore, cases are often unreported or are misdiagnosed as another disease, such as malaria, in endemic regions of Africa, implying that the actual incidence is higher than previously reported [28,38]. Despite the clear public health threat that Old World RF spirochetes represent, very little is known regarding the molecular pathogenesis of these bacteria.       Use of molecular Koch\\u2019s postulates provides the most direct way to determine a causal relationship between a bacterial gene, the protein it encodes, and a requirement during the bacterial lifecycle [39,40]. To apply molecular Koch\\u2019s postulates, the gene of interest is inactivated, and the phenotype of the mutant is compared to that of the wild-type parent. Additionally, expression of the gene must be restored in the mutant to assess if the phenotype of interest is reversed. While a limited number of studies have applied genetic manipulation in RF Borrelia to investigate proteins potentially required during the bacteria\\u2019s enzootic cycle, these studies have only been conducted with the New World RF spirochetes, B. hermsii and B. turicatae [41\\u201350]. Extensive geographic separation and phylogenetic differences between New World and Old World RF spirochetes suggests that these two groups likely evolved distinct regulatory pathways and proteins required for mammalian virulence, as well as vector acquisition, colonization, and transmission. Therefore, there is a significant need to develop molecular techniques for the study of Old World RF Borrelia.       Herein, we aimed to develop a genetic system for use in the Old World TBRF spirochete, B. duttonii. First, we developed shuttle vector- and integration-based constructs and methods for electroporation with B. duttonii, creating green fluorescent protein (gfp)-expressing B. duttonii strains to validate the approach. Allelic exchange mutagenesis was then used to inactivate the gene encoding the B. duttonii homolog of P66, a known virulence factor of the LD spirochete, Borrelia burgdorferi [51,52]. Using a murine model of RF, we demonstrated that the B. duttonii p66 mutant was attenuated. Finally, this p66 mutant was complemented using shuttle vector- and cis integration-based approaches to restore production of P66 and murine infectivity of the p66 mutant strains. The techniques described in this manuscript represent a foundation for the application of molecular tools to study Old World RF Borrelia and will ultimately guide research to better understand these understudied pathogens.\",\n          \"1. Introduction       When gastrointestinal pathogens are detected in stool samples of patients with infectious gastroenteritis living in regions where such pathogens as well as associated infectious gastroenteritis are frequent, it can be difficult to say whether the detected pathogen is really the cause of the observed clinical symptoms. This is particularly true if more pathogens than just a single one are detected in the same stool sample. The problem that most enteric pathogens in regions with high prevalence for infectious gastroenteritis can both cause enteric disease or just persist as harmless colonizers is called \\u201cfacultative pathogenicity\\u201d [1,2,3,4,5]. Previously, other researchers have tried to link the quantity of pathogens in stool samples with their likelihood of causing infectious gastroenteritis in such patients. In the case of diagnostic real-time PCR, so-called cycle threshold (Ct) values are an indirect option for target quantification because low Ct values indicate high quantities of the PCR target and vice versa. However, such attempts at linking pathogen quantity in stool samples with the likelihood of this pathogen causing clinical disease were only partly successful [2,3,6]. Consequently, generally accepted cut-offs for a Ct-value-based assignment of etiological relevance of enteric pathogens do not exist.       Obviously, more factors influence the likelihood of an association between clinically observed infectious gastroenteritis and the detection of an enteric pathogen in a human stool sample. One of these factors is \\u201csemi-immunity\\u201d, which means immunological adaptation of the human gut to rapid cycles of repeated pathogen exposure under poor hygiene conditions. This has been observed repeatedly in resource-limited tropical regions [7,8,9]. Although it is not yet completely understood how enteric semi-immunity works on a cellular or molecular level, some well-defined hypotheses regarding host\\u2013pathogen coexistence have been proposed, particularly for enteric helminth infections, as summarized elsewhere [10]. Furthermore, the semi-immunity concept is already in preventive medical use. Oral typhoid fever vaccination is the most commonly known example of induced short-term immunity on enteric mucous membranes [11].       Next to immunological adaptations, the enteric microbiome\\u2019s composition is believed to play a role in the degree of susceptibility towards the virulence of enteric pathogens. For both laboratory animals and human individuals, favorable microbiome compositions have been demonstrated to mitigate colonization resistance towards enteric pathogens, thus making gastroenteric infections less likely [12,13,14]. In addition, there is an increasing body of evidence suggesting likely interaction between enteric pathogens and commensals, which affects the clinical outcome [5,15].       In order to contribute to deciphering such microbial interactions in the gut of individuals from high-endemicity settings for infectious gastroenteritis, our study group recently conducted a cluster analysis assessing gastroenteric pathogens in stool samples of a Colombian indigenous population [16]. Within this population, a cluster consisting of Blastocystis hominis, Campylobacter spp., and Giardia duodenalis was shown to interact with Dientamoeba fragilis and Ascaris lumbricoides in a microbial-density-dependent way [16]. During the interpretation of this finding [16], however, it remained uncertain whether this observation just represented a regional peculiarity or a general pattern of microbial interaction in the human gut.       As such, it seemed promising to repeat the analysis with populations from other tropical high-endemicity settings for gastroenteric infections. Ghana, in the West African region, is an example of a country where enteric pathogens can be detected both in association with diarrheal disease and in asymptomatic individuals [4,17]. Especially high detection rates of gastroenteric pathogens can be expected in Ghanaian children [18,19,20], and in older studies, when diagnostic approaches with poor diagnostic accuracy, like Widal testing, were still in use [21], even underestimations of the true prevalence were likely. In such populations, co-occurrence of various microbial agents in stool samples as well as rapid pathogen acquisition cycles have been reported previously [22,23]. Furthermore, the common co-occurrence of genetic resistance determinants in Ghanaian stool samples [24,25] bears the risk of resistance transmission to bacterial enteric pathogens via mobile genetic elements. For Campylobacter spp. detections in Ghana, resistance was pronounced in cases of HIV-positive patients [26]. Enteric protozoan parasite infections of the gut have been reported to be particularly frequent in Ghanaian patients with diabetes [27]. Ghanaian farm environments provide reservoirs for several enteric pathogens like, e.g., salmonellae [28] and Cryptosporidium spp. [29]. Fecal contamination of regional environments is common [30], and food-borne or water-borne transmissions of infectious gastroenteritis are frequent events, particularly for poor Ghanaians [31,32,33]. Consequently, evidence of long-term efficiency of water filtration has recently been confirmed for Ghana [34], and, in line with the abovementioned information, colonized food vendors play a relevant role for such food-borne transmission events in Ghana [35].       Based on such previous experiences, Ghana was chosen as a suitable candidate region for cross-checking the experience from Colombia [16] in another tropical high-endemicity setting. To do so, cluster analyses were performed both with the set of microbial parameters chosen for the Colombian assessment [16] alone as well as in comparison with the Colombian results published elsewhere [16] and also with a broadened dataset available for the Ghanaian samples only. Furthermore, the dataset on Ghanaian individuals was sub-divided into subsets comprising children under 2 years of age and HIV-positive individuals. The rationale of these analytical steps is as follows. If microbial interactions are stable, comparable clustering should appear despite regionally different populations and despite interindividual differences, including factors like medical conditions and environmental factors. In summary, the study aimed at identifying stable clusters as markers of microbial interaction with potential importance for etiological relevance assignment in cases of multiple enteric pathogen detections, and the inclusion of different subpopulations, including children and HIV-positive individuals, was performed to further challenge the stability of potentially observed clustering.\",\n          \"1       Introduction       Helicobacter pylori (H. pylori) infection is a global health problem, resulting in chronic inflammation and the increased risk of peptic ulcer disease, atrophic gastritis, gastric carcinoma, and mucosa-associated lymphoid tissue (MALT) lymphoma (1). The global pooled prevalence of H. pylori infection has been reported to be 48.5%, ranging from 70.1% in Africa to 24.4% in Oceania (2). A recent meta-analysis reported that the prevalence of H. pylori infection in mainland China was 40.0% during the period 2015\\u20132019, highlighting China as one of the countries with a substantial burden of H. pylori infection, despite a declining trend from 2000 to 2019 (3). Besides, considering the high resistance rates of clarithromycin and metronidazole (4), bismuth quadruple therapy is recommended as the first-line treatment in China (5), including 2 antibiotics (with five different regimens), a standard dose of a proton pump inhibitor (PPI), and bismuth (6). Both antibiotics and PPIs are known to disrupt the gut bacterial equilibrium, and the bacterial shifts caused by eradication therapy may contribute to the disturbance of carbohydrate and lipid metabolism (7). Consequently, it is necessary to develop a therapeutic regimen for H. pylori infection that not only achieves a satisfactory eradication rate but also minimizes the impact on the gut microbiota.       Fucoidan is a complex sulfated polysaccharide extracted from various species of brown seaweed (8). There is a growing body of evidence suggesting that fucoidan possesses various physiological properties, including high antioxidant activity (9), anti-tumor effect (10), and anti-allergic property (11). Besides, fucoidan can exert anti-inflammatory effects by modulating neutrophil migration and the expression of genes related to COX-2 and the NF-\\u03baB signaling pathway (12). Therefore, fucoidan is regarded as a promising component in functional food or adjuvant therapy. Fucoidan has two primary structural backbones: Type I, such as fucoidan from Laminaria japonica, is composed of repeated \\u03b1 (1\\u2009\\u2192\\u20093)-linked \\u03b1-L-fucopyranose, with fucose-linked sulfate groups in C2- and C4- positions; Type II, such as fucoidan from Ascophyllum nodosum, is composed of alternate \\u03b1 (1\\u2009\\u2192\\u20093)- and \\u03b1 (1\\u2009\\u2192\\u20094)-linked \\u03b1-L-fucopyranose, with fucose-linked sulfate groups in C2-, C3-, and C4-positions (13). The sulfate groups are important determinants for the bioactivities of fucoidan, depending on the sulfate content and position of the groups (14).       Research in a rat model of breast cancer has indicated that fucoidan intervention can enhance the diversity of the intestinal microbiota and change the microbiota composition, including a decrease in the abundance of Collinsella, Coprococcus, Oscillospira, and Sporosarcina as well as an increase in the abundance of Parabacteroides and Blautia (15). In an ulcerative colitis mice model, fucoidan was also found to significantly increase the abundance of Ruminococcaceae and Clostridium-IV (16). Moreover, fucoidan has been demonstrated to increase several beneficial bacterial taxa and decrease several harmful taxa in mice intervened with ciprofloxacin-metronidazole (17), indicating a potential capacity to mitigate dysbiosis caused by antibiotics, which is common during the eradication therapy of H. pylori infection. Nevertheless, data on the effects of fucoidan on the human gut microbiota are limited. In patients with chronic superficial gastritis, a combination of wheat peptides and fucoidan was reported to relieve symptoms, reduce gastric mucosal damage, and regulate gut microbiome, promoting the growth of Bifidobacterium pseudocatenulatum, Eubacterium siraeum, and Bacteroides intestinali (18), but the isolated effects of fucoidan were not evaluated.       Both in vitro and in vivo studies have shown that fucoidan can prohibit H. pylori adhesion to host cells, reduce cytotoxic effects in H. pylori, and decrease the infection rate of H. pylori in a dose-dependent manner (19). However, clinical data regarding the effects of fucoidan on H. pylori infection remains limited. Thus, we conducted a prospective, randomized, open-label trial in patients with H. pylori infection. The aim was to assess the effects of fucoidan supplementation, either alone or in combination with synbiotics supplementation, on the eradication rate and the maintenance of gut microbiota homeostasis in these patients.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"
            }
          },
          "metadata": {},
          "execution_count": 9
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# PMCID, DOI, Introduction 포함\n",
        "df.to_csv('articles.tsv', sep='\\t', index=False)"
      ],
      "metadata": {
        "id": "qo-PnhvgR3dv"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "!zip articles_xml.zip *.xml"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "PNLXjVuRQPdF",
        "outputId": "fba9f28e-8180-40fe-cb49-afa1a6f128c0"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "  adding: 11240451.xml (deflated 79%)\n",
            "  adding: 11244529.xml (deflated 82%)\n",
            "  adding: 11246856.xml (deflated 81%)\n",
            "  adding: 11249912.xml (deflated 80%)\n",
            "  adding: 11262882.xml (deflated 78%)\n",
            "  adding: 11263920.xml (deflated 82%)\n",
            "  adding: 11265448.xml (deflated 80%)\n",
            "  adding: 11273848.xml (deflated 78%)\n",
            "  adding: 11279407.xml (deflated 82%)\n",
            "  adding: 11285316.xml (deflated 76%)\n",
            "  adding: 11289428.xml (deflated 82%)\n",
            "  adding: 11300865.xml (deflated 77%)\n",
            "  adding: 11302304.xml (deflated 78%)\n",
            "  adding: 11302663.xml (deflated 80%)\n",
            "  adding: 11303641.xml (deflated 83%)\n",
            "  adding: 11304630.xml (deflated 81%)\n",
            "  adding: 11311926.xml (deflated 78%)\n",
            "  adding: 11316331.xml (deflated 76%)\n",
            "  adding: 11318556.xml (deflated 81%)\n",
            "  adding: 11319344.xml (deflated 78%)\n",
            "  adding: 11340806.xml (deflated 78%)\n",
            "  adding: 11341105.xml (deflated 76%)\n",
            "  adding: 11341846.xml (deflated 80%)\n",
            "  adding: 11342206.xml (deflated 81%)\n",
            "  adding: 11342756.xml (deflated 80%)\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# pdf download"
      ],
      "metadata": {
        "id": "BFAgXSPsQYMq"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "pdf_dir = '/content/PMC_pdfs'\n",
        "if not os.path.exists(pdf_dir):\n",
        "    os.makedirs(pdf_dir)"
      ],
      "metadata": {
        "id": "8Tbs9sCrRW1y"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "pmcid_list = []\n",
        "for item in records:\n",
        "    pmcid = item.get('pmcid')\n",
        "    if pmcid:\n",
        "        pmcid_list.append(pmcid)\n",
        "print(pmcid_list)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "AAkHVTu91zkh",
        "outputId": "cb7114fa-2ed6-4770-d03a-91833cb434c7"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "['11342756', '11342206', '11341105', '11340603', '11340327', '11326835', '11324020', '11323402', '11322368', '11319344', '11318556', '11316331', '11315073', '11313837', '11313783', '11312530', '11312258', '11312129', '11311926', '11311727', '11304630', '11304591', '11303641', '11302663', '11302304', '11300865', '11237486', '11293832', '11293673', '11291472', '11289428', '11288818', '11285316', '11281493', '11279407', '11278787', '11273848', '11272550', '11269248', '11265448', '11263920', '11262882', '11261454', '11260827', '11257890', '11249912', '11246856', '11244529', '11243892', '11243103', '11242183', '11240787', '11240451', '11238125', '11343591', '11341846', '11340806', '11340127', '11339456']\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "print(len(pmcid_list))"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "ykzo57gu12eY",
        "outputId": "3a7a0da5-6f92-44de-d8aa-ba4a05ff55b2"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "59\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "records = []\n",
        "# Europe PMC API URL\n",
        "for PMCID in pmcid_list:\n",
        "    europe_pmc_url = f'https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC{PMCID}&resultType=core&format=json'\n",
        "\n",
        "    try:\n",
        "        response = requests.get(europe_pmc_url, timeout=10)\n",
        "\n",
        "        if response.status_code == 200:\n",
        "            print(europe_pmc_url)\n",
        "            result_json = response.json()\n",
        "\n",
        "            for article in result_json.get(\"resultList\", {}).get(\"result\", []):\n",
        "                record = {\n",
        "                    'pmcid': article.get('pmcid', None),\n",
        "                    'pmid': article.get('pmid', None),\n",
        "                    'doi': article.get('doi', None),\n",
        "                    'title': article.get('title', None),\n",
        "                    'abstractText': article.get('abstractText', None),\n",
        "                    'pdf_urls': None,\n",
        "                    'printPublicationDate': article.get('journalInfo', {}).get('printPublicationDate', None),\n",
        "                    'dateOfCreation': article.get('dateOfCreation', None),\n",
        "                    'firstIndexDate': article.get('firstIndexDate', None),\n",
        "                    'fullTextReceivedDate': article.get('fullTextReceivedDate', None),\n",
        "                    'dateOfRevision': article.get('dateOfRevision', None),\n",
        "                    'electronicPublicationDate': article.get('electronicPublicationDate', None),\n",
        "                    'firstPublicationDate': article.get('firstPublicationDate', None),\n",
        "                    'citedByCount': article.get('citedByCount', None)\n",
        "                }\n",
        "\n",
        "                # Open Access PDF URL 추출\n",
        "                if article.get(\"isOpenAccess\", False):\n",
        "                    pdf_link = article.get(\"fullTextUrlList\", {}).get(\"fullTextUrl\", [])\n",
        "                    pdf_list = [link.get(\"url\") for link in pdf_link if link.get(\"documentStyle\", \"\") == \"pdf\"\n",
        "                                and (link.get(\"availability\", \"\") == \"Open access\"\n",
        "                                     or link.get(\"availability\", \"\") == \"Free\")]\n",
        "                    if pdf_list:\n",
        "                        record['pdf_urls'] = pdf_list\n",
        "\n",
        "                        # timeout / error handling\n",
        "                        for attempt, url in enumerate(record['pdf_urls']):\n",
        "                            headers = {\n",
        "                                'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36'\n",
        "                            }\n",
        "                            try:\n",
        "                                pdf_response = requests.get(url, headers=headers, verify=False, timeout=10)\n",
        "                                if pdf_response.status_code == 200:\n",
        "                                    pdf_path = os.path.join(pdf_dir, f\"PMCID_{PMCID}.pdf\")\n",
        "                                    with open(pdf_path, 'wb') as f:\n",
        "                                        f.write(pdf_response.content)\n",
        "                                        print(f\"Downloaded: {pdf_path}\")\n",
        "                                        break\n",
        "                                else:\n",
        "                                    print(f\"Failed to download {url}, status code: {pdf_response.status_code}\")\n",
        "                                    time.sleep(2)\n",
        "                            except requests.exceptions.RequestException as e:\n",
        "                                print(f\"Error occurred for {url}: {e}\")\n",
        "                                time.sleep(2)\n",
        "\n",
        "                records.append(record)\n",
        "\n",
        "    except requests.exceptions.Timeout as e:\n",
        "        print(f\"Timeout error occurred for PMC{PMCID}: {e}\")\n",
        "    except requests.exceptions.RequestException as e:\n",
        "        print(f\"Request error occurred for PMC{PMCID}: {e}\")\n",
        "\n",
        "df = pd.DataFrame(records)\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "BSSNYuoJ8zG6",
        "outputId": "f000ed11-066c-4f76-b7d8-12ed1b27ec40"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11342756&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11342756.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11342206&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.ncbi.nlm.nih.gov'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.ncbi.nlm.nih.gov'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11342206.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11341105&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.ncbi.nlm.nih.gov'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.ncbi.nlm.nih.gov'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11341105.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11340603&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.ncbi.nlm.nih.gov'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.ncbi.nlm.nih.gov'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11340603.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11340327&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11340327.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11326835&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11326835.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11324020&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11324020.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11323402&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11323402.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11322368&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11322368.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11319344&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11319344.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11318556&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11318556.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11316331&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11316331.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11315073&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11315073.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11313837&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.mdpi.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'mdpi-res.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11313837.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11313783&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.mdpi.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'mdpi-res.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11313783.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11312530&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11312530.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11312258&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11312258.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11312129&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11312129.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11311926&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'gutpathogens.biomedcentral.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'gutpathogens.biomedcentral.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11311926.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11311727&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11311727.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11304630&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11304630.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11304591&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11304591.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11303641&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11303641.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11302663&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11302663.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11302304&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11302304.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11300865&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11300865.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11237486&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11237486.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11293832&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11293832.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11293673&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11293673.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11291472&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11291472.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11289428&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11289428.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11288818&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.frontiersin.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11288818.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11285316&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11285316.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11281493&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11281493.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11279407&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11279407.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11278787&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11278787.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11273848&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11273848.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11272550&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11272550.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11269248&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11269248.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11265448&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11265448.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11263920&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11263920.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11262882&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11262882.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11261454&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11261454.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11260827&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11260827.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11257890&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11257890.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11249912&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11249912.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11246856&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11246856.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11244529&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11244529.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11243892&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.ncbi.nlm.nih.gov'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.ncbi.nlm.nih.gov'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11243892.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11243103&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.ncbi.nlm.nih.gov'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.ncbi.nlm.nih.gov'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11243103.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11242183&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.ncbi.nlm.nih.gov'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.ncbi.nlm.nih.gov'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11242183.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11240787&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.ncbi.nlm.nih.gov'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.ncbi.nlm.nih.gov'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11240787.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11240451&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.ncbi.nlm.nih.gov'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.ncbi.nlm.nih.gov'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11240451.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11238125&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11238125.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11343591&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'europepmc.org'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11343591.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11341846&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.ncbi.nlm.nih.gov'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.ncbi.nlm.nih.gov'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11341846.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11340806&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.ncbi.nlm.nih.gov'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.ncbi.nlm.nih.gov'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11340806.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11340127&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.ncbi.nlm.nih.gov'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.ncbi.nlm.nih.gov'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11340127.pdf\n",
            "https://www.ebi.ac.uk/europepmc/webservices/rest/search?query=PMCID:PMC11339456&resultType=core&format=json\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.ncbi.nlm.nih.gov'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n",
            "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py:1100: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.ncbi.nlm.nih.gov'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Downloaded: /content/PMC_pdfs/PMCID_11339456.pdf\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "df.head()"
      ],
      "metadata": {
        "id": "gJO3x526mv7G",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 486
        },
        "outputId": "68f2365c-74d3-4081-bfae-2eef16b5227c"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "        pmcid      pmid                            doi  \\\n",
              "0  PMC9493680  36160211      10.3389/fmicb.2022.996794   \n",
              "1  PMC8414663  34479650     10.1186/s13104-021-05759-6   \n",
              "2  PMC7587089  32817086           10.1128/jcm.01493-20   \n",
              "3  PMC7492956  32963655           10.1155/2020/9373984   \n",
              "4  PMC7461712  32884951  10.30498/ijb.2020.127412.2244   \n",
              "\n",
              "                                               title  \\\n",
              "0  Detection of <i>Escherichia coli</i>, <i>Pseud...   \n",
              "1  RAPD and ERIC-PCR coupled with HRM for species...   \n",
              "2  Additional Og-Typing PCR Techniques Targeting ...   \n",
              "3  Development and Evaluation of an iiPCR Assay f...   \n",
              "4  Development and Application of an Immunocaptur...   \n",
              "\n",
              "                                        abstractText  \\\n",
              "0  To eliminate the influences of excipients and ...   \n",
              "1  <h4>Objective</h4>Species identification of Sh...   \n",
              "2  The O-serogrouping of pathogenic <i>Escherichi...   \n",
              "3  <h4>Background</h4><i>Salmonella</i> and <i>Sh...   \n",
              "4  <h4>Background</h4><i>Shigella</i> is among th...   \n",
              "\n",
              "                                            pdf_urls printPublicationDate  \\\n",
              "0  [https://www.frontiersin.org/articles/10.3389/...           2022-01-01   \n",
              "1  [https://bmcresnotes.biomedcentral.com/counter...           2021-09-01   \n",
              "2  [https://europepmc.org/articles/PMC7587089?pdf...           2020-10-01   \n",
              "3  [https://downloads.hindawi.com/journals/cjidmm...           2020-01-01   \n",
              "4  [https://europepmc.org/articles/PMC7461712?pdf...           2020-01-01   \n",
              "\n",
              "  dateOfCreation firstIndexDate fullTextReceivedDate dateOfRevision  \\\n",
              "0     2022-09-26     2022-09-28           2022-09-24     2022-09-28   \n",
              "1     2021-09-04     2021-09-05           2021-09-14     2021-09-10   \n",
              "2     2020-08-21     2020-08-21           2021-04-27     2021-06-23   \n",
              "3     2020-09-23     2020-09-24           2020-09-24     2022-04-17   \n",
              "4     2020-09-05     2020-09-05           2020-09-04     2022-04-16   \n",
              "\n",
              "  electronicPublicationDate firstPublicationDate  citedByCount  \n",
              "0                2022-09-08           2022-09-08             0  \n",
              "1                2021-09-03           2021-09-03             1  \n",
              "2                2020-10-21           2020-10-21            13  \n",
              "3                2020-09-07           2020-09-07             0  \n",
              "4                2020-01-01           2020-01-01             2  "
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-1040fe92-4548-473a-a14e-80dc7910ee4c\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>pmcid</th>\n",
              "      <th>pmid</th>\n",
              "      <th>doi</th>\n",
              "      <th>title</th>\n",
              "      <th>abstractText</th>\n",
              "      <th>pdf_urls</th>\n",
              "      <th>printPublicationDate</th>\n",
              "      <th>dateOfCreation</th>\n",
              "      <th>firstIndexDate</th>\n",
              "      <th>fullTextReceivedDate</th>\n",
              "      <th>dateOfRevision</th>\n",
              "      <th>electronicPublicationDate</th>\n",
              "      <th>firstPublicationDate</th>\n",
              "      <th>citedByCount</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>PMC9493680</td>\n",
              "      <td>36160211</td>\n",
              "      <td>10.3389/fmicb.2022.996794</td>\n",
              "      <td>Detection of &lt;i&gt;Escherichia coli&lt;/i&gt;, &lt;i&gt;Pseud...</td>\n",
              "      <td>To eliminate the influences of excipients and ...</td>\n",
              "      <td>[https://www.frontiersin.org/articles/10.3389/...</td>\n",
              "      <td>2022-01-01</td>\n",
              "      <td>2022-09-26</td>\n",
              "      <td>2022-09-28</td>\n",
              "      <td>2022-09-24</td>\n",
              "      <td>2022-09-28</td>\n",
              "      <td>2022-09-08</td>\n",
              "      <td>2022-09-08</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>PMC8414663</td>\n",
              "      <td>34479650</td>\n",
              "      <td>10.1186/s13104-021-05759-6</td>\n",
              "      <td>RAPD and ERIC-PCR coupled with HRM for species...</td>\n",
              "      <td>&lt;h4&gt;Objective&lt;/h4&gt;Species identification of Sh...</td>\n",
              "      <td>[https://bmcresnotes.biomedcentral.com/counter...</td>\n",
              "      <td>2021-09-01</td>\n",
              "      <td>2021-09-04</td>\n",
              "      <td>2021-09-05</td>\n",
              "      <td>2021-09-14</td>\n",
              "      <td>2021-09-10</td>\n",
              "      <td>2021-09-03</td>\n",
              "      <td>2021-09-03</td>\n",
              "      <td>1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>PMC7587089</td>\n",
              "      <td>32817086</td>\n",
              "      <td>10.1128/jcm.01493-20</td>\n",
              "      <td>Additional Og-Typing PCR Techniques Targeting ...</td>\n",
              "      <td>The O-serogrouping of pathogenic &lt;i&gt;Escherichi...</td>\n",
              "      <td>[https://europepmc.org/articles/PMC7587089?pdf...</td>\n",
              "      <td>2020-10-01</td>\n",
              "      <td>2020-08-21</td>\n",
              "      <td>2020-08-21</td>\n",
              "      <td>2021-04-27</td>\n",
              "      <td>2021-06-23</td>\n",
              "      <td>2020-10-21</td>\n",
              "      <td>2020-10-21</td>\n",
              "      <td>13</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>PMC7492956</td>\n",
              "      <td>32963655</td>\n",
              "      <td>10.1155/2020/9373984</td>\n",
              "      <td>Development and Evaluation of an iiPCR Assay f...</td>\n",
              "      <td>&lt;h4&gt;Background&lt;/h4&gt;&lt;i&gt;Salmonella&lt;/i&gt; and &lt;i&gt;Sh...</td>\n",
              "      <td>[https://downloads.hindawi.com/journals/cjidmm...</td>\n",
              "      <td>2020-01-01</td>\n",
              "      <td>2020-09-23</td>\n",
              "      <td>2020-09-24</td>\n",
              "      <td>2020-09-24</td>\n",
              "      <td>2022-04-17</td>\n",
              "      <td>2020-09-07</td>\n",
              "      <td>2020-09-07</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>PMC7461712</td>\n",
              "      <td>32884951</td>\n",
              "      <td>10.30498/ijb.2020.127412.2244</td>\n",
              "      <td>Development and Application of an Immunocaptur...</td>\n",
              "      <td>&lt;h4&gt;Background&lt;/h4&gt;&lt;i&gt;Shigella&lt;/i&gt; is among th...</td>\n",
              "      <td>[https://europepmc.org/articles/PMC7461712?pdf...</td>\n",
              "      <td>2020-01-01</td>\n",
              "      <td>2020-09-05</td>\n",
              "      <td>2020-09-05</td>\n",
              "      <td>2020-09-04</td>\n",
              "      <td>2022-04-16</td>\n",
              "      <td>2020-01-01</td>\n",
              "      <td>2020-01-01</td>\n",
              "      <td>2</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-1040fe92-4548-473a-a14e-80dc7910ee4c')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-1040fe92-4548-473a-a14e-80dc7910ee4c button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-1040fe92-4548-473a-a14e-80dc7910ee4c');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-603fc6fd-4cc8-43a6-a41c-b08ced3ce7fb\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-603fc6fd-4cc8-43a6-a41c-b08ced3ce7fb')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-603fc6fd-4cc8-43a6-a41c-b08ced3ce7fb button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "variable_name": "df",
              "summary": "{\n  \"name\": \"df\",\n  \"rows\": 38,\n  \"fields\": [\n    {\n      \"column\": \"pmcid\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 38,\n        \"samples\": [\n          \"PMC284770\",\n          \"PMC182838\",\n          \"PMC7461712\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"pmid\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 38,\n        \"samples\": [\n          \"7840594\",\n          \"2059028\",\n          \"32884951\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"doi\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 34,\n        \"samples\": [\n          \"10.1128/jcm.01397-13\",\n          \"10.4269/ajtmh.2012.11-0671\",\n          \"10.1128/aem.68.5.2580-2583.2002\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"title\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 38,\n        \"samples\": [\n          \"Detection of 4-quinolone resistance mutation in gyrA gene of Shigella dysenteriae type 1 by PCR.\",\n          \"Detection of Escherichia coli and Shigella spp. in water by using the polymerase chain reaction and gene probes for uid.\",\n          \"Development and Application of an Immunocapture PCR Diagnostic Assay Based on the Monoclonal Antibody for the Detection of <i>Shigella</i>.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"abstractText\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"num_unique_values\": 38,\n        \"samples\": [\n          \"To study 4-quinolone resistance, the N-terminal coding region of gyrA from nalidixic acid-susceptible and -resistant isolates of Shigella dysenteriae type 1 was amplified by PCR, cloned, and sequenced. DNA sequence analysis of gyrA from nalidixic acid-resistant isolates revealed a C-to-T transition at nucleotide position 248 leading to a Ser-83-to-Leu substitution which was absent in susceptible clinical isolates. Direct HinfI digestion of PCR-amplified DNA detected similar mutations. Thus, DNA gyrase A subunit mutation Ser-83 to Leu is implicated in 4-quinolone resistance in S. dysenteriae type 1.\",\n          \"A method was developed for the detection of the fecal coliform bacterium Escherichia coli, using the polymerase chain reaction and gene probes, based on amplifying regions of the uid gene that code for beta-glucuronidase, expression of which forms the basis for fecal coliform detection by the commercially available Colilert method. Amplification and gene probe detection of four different regions of uid specifically detected E. coli and Shigella species, including beta-glucuronidase-negative strains of E. coli; no amplification was observed for other coliform and nonenteric bacteria.\",\n          \"<h4>Background</h4><i>Shigella</i> is among the most important human pathogenic microorganisms, infecting both humans and nonhuman and causing clinically severe diarrhea. <i>Shigella</i> must be enriched before detection, which is time-consuming.<h4>Objectives</h4>To develop a sensitive, rapid, and specific method for <i>Shigella</i> detection.<h4>Materials and methods</h4><i>Shigella</i> was used as an antigen to generate monoclonal antibodies (mAbs). mAbs were screened via indirect enzyme-linked immunosorbent assay (ELISA) and western blot, and two mAbs were selected. The mAb A3 showed high affinity and specificity and was used to develop immune magnetic beads (IMBs) for <i>Shigella</i> enrichment. An immunocapture (IC)-PCR primer was designed from the <i>ipaH</i> gene, and IC-PCR was developed based on the IMBs and PCR.<h4>Results</h4>This system shortened the <i>Shigella</i> detection time to 70 min. The sensitivity of the IC-PCR was 9 colony-forming units.mL<sup>-1</sup> in artificial milk. The accuracy of the IC-PCR was confirmed using 46 clinical samples collected from monkeys. The IC-PCR results were consistent with the serological and biochemical assays.<h4>Conclusion</h4>The IC-PCR described herein accurately detected <i>Shigella</i> from milk samples, monkeys and can thus be used to complement classical detection methods.\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"pdf_urls\",\n      \"properties\": {\n        \"dtype\": \"object\",\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"printPublicationDate\",\n      \"properties\": {\n        \"dtype\": \"object\",\n        \"num_unique_values\": 34,\n        \"samples\": [\n          \"2013-09-01\",\n          \"2012-01-01\",\n          \"2002-05-01\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"dateOfCreation\",\n      \"properties\": {\n        \"dtype\": \"object\",\n        \"num_unique_values\": 38,\n        \"samples\": [\n          \"1994-10-01\",\n          \"1991-04-01\",\n          \"2020-09-05\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"firstIndexDate\",\n      \"properties\": {\n        \"dtype\": \"object\",\n        \"num_unique_values\": 31,\n        \"samples\": [\n          \"2010-09-24\",\n          \"2014-02-22\",\n          \"2012-06-06\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"fullTextReceivedDate\",\n      \"properties\": {\n        \"dtype\": \"object\",\n        \"num_unique_values\": 15,\n        \"samples\": [\n          \"2020-07-08\",\n          \"2020-10-27\",\n          \"2022-09-24\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"dateOfRevision\",\n      \"properties\": {\n        \"dtype\": \"object\",\n        \"num_unique_values\": 23,\n        \"samples\": [\n          \"2021-10-21\",\n          \"2020-10-01\",\n          \"2022-09-28\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"electronicPublicationDate\",\n      \"properties\": {\n        \"dtype\": \"object\",\n        \"num_unique_values\": 25,\n        \"samples\": [\n          \"2015-12-14\",\n          \"2013-03-27\",\n          \"2022-09-08\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"firstPublicationDate\",\n      \"properties\": {\n        \"dtype\": \"object\",\n        \"num_unique_values\": 38,\n        \"samples\": [\n          \"1994-10-01\",\n          \"1991-04-01\",\n          \"2020-01-01\"\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"citedByCount\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 27,\n        \"min\": 0,\n        \"max\": 137,\n        \"num_unique_values\": 30,\n        \"samples\": [\n          32,\n          30,\n          101\n        ],\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"
            }
          },
          "metadata": {},
          "execution_count": 20
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "len(df)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "d0PH70v-9oCl",
        "outputId": "0177b7c8-6341-45a1-8dcd-ef0b28c8ebc8"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "59"
            ]
          },
          "metadata": {},
          "execution_count": 16
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "df['pdf_urls'].notnull().sum()"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "lKq1CT6Y9pgf",
        "outputId": "e8f07e7f-901e-442e-9e21-408f3cd22565"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "59"
            ]
          },
          "metadata": {},
          "execution_count": 17
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "df.to_csv('europepmc_pmcid_result.csv', index='False')"
      ],
      "metadata": {
        "id": "MgcSoLcD-613"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "!ls -l /content/PMC_pdfs | wc -l"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "j-4HK0R7Mv8f",
        "outputId": "5156d795-b25e-448a-ba0e-195442babb07"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "60\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!zip -r /content/PMC_pdfs.zip /content/PMC_pdfs"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "QFsmnoVFOXEA",
        "outputId": "0567992b-a7e3-4678-feb2-280d58eebb7a"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "  adding: content/PMC_pdfs/ (stored 0%)\n",
            "  adding: content/PMC_pdfs/PMCID_10965209.pdf (deflated 15%)\n",
            "  adding: content/PMC_pdfs/PMCID_10991388.pdf (deflated 14%)\n",
            "  adding: content/PMC_pdfs/PMCID_11052509.pdf (deflated 8%)\n",
            "  adding: content/PMC_pdfs/PMCID_11221667.pdf (deflated 16%)\n",
            "  adding: content/PMC_pdfs/PMCID_11281682.pdf (deflated 4%)\n",
            "  adding: content/PMC_pdfs/PMCID_11097627.pdf (deflated 15%)\n",
            "  adding: content/PMC_pdfs/PMCID_11013254.pdf (deflated 10%)\n",
            "  adding: content/PMC_pdfs/PMCID_10974990.pdf (deflated 11%)\n",
            "  adding: content/PMC_pdfs/PMCID_11275270.pdf (deflated 10%)\n",
            "  adding: content/PMC_pdfs/PMCID_10974247.pdf (deflated 7%)\n",
            "  adding: content/PMC_pdfs/PMCID_11279579.pdf (deflated 24%)\n",
            "  adding: content/PMC_pdfs/PMCID_11270334.pdf (deflated 6%)\n",
            "  adding: content/PMC_pdfs/PMCID_11288998.pdf (deflated 16%)\n",
            "  adding: content/PMC_pdfs/PMCID_11142709.pdf (deflated 15%)\n",
            "  adding: content/PMC_pdfs/PMCID_11253681.pdf (deflated 11%)\n",
            "  adding: content/PMC_pdfs/PMCID_11254000.pdf (deflated 29%)\n",
            "  adding: content/PMC_pdfs/PMCID_11014864.pdf (deflated 29%)\n",
            "  adding: content/PMC_pdfs/PMCID_11043942.pdf (deflated 13%)\n",
            "  adding: content/PMC_pdfs/PMCID_11281436.pdf (deflated 17%)\n",
            "  adding: content/PMC_pdfs/PMCID_11302493.pdf (deflated 12%)\n",
            "  adding: content/PMC_pdfs/PMCID_11253070.pdf (deflated 18%)\n",
            "  adding: content/PMC_pdfs/PMCID_11298542.pdf (deflated 17%)\n",
            "  adding: content/PMC_pdfs/PMCID_11293362.pdf (deflated 17%)\n",
            "  adding: content/PMC_pdfs/PMCID_11312644.pdf (deflated 14%)\n",
            "  adding: content/PMC_pdfs/PMCID_11318531.pdf (deflated 28%)\n",
            "  adding: content/PMC_pdfs/PMCID_11209508.pdf (deflated 10%)\n",
            "  adding: content/PMC_pdfs/PMCID_11122424.pdf (deflated 2%)\n",
            "  adding: content/PMC_pdfs/PMCID_10952654.pdf (deflated 39%)\n",
            "  adding: content/PMC_pdfs/PMCID_11300543.pdf (deflated 13%)\n",
            "  adding: content/PMC_pdfs/PMCID_11065202.pdf (deflated 16%)\n",
            "  adding: content/PMC_pdfs/PMCID_11209087.pdf (deflated 14%)\n",
            "  adding: content/PMC_pdfs/PMCID_11239677.pdf (deflated 20%)\n",
            "  adding: content/PMC_pdfs/PMCID_11263921.pdf (deflated 20%)\n",
            "  adding: content/PMC_pdfs/PMCID_11142717.pdf (deflated 9%)\n",
            "  adding: content/PMC_pdfs/PMCID_11031556.pdf (deflated 20%)\n",
            "  adding: content/PMC_pdfs/PMCID_11334581.pdf (deflated 13%)\n",
            "  adding: content/PMC_pdfs/PMCID_11304323.pdf (deflated 4%)\n",
            "  adding: content/PMC_pdfs/PMCID_10975712.pdf (deflated 2%)\n",
            "  adding: content/PMC_pdfs/PMCID_11283106.pdf (deflated 19%)\n",
            "  adding: content/PMC_pdfs/PMCID_11292847.pdf (deflated 8%)\n",
            "  adding: content/PMC_pdfs/PMCID_10975468.pdf (deflated 8%)\n",
            "  adding: content/PMC_pdfs/PMCID_11227793.pdf (deflated 31%)\n",
            "  adding: content/PMC_pdfs/PMCID_11125733.pdf (deflated 8%)\n",
            "  adding: content/PMC_pdfs/PMCID_11075047.pdf (deflated 27%)\n",
            "  adding: content/PMC_pdfs/PMCID_11077944.pdf (deflated 9%)\n",
            "  adding: content/PMC_pdfs/PMCID_10994276.pdf (deflated 22%)\n",
            "  adding: content/PMC_pdfs/PMCID_11027473.pdf (deflated 11%)\n",
            "  adding: content/PMC_pdfs/PMCID_11237389.pdf (deflated 14%)\n",
            "  adding: content/PMC_pdfs/PMCID_10977837.pdf (deflated 3%)\n",
            "  adding: content/PMC_pdfs/PMCID_11126107.pdf (deflated 17%)\n",
            "  adding: content/PMC_pdfs/PMCID_11281464.pdf (deflated 7%)\n",
            "  adding: content/PMC_pdfs/PMCID_11229740.pdf (deflated 21%)\n",
            "  adding: content/PMC_pdfs/PMCID_11042293.pdf (deflated 9%)\n",
            "  adding: content/PMC_pdfs/PMCID_11281640.pdf (deflated 6%)\n",
            "  adding: content/PMC_pdfs/PMCID_11245005.pdf (deflated 17%)\n",
            "  adding: content/PMC_pdfs/PMCID_11007421.pdf (deflated 9%)\n",
            "  adding: content/PMC_pdfs/PMCID_11018020.pdf (deflated 20%)\n",
            "  adding: content/PMC_pdfs/PMCID_11338879.pdf (deflated 10%)\n",
            "  adding: content/PMC_pdfs/PMCID_11008856.pdf (deflated 14%)\n",
            "  adding: content/PMC_pdfs/PMCID_11280016.pdf (deflated 11%)\n",
            "  adding: content/PMC_pdfs/PMCID_11297268.pdf (deflated 27%)\n",
            "  adding: content/PMC_pdfs/PMCID_11245052.pdf (deflated 12%)\n",
            "  adding: content/PMC_pdfs/PMCID_11259601.pdf (deflated 14%)\n",
            "  adding: content/PMC_pdfs/PMCID_10975951.pdf (deflated 6%)\n",
            "  adding: content/PMC_pdfs/PMCID_11050819.pdf (deflated 11%)\n",
            "  adding: content/PMC_pdfs/PMCID_11311727.pdf (deflated 13%)\n",
            "  adding: content/PMC_pdfs/PMCID_10957905.pdf (deflated 16%)\n",
            "  adding: content/PMC_pdfs/PMCID_11316927.pdf (deflated 23%)\n",
            "  adding: content/PMC_pdfs/PMCID_11319304.pdf (deflated 45%)\n",
            "  adding: content/PMC_pdfs/PMCID_11118645.pdf (deflated 8%)\n",
            "  adding: content/PMC_pdfs/PMCID_11324105.pdf (deflated 11%)\n",
            "  adding: content/PMC_pdfs/PMCID_11054155.pdf (deflated 24%)\n"
          ]
        }
      ]
    }
  ]
}
